# Persian translation of anaconda to Persian.
# Copyright (C) 2003, 2004 Abbas Izad
# Copyright (C) 2004 Hedayat Vatankhah
# Copyright (C) 2005, 2006 Sharif FarsiWeb, Inc.
# This file is distributed under the same license as the anaconda package.
# Abbas Izad <abbasizad@hotmail.com>, 2003, 2004.
# Hedayat Vatankhah <hedavat@mail15.com>, 2004.
# Roozbeh Pournader <roozbeh@farsiweb.info>, 2005, 2006.
# Hamed Malek <hamed@farsiweb.info>, 2005.
# Meelad Zakaria <meelad@farsiweb.info>, 2005, 2006.
# Elnaz Sarbar <elnaz@farsiweb.info>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: anaconda\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2010-08-07 10:23+0200\n"
"PO-Revision-Date: 2006-05-13 15:09+0330\n"
"Last-Translator: Meelad Zakaria <meelad@farsiweb.info>\n"
"Language-Team: Persian\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: anaconda:342
msgid "Press <enter> for a shell"
msgstr ""

#: anaconda:357 rescue.py:332 rescue.py:358 rescue.py:371 rescue.py:454
#: rescue.py:466 text.py:657 loader/cdinstall.c:228 loader/cdinstall.c:232
#: loader/cdinstall.c:248 loader/cdinstall.c:251 loader/cdinstall.c:427
#: loader/cdinstall.c:431 loader/cdinstall.c:504 loader/dirbrowser.c:129
#: loader/driverdisk.c:266 loader/driverdisk.c:422 loader/driverdisk.c:460
#: loader/driverdisk.c:491 loader/driverdisk.c:526 loader/driverdisk.c:542
#: loader/driverdisk.c:554 loader/driverdisk.c:562 loader/driverdisk.c:722
#: loader/driverdisk.c:761 loader/driverselect.c:79 loader/driverselect.c:179
#: loader/hdinstall.c:115 loader/hdinstall.c:260 loader/hdinstall.c:318
#: loader/hdinstall.c:352 loader/hdinstall.c:424 loader/hdinstall.c:467
#: loader/hdinstall.c:480 loader/kbd.c:119 loader/kickstart.c:132
#: loader/kickstart.c:142 loader/kickstart.c:184 loader/kickstart.c:189
#: loader/kickstart.c:296 loader/kickstart.c:328 loader/kickstart.c:510
#: loader/lang.c:114 loader/lang.c:373 loader/loader.c:429 loader/loader.c:465
#: loader/loader.c:505 loader/loader.c:523 loader/loader.c:540
#: loader/loader.c:577 loader/loader.c:1194 loader/loader.c:1375
#: loader/mediacheck.c:47 loader/mediacheck.c:86 loader/mediacheck.c:93
#: loader/mediacheck.c:97 loader/mediacheck.c:106 loader/method.c:277
#: loader/method.c:351 loader/modules.c:382 loader/modules.c:398
#: loader/net.c:578 loader/net.c:949 loader/net.c:1557 loader/net.c:1578
#: loader/net.c:1848 loader/net.c:1867 loader/net.c:1879
#: loader/nfsinstall.c:91 loader/nfsinstall.c:279 loader/nfsinstall.c:296
#: loader/nfsinstall.c:366 loader/telnetd.c:93 loader/urlinstall.c:137
#: loader/urlinstall.c:159 loader/urlinstall.c:221 loader/urlinstall.c:372
#: loader/urlinstall.c:383 loader/urlinstall.c:390 loader/urls.c:259
#: loader/urls.c:330 loader/urls.c:336 textw/constants_text.py:44
msgid "OK"
msgstr "تأیید"

#: anaconda:364
#, fuzzy
msgid "You do not have enough RAM to use the graphical installer."
msgstr ""
"برای استفاده از نصب کنندهٔ گرافیکی به قدر کافی حافظهٔ RAM ندارید. حالت متنی "
"آغاز می‌شود."

#: anaconda:368
msgid ""
"Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""

#: anaconda:371
#, fuzzy
msgid "Not enough RAM"
msgstr "فضای کافی وجود ندارد"

#: anaconda:381
#, fuzzy
msgid "Starting text mode."
msgstr "استفاده از حالت متنی"

#: anaconda:638
#, fuzzy
msgid "Starting graphical installation."
msgstr "نصب گرافیکی آغاز می‌شود..."

#: anaconda:1062
#, fuzzy
msgid "Would you like to use VNC?"
msgstr "می‌خواهید چه‌کار کنید؟"

#: anaconda:1063
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""

#: anaconda:1089
#, fuzzy
msgid "Graphical installation is not available. Starting text mode."
msgstr "نصب گرافیکی ممکن نیست...  حالت متنی آغاز می‌شود."

#: anaconda:1097
msgid "DISPLAY variable not set. Starting text mode."
msgstr ""

#: backend.py:135
#, fuzzy, python-format
msgid "Upgrading %s\n"
msgstr "در حال ارتقاء دادن %s-%s-%s.%s.\n"

#: backend.py:137
#, fuzzy, python-format
msgid "Installing %s\n"
msgstr "_نصب %s"

#: backend.py:150
msgid "Copying File"
msgstr "در حال نسخه‌برداری از پرونده"

#: backend.py:151
#, fuzzy
msgid "Transferring install image to hard drive"
msgstr "در حال انتقال تصویر نصب به دیسک‌سخت..."

#: backend.py:161
#, fuzzy
msgid ""
"An error occurred transferring the install image to your hard drive.  This "
"is often cause by damaged or low quality media."
msgstr ""
"هنگام انتقال تصویر نصب به دیسک‌سخت شما خطایی رخ داد. احتمالاً دیسک‌تان جا ندارد."

#: backend.py:165
msgid ""
"An error occurred transferring the install image to your hard drive. You are "
"probably out of disk space."
msgstr ""
"هنگام انتقال تصویر نصب به دیسک‌سخت شما خطایی رخ داد. احتمالاً دیسک‌تان جا ندارد."

#: backend.py:169 image.py:289 livecd.py:198 livecd.py:400
#: partIntfHelpers.py:237 text.py:355 text.py:359 yuminstall.py:416
#: yuminstall.py:805 yuminstall.py:923 yuminstall.py:928 yuminstall.py:1190
#: yuminstall.py:1249 yuminstall.py:1440 yuminstall.py:1462
#: iw/advanced_storage.py:90 iw/advanced_storage.py:100
#: iw/advanced_storage.py:176 iw/advanced_storage.py:179
#: iw/advanced_storage.py:207 iw/autopart_type.py:96 iw/cleardisks_gui.py:43
#: iw/cleardisks_gui.py:53 iw/filter_gui.py:408 iw/osbootwidget.py:211
#: iw/osbootwidget.py:220 iw/raid_dialog_gui.py:215 iw/raid_dialog_gui.py:766
#: iw/raid_dialog_gui.py:805 iw/task_gui.py:54 iw/task_gui.py:155
#: iw/task_gui.py:302 iw/task_gui.py:463 loader/cdinstall.c:228
#: loader/cdinstall.c:504 loader/driverdisk.c:266 loader/driverdisk.c:491
#: loader/driverdisk.c:526 loader/driverdisk.c:554 loader/driverdisk.c:562
#: loader/driverdisk.c:626 loader/hdinstall.c:115 loader/hdinstall.c:318
#: loader/hdinstall.c:424 loader/hdinstall.c:467 loader/hdinstall.c:480
#: loader/kickstart.c:296 loader/lang.c:114 loader/loader.c:429
#: loader/loader.c:540 loader/loader.c:1194 loader/mediacheck.c:47
#: loader/mediacheck.c:86 loader/mediacheck.c:93 loader/mediacheck.c:97
#: loader/method.c:277 loader/method.c:351 loader/nfsinstall.c:279
#: loader/nfsinstall.c:296 loader/telnetd.c:93 loader/urlinstall.c:137
#: loader/urlinstall.c:159 loader/urlinstall.c:221 loader/urls.c:330
#: loader/urls.c:336 storage/__init__.py:215 storage/__init__.py:1785
#: storage/__init__.py:1872 textw/netconfig_text.py:282
#: textw/partition_text.py:186 textw/partition_text.py:192
#: textw/partition_text.py:198 textw/partition_text.py:227
#: textw/partition_text.py:273 textw/upgrade_text.py:181
#: textw/upgrade_text.py:188 sabayon/livecd.py:476 sabayon/utils.py:1005
msgid "Error"
msgstr "خطا"

#: bootloader.py:55 bootloader.py:221 bootloader.py:227 gui.py:1103
#: gui.py:1163 image.py:82 installinterfacebase.py:41 text.py:486 text.py:546
#: yuminstall.py:1418 yuminstall.py:1622 yuminstall.py:1657
#: iw/blpasswidget.py:148 iw/upgrade_swap_gui.py:190
#: iw/upgrade_swap_gui.py:198 iw/upgrade_swap_gui.py:205
#: textw/upgrade_text.py:193
msgid "Warning"
msgstr "اخطار"

#: bootloader.py:56
#, fuzzy
msgid ""
"Filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""
"گزینه‌های افرازبندی‌ای که انتخاب کرده‌اید از قبل فعال شده‌اند. دیگر نمی‌توانید به "
"صفحهٔ ویرایش دیسک بازگردید. آیا می‌خواهید روند نصب را ادامه دهید؟"

#: bootloader.py:60 gui.py:1054 gui.py:1199 gui.py:1406 image.py:91
#: kickstart.py:156 livecd.py:205 livecd.py:407 packages.py:130 upgrade.py:56
#: upgrade.py:237 yuminstall.py:257 yuminstall.py:703 yuminstall.py:925
#: yuminstall.py:930 yuminstall.py:1010 yuminstall.py:1016 yuminstall.py:1169
#: yuminstall.py:1186 yuminstall.py:1239 yuminstall.py:1427 yuminstall.py:1446
#: yuminstall.py:1469 storage/__init__.py:112 storage/__init__.py:219
#: storage/__init__.py:1966 storage/dasd.py:139 sabayon/livecd.py:483
#, fuzzy
msgid "_Exit installer"
msgstr "نصب کنندهٔ %s"

#: bootloader.py:60 image.py:92 image.py:251 kickstart.py:1350
#: kickstart.py:1389 upgrade.py:56 yuminstall.py:1244 yuminstall.py:1428
#: iw/partition_gui.py:1603 storage/__init__.py:1967
msgid "_Continue"
msgstr "_ادامه"

#: bootloader.py:158
msgid "Bootloader"
msgstr "بارکنندهٔ راه‌انداز"

#: bootloader.py:158
#, fuzzy
msgid "Installing bootloader."
msgstr "در حال نصب بارکنندهٔ راه‌انداز..."

#: bootloader.py:222
msgid ""
"There was an error installing the bootloader.  The system may not be "
"bootable."
msgstr ""

#: bootloader.py:228
#, fuzzy
msgid ""
"No kernel packages were installed on the system.  Bootloader configuration "
"will not be changed."
msgstr ""
"هیچ‌کدام از بسته‌ها‌ی هسته روی سیستم شما نصب نشده است. پیکربندی بارکنندهٔ "
"راه‌انداز شما تغییر داده نخواهد شد."

#: cmdline.py:56
msgid "Completed"
msgstr "تمام شد"

#: cmdline.py:64
#, fuzzy
msgid "In progress"
msgstr "در حال پیشرفت...   "

#: cmdline.py:89 gui.py:1194 kickstart.py:1214 kickstart.py:1222
#: kickstart.py:1260 kickstart.py:1268 text.py:392
#, fuzzy, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""
"خطای زیر هنگام تجزیهٔ پیکربندی kickstart پیدا شد:\n"
"\n"
"‏%s"

#: cmdline.py:101
msgid ""
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""

#: cmdline.py:120 cmdline.py:127 cmdline.py:134 cmdline.py:144 cmdline.py:154
msgid "Can't have a question in command line mode!"
msgstr "در حالت سطر فرمان نمی‌شود سؤال پرسید!"

#: constants.py:75
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""

#: constants.py:81
msgid " with the provider of this software."
msgstr ""

#: constants.py:85
#, fuzzy, python-format
msgid " against anaconda at %s"
msgstr "در حال خواندن به‌هنگام‌سازی‌های آناکوندا..."

#: gui.py:110
#, fuzzy
msgid "An error occurred saving screenshots to disk."
msgstr "هنگام نسخه‌برداری از عکس‌های ‌صفحه خطایی رخ داد."

#: gui.py:121
msgid "Screenshots Copied"
msgstr "عکس‌های ‌صفحه نسخه‌برداری شد"

#: gui.py:122
#, fuzzy
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""
"عکس‌های صفحه در شاخهٔ زیر ذخیره شده‌اند:\n"
"\n"
"‏\t/root/anaconda-screenshots/\n"
"\n"
"شما می‌توانید با راه‌اندازی مجدد و ورود به سیستم به عنوان root به آنها دست "
"یابید."

#: gui.py:165
msgid "Saving Screenshot"
msgstr "در حال ذخیرهٔ عکس‌صفحه"

#: gui.py:166
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr "عکس‌صفحه‌ای به نام «%s» ذخیره شد."

#: gui.py:169
msgid "Error Saving Screenshot"
msgstr "خطا در ذخیرهٔ عکس‌صفحه"

#: gui.py:170
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""
"هنگام ذخیرهٔ عکس‌صفحه خطایی رخ داد. اگر خطا هنگام نصب بسته رخ داده است، شاید "
"لازم باشد چندین بار تلاش کنید تا موفق شوید."

#: gui.py:511
#, fuzzy
msgid "Installation Key"
msgstr "روش نصب"

#: gui.py:582 text.py:150
msgid ""
"Choose a passphrase for the encrypted devices. You will be prompted for this "
"passphrase during system boot."
msgstr ""

#: gui.py:601 gui.py:609 text.py:188 text.py:198
#, fuzzy
msgid "Error with passphrase"
msgstr "خطا در گذرواژه"

#: gui.py:602 text.py:189
#, fuzzy
msgid "The passphrases you entered were different.  Please try again."
msgstr "گذرواژه‌هایی که وارد کردید فرق داشتند. لطفاً دوباره تلاش کنید."

#: gui.py:610
#, fuzzy
msgid "The passphrase must be at least eight characters long."
msgstr "طول گذرواژه باید حداقل شش نویسه باشد."

#: gui.py:644 text.py:226
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: gui.py:735 gui.py:1406 partIntfHelpers.py:157 partIntfHelpers.py:342
#: text.py:101 text.py:102 iw/account_gui.py:130 loader/dirbrowser.c:129
#: loader/driverdisk.c:423 loader/kickstart.c:328 loader/loader.c:465
#: loader/loader.c:577 textw/constants_text.py:48
msgid "Cancel"
msgstr "انصراف"

#: gui.py:747 gui.py:748 gui.py:864 gui.py:865 ui/anaconda.glade.h:3
msgid "_Debug"
msgstr "_اشکال‌زدایی"

#: gui.py:1104 text.py:487
#, python-format
msgid ""
"Error processing drive:\n"
"\n"
"%(path)s\n"
"%(size)-0.fMB\n"
"%(description)s\n"
"\n"
"This device may need to be reinitialized.\n"
"\n"
"REINITIALIZING WILL CAUSE ALL DATA TO BE LOST!\n"
"\n"
"This action may also be applied to all other disks needing reinitialization."
"%(details)s"
msgstr ""

#: gui.py:1113 gui.py:1171 text.py:496 text.py:554
#, fuzzy
msgid "_Ignore"
msgstr "نادیده گرفته شود"

#: gui.py:1114 gui.py:1172 text.py:497 text.py:555
#, fuzzy
msgid "Ignore _all"
msgstr "نادیده گرفته شود"

#: gui.py:1115 gui.py:1173 text.py:498 text.py:556
#, fuzzy
msgid "_Re-initialize"
msgstr "راه‌اندازی"

#: gui.py:1116 gui.py:1174 text.py:499 text.py:557
#, fuzzy
msgid "Re-ini_tialize all"
msgstr "راه‌اندازی"

#: gui.py:1164 text.py:547
#, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %(msg)s.  You can reinitialize all related "
"PVs (%(pvs)s) which will erase the LVM metadata, or ignore which will "
"preserve the contents.  This action may also be applied to all other PVs "
"with inconsistent metadata."
msgstr ""

#: gui.py:1196 text.py:394
msgid "Error Parsing Kickstart Config"
msgstr "خطا در تجزیهٔ پیکربندی kickstart"

#: gui.py:1224
msgid "default:LTR"
msgstr "default:RTL"

#: gui.py:1304 text.py:622
msgid "Error!"
msgstr "خطا!"

#: gui.py:1305 text.py:623
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""
"هنگام تلاش برای بار کردن مؤلفهٔ واسط نصب خطایی رخ داد.\n"
"\n"
"className = %s"

#: gui.py:1310 gui.py:1314 image.py:164 image.py:194 packages.py:328
#: storage/__init__.py:1784
msgid "_Exit"
msgstr "_خروج"

#: gui.py:1311 image.py:164 image.py:194 livecd.py:205 yuminstall.py:802
#: yuminstall.py:1186 yuminstall.py:1239 yuminstall.py:1446
msgid "_Retry"
msgstr "_دوباره سعی شود"

#: gui.py:1313
#, fuzzy
msgid "The installer will now exit."
msgstr "حالا برنامه نصب خارج می‌شود..."

#: gui.py:1316
#, fuzzy
msgid "Exiting"
msgstr "خروج"

#: gui.py:1403 livecd.py:126 text.py:271 upgrade.py:228 sabayon/livecd.py:69
#, fuzzy
msgid "Exit installer"
msgstr "نصب کنندهٔ %s"

#: gui.py:1404
#, fuzzy
msgid "Are you sure you wish to exit the installer?"
msgstr "آیا مطمئنید می‌خواهید جلد منطقی «%s» را حذف کنید؟"

#: gui.py:1413
#, python-format
msgid "%s Installer"
msgstr "نصب کنندهٔ %s"

#: gui.py:1419
msgid "Unable to load title bar"
msgstr "بار کردن نوار عنوان ممکن نیست"

#: gui.py:1487
msgid "Install Window"
msgstr "پنجرهٔ نصب"

#: image.py:83
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""

#: image.py:156
msgid "Couldn't Mount ISO Source"
msgstr ""

#: image.py:157
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""

#: image.py:185
msgid "Missing ISO 9660 Image"
msgstr ""

#: image.py:186
#, python-format
msgid ""
"The installer has tried to mount image #%s, but cannot find it on the hard "
"drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""

#: image.py:241
msgid "Required Install Media"
msgstr "رسانهٔ نصب مورد نیاز"

#: image.py:242
#, fuzzy, python-format
msgid ""
"The software you have selected to install will require the following "
"%(productName)s %(productVersion)s discs:\n"
"\n"
"%(reqcdstr)s\n"
"Please have these ready before proceeding with the installation.  If you "
"need to abort the installation and exit please select \"Reboot\"."
msgstr ""
"نرم‌افزارهایی که شما برای نصب انتخاب کرده‌اید به سی‌دی‌های زیر نیاز خواهند "
"داشت:\n"
"\n"
"%s\n"
"لطفاً قبل از ادامهٔ نصب آنها را آماده داشته باشید. اگر احتیاج به قطع کردن نصب "
"و راه‌اندازی مجدد دارید لطفاً روی دکمهٔ «راه‌اندازی مجدد» کلیک کنید."

#: image.py:251 packages.py:332 yuminstall.py:1487
msgid "_Reboot"
msgstr "_راه‌اندازی مجدد"

#: image.py:251 livecd.py:406 packages.py:332 upgrade.py:236 yuminstall.py:930
#: yuminstall.py:1016 yuminstall.py:1427 yuminstall.py:1469 yuminstall.py:1487
#: ui/anaconda.glade.h:2 sabayon/livecd.py:482
msgid "_Back"
msgstr "_عقب"

#: image.py:290
#, fuzzy, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""
"هنگام پیاده کردن سی‌دی خطایی رخ داد. لطفاً مطمئن شوید که شما از طریق پوسته روی "
"tty2 به %s دست‌یابی ندارید و سپس برای تلاش مجدد روی دکمهٔ «تأیید» کلیک کنید."

#: installclass.py:71
msgid "Install on System"
msgstr "نصب روی سیستم"

#: installinterfacebase.py:42
#, python-format
msgid ""
"Disk %s contains BIOS RAID metadata, but is not part of any recognized BIOS "
"RAID sets. Ignoring disk %s."
msgid_plural ""
"Disks %s contain BIOS RAID metadata, but are not part of any recognized BIOS "
"RAID sets. Ignoring disks %s."
msgstr[0] ""
msgstr[1] ""

#: iutil.py:816
#, python-format
msgid ""
"Error: On open, cannot set reIPL method to %(reipl_type)s (%(filename)s: "
"%(e)s)"
msgstr ""

#: iutil.py:827
#, python-format
msgid ""
"Error: On write, cannot set reIPL method to %(reipl_type)s (%(filename)s: "
"%(e)s)"
msgstr ""

#: iutil.py:836
#, python-format
msgid ""
"Error: On close, cannot set reIPL method to %(reipl_type)s (%(filename)s: "
"%(e)s)"
msgstr ""

#: iutil.py:855
#, python-format
msgid "Error: Could not set %(device)s as reIPL device (%(e)s)"
msgstr ""

#: iutil.py:865
#, python-format
msgid "Error: Could not reset loadparm (%s)"
msgstr ""

#: iutil.py:874
#, python-format
msgid "Warning: Could not reset parm (%s)"
msgstr ""

#: iutil.py:887
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from DASD device %s to continue "
"installation"
msgstr ""

#: iutil.py:909
#, python-format
msgid "Error: reading FCP property %(syspath_property)s for reIPL (%(e)s)"
msgstr ""

#: iutil.py:924
#, python-format
msgid "Error: writing FCP property %(reipl_property)s for reIPL (%(e)s)"
msgstr ""

#: iutil.py:939
#, python-format
msgid ""
"Error: writing default FCP property %(reipl_property)s for reIPL (%(e)s)"
msgstr ""

#: iutil.py:954
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from FCP %(device)s with WWPN "
"%(wwpn)s and LUN %(lun)s to continue installation"
msgstr ""

#: iutil.py:971
msgid ""
"After shutdown, please perform a manual IPL from the device now containing /"
"boot to continue installation"
msgstr ""

#: iutil.py:982
msgid "Error determining boot device's disk name"
msgstr ""

#: iutil.py:986
msgid "The mount point /boot or / is on a disk that we are not familiar with"
msgstr ""

#: kickstart.py:115
#, fuzzy, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  You may "
"examine the output in %(msgs)s.  This is a fatal error and installation will "
"be aborted.  Press the OK button to exit the installer."
msgstr ""
"در اجرای کدنوشتک خطایی پیش آمد. می‌توانید خروجی را در %s بررسی کنید. این خطا "
"مهلک است و نصب شما قطع خواهد شد.\n"
"\n"
"برای راه‌اندازی مجدد سیستم روی دکمهٔ «تأیید» کلیک کنید."

#: kickstart.py:123 kickstart.py:125
msgid "Scriptlet Failure"
msgstr "خرابی کدنوشتک"

#: kickstart.py:150 rescue.py:248 yuminstall.py:698 yuminstall.py:1164
#: iw/task_gui.py:308
#, fuzzy
msgid "No Network Available"
msgstr "دستگاه‌های شبکه‌ای"

#: kickstart.py:151
msgid ""
"Encryption key escrow requires networking, but there was an error enabling "
"the network on your system."
msgstr ""

#: kickstart.py:164
#, fuzzy, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""
"خطای زیر هنگام تجزیهٔ پیکربندی kickstart پیدا شد:\n"
"\n"
"‏%s"

#: kickstart.py:1233
#, fuzzy, python-format
msgid "Error processing %%ksappend lines: %s"
msgstr "خطا در باز کردن پروندهٔ راه‌اندازی %s:‏ %s"

#: kickstart.py:1236
#, python-format
msgid "Unknown error processing %%ksappend lines: %s"
msgstr ""

#: kickstart.py:1291 livecd.py:229
#, fuzzy
msgid "Post-Installation"
msgstr "پس از نصب"

#: kickstart.py:1292
#, fuzzy
msgid "Running post-installation scripts"
msgstr "نصب %s %s"

#: kickstart.py:1308
#, fuzzy
msgid "Pre-Installation"
msgstr "نصب بسته"

#: kickstart.py:1309
#, fuzzy
msgid "Running pre-installation scripts"
msgstr "نصب %s %s"

#: kickstart.py:1341
msgid "Missing Package"
msgstr "بستهٔ ناموجود"

#: kickstart.py:1342
#, fuzzy, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort this installation?"
msgstr ""
"شما مشخص کرده‌اید که بستهٔ «%s» نصب شود. این بسته وجود ندارد. مایلید ادامه "
"دهید یا نصب خود را قطع می‌کنید؟"

#: kickstart.py:1348 kickstart.py:1387
msgid "_Abort"
msgstr "_قطع شود"

#: kickstart.py:1349 kickstart.py:1388
#, fuzzy
msgid "_Ignore All"
msgstr "نادیده گرفته شود"

#: kickstart.py:1379
msgid "Missing Group"
msgstr "گروه ناموجود"

#: kickstart.py:1380
#, fuzzy, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort this installation?"
msgstr ""
"شما مشخص کرده‌اید که بستهٔ «%s» نصب شود. این بسته وجود ندارد. مایلید ادامه "
"دهید یا نصب خود را قطع می‌کنید؟"

#: kickstart.py:1495
#, python-format
msgid ""
"The kickstart configuration file is missing required information that "
"anaconda cannot prompt for.  Please add the following sections and try "
"again:\n"
"%s"
msgstr ""

#: livecd.py:121 sabayon/livecd.py:64
#, fuzzy
msgid "Unable to find image"
msgstr "تصویر نصب %s پیدا نشد"

#: livecd.py:122
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""

#: livecd.py:180
#, fuzzy
msgid "Copying live image to hard drive."
msgstr "در حال انتقال تصویر نصب به دیسک‌سخت..."

#: livecd.py:199
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""

#: livecd.py:230
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes."
msgstr ""

#: livecd.py:401 sabayon/livecd.py:477
#, python-format
msgid ""
"The root filesystem you created is not large enough for this live image "
"(%.2f MB required)."
msgstr ""

#: network.py:56
#, fuzzy
msgid "Hostname must be 255 or fewer characters in length."
msgstr "نام‌میزبان باید ۶۴ نویسه یا کوتاه‌تر باشد."

#: network.py:62
#, fuzzy
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or "
"'0-9'"
msgstr "نام‌میزبان باید با نویسه‌ایی معتبر در محدودهٔ «a-z» یا «A-Z» آغاز شود"

#: network.py:67
#, fuzzy
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr "نام‌میزبان فقط می‌تواند شامل نویسه‌های «a‐z»، «A‐Z»، '-' یا «.» باشد"

#: network.py:180
#, fuzzy
msgid "IP address is missing."
msgstr "نشانی IP وجود ندارد"

#: network.py:184
#, fuzzy
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr "نشانی‌های IP باید حاوی اعداد بین ۰ و ۲۵۵ باشند"

#: network.py:187
#, fuzzy, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr "%s نام میزبان معتبری نیست."

#: network.py:189
#, fuzzy, python-format
msgid "'%s' is an invalid IP address."
msgstr "%s نام میزبان معتبری نیست."

#: packages.py:111
#, fuzzy
msgid "Resizing Failed"
msgstr "در حال بازیابی"

#: packages.py:112
#, fuzzy, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: packages.py:120
#, fuzzy
msgid "Migration Failed"
msgstr "سوار کردن شکست خورد"

#: packages.py:121
#, fuzzy, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: packages.py:130
msgid "_File Bug"
msgstr ""

#: packages.py:312 packages.py:333
msgid "Warning! This is pre-release software!"
msgstr "اخطار! این نرم‌افزار نسخهٔ آزمایشی است!"

#: packages.py:313
#, fuzzy, python-format
msgid ""
"Thank you for downloading this pre-release of %(productName)s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %(bugzillaUrl)s\n"
"\n"
"and file a report against '%(fileagainst)s'.\n"
msgstr ""
"از بارگیری این نسخهٔ آزمایشی %s متشکریم.\n"
"\n"
"این نسخه، نسخهٔ نهایی نیست و برای استفاده در سیستم‌های کاری منظور نشده است. "
"هدف از این انتشار جمع‌آوری بازخورد از آزمایش‌کنندگان است، و برای استفادهٔ "
"روزمره مناسب نیست.\n"
"\n"
"برای گزارش بازخورد، لطفاً به:\n"
"\n"
"‏‎   %s\n"
"\n"
"مراجعه کنید و گزارشی برای «%s» ثبت کنید.\n"

#: packages.py:328
msgid "_Install anyway"
msgstr "_با این وجود نصب شود"

#: packages.py:331
msgid "Your system will now be rebooted..."
msgstr "حالا سیستم شما مجدداً راه‌اندازی می‌شود..."

#: partIntfHelpers.py:41
msgid "Please enter a volume group name."
msgstr "لطفاً نام گروه جلد را وارد کنید."

#: partIntfHelpers.py:45
msgid "Volume Group Names must be less than 128 characters"
msgstr "نام گروه جلد باید کمتر از ۱۲۸ نویسه داشته باشد"

#: partIntfHelpers.py:48
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr "خطا ‐ نام گروه جلد %s معتبر نیست."

#: partIntfHelpers.py:53
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"خطا ‐ نام گروه جلد دارای نویسه‌های غیرمجاز یا فاصله است. نویسه‌های مورد قبول "
"حروف لاتینی، ارقام، «.» یا «_» هستند."

#: partIntfHelpers.py:63
msgid "Please enter a logical volume name."
msgstr "لطفاً نام جلد منطقی را وارد کنید."

#: partIntfHelpers.py:67
msgid "Logical Volume Names must be less than 128 characters"
msgstr "نام جلدهای منطقی باید کمتر از ۱۲۸ نویسه داشته باشند"

#: partIntfHelpers.py:71
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr "خطا ‐ نام جلد منطقی %s معتبر نیست."

#: partIntfHelpers.py:77
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""
"خطا ‐ نام جلد منطقی دارای نویسه‌های غیرمجاز یا فاصله است. نویسه‌های مورد قبول "
"حروف لاتینی، اعداد، «.» یا «_» هستند."

#: partIntfHelpers.py:101
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""

#: partIntfHelpers.py:108
msgid "Please specify a mount point for this partition."
msgstr "لطفاً یک نقطهٔ سوار کردن برای این افراز مشخص کنید."

#: partIntfHelpers.py:121 partIntfHelpers.py:128
msgid "Unable To Delete"
msgstr "حذف ممکن نیست"

#: partIntfHelpers.py:122
msgid "You must first select a partition to delete."
msgstr "باید اول افرازی را برای حذف انتخاب کنید."

#: partIntfHelpers.py:153 partIntfHelpers.py:341 iw/lvm_dialog_gui.py:862
msgid "Confirm Delete"
msgstr "تصدیق حذف"

#: partIntfHelpers.py:154
#, fuzzy, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr "شما در آستانهٔ حذف همهٔ افرازهای روی دستگاه «‎/dev/%s» هستید."

#: partIntfHelpers.py:157 partIntfHelpers.py:342 iw/lvm_dialog_gui.py:865
#: iw/lvm_dialog_gui.py:1434 iw/osbootwidget.py:104 iw/partition_gui.py:1816
#: iw/partition_gui.py:1828
msgid "_Delete"
msgstr "_حذف"

#: partIntfHelpers.py:204
msgid "Notice"
msgstr "توجه"

#: partIntfHelpers.py:205
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""
"افرازهای زیر حذف نشدند چون از آنها استفاده می‌شود:\n"
"\n"
"‏%s"

#: partIntfHelpers.py:220
msgid "Format as Swap?"
msgstr "به شکل حافظهٔ مبادله قالب‌بندی شود؟"

#: partIntfHelpers.py:221
#, fuzzy, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""
"‏‎/dev/%s دارای نوع افراز 0x82 (مبادله لینوکس) است ولی به‌نظر نمی‌رسد که به "
"عنوان افراز حافظهٔ مبادلهٔ لینوکس قالب‌بندی شده باشد.\n"
"\n"
"آیا می‌خواهید این افراز را به عنوان افراز حافظهٔ مبادله قالب‌بندی کنید؟"

#: partIntfHelpers.py:236
#, fuzzy, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr "باید حداقل یک دیسک‌سخت انتخاب کنید تا %s روی آن نصب شود."

#: partIntfHelpers.py:241
#, fuzzy
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""
"شما استفاده از یک افراز موجود را بدون قالب‌بندی کردن آن برای نصب انتخاب "
"کرده‌اید. توصیه می‌کنیم برای اطمینان از این که پرونده‌های نصب قبلی سیستم عامل "
"باعث اشکالاتی برای این نصب لینوکس نگردند، این افراز را قالب‌بندی کنید. با این "
"وجود، اگر این افراز شامل پرونده‌هایی است که می‌خواهید نگه دارید، مثل شاخه‌های "
"آغازه، در این صورت می‌توانیدکار را بدون قالب‌بندی این افراز ادامه دهید."

#: partIntfHelpers.py:249
msgid "Format?"
msgstr "قالب‌بندی شود؟"

#: partIntfHelpers.py:249 iw/partition_gui.py:1601
msgid "_Modify Partition"
msgstr "_تغییر افراز"

#: partIntfHelpers.py:249
msgid "Do _Not Format"
msgstr "قالب‌بندی _نشود"

#: partIntfHelpers.py:257
msgid "Error with Partitioning"
msgstr "خطا در افرازبندی"

#: partIntfHelpers.py:258
#, fuzzy, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of "
"%(productName)s.\n"
"\n"
"%(errorstr)s"
msgstr ""
"خطاهای بحرانی زیر در شِمای افرازبندی درخواست شدهٔ شما وجود دارند.\n"
"پیش از ادامه به نصب %s این خطاها باید تصحیح شوند.\n"
"\n"
"‏%s"

#: partIntfHelpers.py:274
msgid "Partitioning Warning"
msgstr "اخطار افرازبندی"

#: partIntfHelpers.py:275
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""
"اخطارهای زیر در شِمای افرازبندی درخواست شدهٔ شما وجود دارند.\n"
"\n"
"‏%s\n"
"\n"
"آیا می‌خواهید با شِمای افرازبندی درخواستی‌تان ادامه دهید؟"

#: partIntfHelpers.py:289
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""
"افرازهای از پیش موجود زیر برای قالب‌بندی انتخاب شده‌اند، همهٔ داده‌ها از بین "
"خواهد رفت."

#: partIntfHelpers.py:292
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""
"برای ادامه دادن و قالب‌بندی این افرازها روی دکمهٔ «بله»، و برای عقب رفتن و "
"تغییر این تنظیمات روی دکمهٔ «خیر» کلیک کنید."

#: partIntfHelpers.py:298
msgid "Format Warning"
msgstr "اخطار قالب‌بندی"

#: partIntfHelpers.py:325
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""
"شما در آستانهٔ حذف گروه جلدی «%s» هستید.\n"
"\n"
"تمام جلد‌های منطقی در این گروه جلدی از دست خواهند رفت!"

#: partIntfHelpers.py:329
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr "شما در آستانهٔ حذف جلد منطقی «%s» هستید."

#: partIntfHelpers.py:332
msgid "You are about to delete a RAID device."
msgstr "شما در آستانهٔ حذف دستگاه RAID هستید."

#: partIntfHelpers.py:334
#, fuzzy, python-format
msgid "You are about to delete the %s partition."
msgstr "شما در آستانهٔ حذف افراز ‎/dev/%s هستید."

#: partIntfHelpers.py:338
#, fuzzy, python-format
msgid "You are about to delete the %(type)s %(name)s"
msgstr "شما در آستانهٔ حذف افراز ‎/dev/%s هستید."

#: partIntfHelpers.py:349
msgid "Confirm Reset"
msgstr "تصدیق برگرداندن به حالت پیش‌فرض"

#: partIntfHelpers.py:350
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr "آیا مطمئنید که می‌خواهید جدول افرازها را به وضعیت اولیهٔ خود بازگردانید؟"

#: platform.py:98 platform.py:344 platform.py:412 platform.py:520
#: ui/create-storage.glade.h:17
msgid "RAID Device"
msgstr "دستگاه RAID"

#: platform.py:99 platform.py:102 platform.py:345 platform.py:413
#: platform.py:521 platform.py:524
msgid "Master Boot Record (MBR)"
msgstr "رکورد راه‌اندازی اصلی (MBR)"

#: platform.py:101 platform.py:523
msgid "First sector of boot partition"
msgstr "اولین بخش افراز راه‌انداز"

#: platform.py:113
#, fuzzy
msgid "You have not created a bootable partition."
msgstr "شما باید یک افراز راه‌اندازی PPC PReP ایجاد کنید."

#: platform.py:118
#, fuzzy
msgid "Bootable partitions cannot be on a RAID device."
msgstr "افرازهای قابل راه‌اندازی فقط می‌توانند روی دستگاه‌های RAID1 باشند."

#: platform.py:120
msgid "Bootable partitions can only be on RAID1 devices."
msgstr "افرازهای قابل راه‌اندازی فقط می‌توانند روی دستگاه‌های RAID1 باشند."

#: platform.py:124
msgid "Bootable RAID1 set members must be partitions."
msgstr ""

#: platform.py:129
msgid "Bootable partitions cannot be on a logical volume."
msgstr "افرازهای قابل راه‌اندازی نمی‌توانند روی یک جلد منطقی باشند."

#: platform.py:136
#, fuzzy, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr "افرازهای قابل راه‌اندازی نمی‌توانند روی یک جلد منطقی باشند."

#: platform.py:140 platform.py:145
#, fuzzy
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr "افرازهای قابل راه‌اندازی نمی‌توانند روی یک جلد منطقی باشند."

#: platform.py:217
#, fuzzy
msgid "EFI System Partition"
msgstr "ویرایش افراز"

#: platform.py:229
#, fuzzy
msgid "You have not created a /boot/efi partition."
msgstr "شما باید یک افراز راه‌اندازی PPC PReP ایجاد کنید."

#: platform.py:235
msgid "/boot/efi is not EFI."
msgstr ""

#: platform.py:251
#, python-format
msgid "%s must have a %s disk label."
msgstr ""

#: platform.py:289
#, python-format
msgid "%s must have a bsd disk label."
msgstr ""

#: platform.py:301
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr ""

#: platform.py:347
msgid "PPC PReP Boot"
msgstr "راه‌اندازی PPC PReP"

#: platform.py:363
msgid "The boot partition must be within the first 4MB of the disk."
msgstr ""

#: platform.py:415 platform.py:418
msgid "Apple Bootstrap"
msgstr "راه‌اندازی Apple"

#: platform.py:432
#, python-format
msgid "%s must have a mac disk label."
msgstr ""

#: rescue.py:207
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""
"پس از پایان کارتان لطفاً از پوسته خارج شوید تا سیستم‌تان مجدداً راه‌اندازی شود."

#: rescue.py:221
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""

#: rescue.py:240
msgid "Setup Networking"
msgstr "برپاسازی شبکه"

#: rescue.py:241
msgid "Do you want to start the network interfaces on this system?"
msgstr "آیا می‌خواهید واسط‌های شبکه را روی این سیستم آغاز کنید؟"

#: rescue.py:242 loader/driverdisk.c:666 loader/driverdisk.c:676
#: loader/hdinstall.c:206 textw/constants_text.py:56 sabayon/utils.py:928
#: sabayon/utils.py:1172
msgid "Yes"
msgstr "بله"

#: rescue.py:242 rescue.py:244 loader/driverdisk.c:666 loader/driverdisk.c:676
#: textw/constants_text.py:60 sabayon/utils.py:928 sabayon/utils.py:1172
msgid "No"
msgstr "خیر"

#: rescue.py:249
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""

#: rescue.py:284 rescue.py:353 rescue.py:364 rescue.py:449
msgid "Rescue"
msgstr "نجات"

#: rescue.py:285
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""
"محیط نجات اکنون تلاش می‌کند نصب لینوکس شما را پیدا کند و ‌آن را در شاخهٔ %s "
"سوار کند. پس از آن می‌توانید تغییرات لازم را در سیستم خود ایجاد کنید. اگر "
"می‌خواهید این کار را انجام دهید روی دکمهٔ «ادامه» کلیک کنید. همچنین می‌توانید  "
"با کلیک کردن روی دکمهٔ «فقط‌خواندنی»، سیستم‌های پرونده‌ای‌تان را به جای "
"خواندنی‌نوشتنی، فقط‌خواندنی سوار کنید.\n"
"\n"
"اگر به دلیلی این روند شکست خورد می‌توانید روی دکمهٔ «پرش» کلیک کنید و این "
"مرحله نادیده گرفته خواهد شد و شما مستقیماً به پوستهٔ سطر فرمان خواهید رفت.\n"
"\n"

#: rescue.py:295 iw/partition_gui.py:781 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/driverdisk.c:627 storage/__init__.py:157
#: storage/devicetree.py:90
msgid "Continue"
msgstr "ادامه"

#: rescue.py:295 rescue.py:300
msgid "Read-Only"
msgstr "فقط‌خواندنی"

#: rescue.py:295 rescue.py:297 loader/cdinstall.c:248 loader/cdinstall.c:251
#: loader/method.c:324 storage/__init__.py:1784 textw/upgrade_text.py:139
msgid "Skip"
msgstr "پرش"

#: rescue.py:329
msgid "System to Rescue"
msgstr "نجات سیستم"

#: rescue.py:330
#, fuzzy
msgid "Which device holds the root partition of your installation?"
msgstr "افراز ریشهٔ نصب شما در کدام افراز قرار دارد؟"

#: rescue.py:332 rescue.py:336 text.py:627 text.py:629
#: textw/complete_text.py:31
msgid "Exit"
msgstr "خروج"

#: rescue.py:354
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions.  "
"The system will reboot automatically when you exit from the shell."
msgstr ""
"سیستم شما سیستم‌های پرونده‌ای کثیفی داشت که شما سوار کردن‌شان را انتخاب نکردید. "
"برای دسترسی به پوسته‌ای که بتوانید از آن fsck کنید و افرازهایتان را سوار کنید "
"کلید ورود را فشار دهید. هنگامی که از پوسته خارج شوید سیستم به طور خودکار "
"مجدداً راه اندازی خواهد شد."

#: rescue.py:365
#, fuzzy, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"The system will reboot automatically when you exit from the shell."
msgstr ""
"سیستم شما زیر %s سوار شده است.\n"
"\n"
"برای دسترسی به پوسته <ورود> را فشار دهید. اگر می‌خواهید سیستم تبدیل به محیط "
"rootشود، فرمان زیر را اجرا کنید:\n"
" \n"
"‏\tchroot %s\n"
"\n"
"هنگامی که از پوسته خارج شوید سیستم به طور خودکار مجدداً راه اندازی خواهد شد."

#: rescue.py:450
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. The system will reboot automatically when you "
"exit from the shell."
msgstr ""
"هنگام تلاش برای سوار کردن بعضی یا همهٔ سیستم شما خطایی رخ داد. ممکن است "
"مقداری از آن زیر %s سوار شده باشد.\n"
"\n"
"برای دسترسی به پوسته <ورود> را فشار دهید. هنگامی که از پوسته خارج شوید سیستم "
"به طور خودکار مجدداً راه اندازی خواهد شد."

#: rescue.py:459
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr ""

#: rescue.py:462
msgid "Rescue Mode"
msgstr "حالت نجات"

#: rescue.py:463
msgid ""
"You don't have any Linux partitions. Press return to get a shell. The system "
"will reboot automatically when you exit from the shell."
msgstr ""
"شما هیچ افراز لینوکسی‌ای ندارید. برای دسترسی به پوسته کلید ورود را فشار دهید. "
"هنگامی که از پوسته خارج شوید سیستم به طور خودکار مجدداً راه اندازی خواهد شد."

#: rescue.py:476
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr "سیستم شما در شاخهٔ %s سوار شد."

#: text.py:155
msgid "Passphrase for encrypted device"
msgstr ""

#: text.py:168
msgid "Also add this passphrase to all existing encrypted devices"
msgstr ""

#: text.py:199
#, fuzzy, python-format
msgid "The passphrase must be at least %d character long."
msgid_plural "The passphrase must be at least %d characters long."
msgstr[0] "طول گذرواژه باید حداقل شش نویسه باشد."
msgstr[1] "طول گذرواژه باید حداقل شش نویسه باشد."

#: text.py:233 ui/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr ""

#: text.py:241 ui/lukspassphrase.glade.h:7
msgid "This is a global passphrase"
msgstr ""

#: text.py:356 text.py:360
#, fuzzy
msgid "Repository editing is not available in text mode."
msgstr "نصب گرافیکی ممکن نیست...  حالت متنی آغاز می‌شود."

#: text.py:422
#, fuzzy, python-format
msgid "Welcome to %(productName)s for %(productArch)s"
msgstr "به %s خوش آمدید"

#: text.py:424
#, python-format
msgid "Welcome to %s"
msgstr "به %s خوش آمدید"

#: text.py:426
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr "  <Tab>/<Alt-Tab> بین اجزا   |  <Space> انتخاب   |  <F12> صفحهٔ بعد"

#: text.py:627 loader/net.c:109 loader/net.c:470 loader/net.c:525
#: loader/net.c:667 loader/net.c:675 loader/net.c:1096 loader/net.c:1104
msgid "Retry"
msgstr "دوباره سعی شود"

#: text.py:653
msgid "Cancelled"
msgstr "لغو شد"

#: text.py:654
msgid "I can't go to the previous step from here. You will have to try again."
msgstr "نمی‌توان از اینجا به مرحلهٔ قبلی رفت. مجبورید دوباره امتحان کنید."

#: upgrade.py:49
msgid "Proceed with upgrade?"
msgstr "با ارتقاء ادامه داده شود؟"

#: upgrade.py:50
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""
"سیستم‌های پرونده‌ای نصب لینوکسی که برای ارتقاء انتخاب کرده‌اید از قبل سوار "
"شده‌اند. بعد از این مرحله دیگر نمی‌توانید به عقب برگردید.\n"
"\n"

#: upgrade.py:54
msgid "Would you like to continue with the upgrade?"
msgstr "آیا مایلید به ارتقاء دادن ادامه دهید؟"

#: upgrade.py:215
msgid "Mount failed"
msgstr "سوار کردن شکست خورد"

#: upgrade.py:216
#, fuzzy, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""
"یک یا چندین سیستم پرونده‌ای فهرست شده در ‎/etc/fstab در سیستم لینوکس شما را "
"نمی‌توان سوار کرد. لطفاً این اشکال را رفع کنید و دوباره برای ارتقاء تلاش کنید."

#: upgrade.py:224 upgrade.py:231
#, fuzzy
msgid "Upgrade root not found"
msgstr "ارتقائی که آغاز شود"

#: upgrade.py:225
msgid "The root for the previously installed system was not found."
msgstr ""

#: upgrade.py:232
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""

#: upgrade.py:255
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""
"پرونده‌های زیر پیوندهای نمادی مطلق هستند که در حین ارتقاء پشتیبانی نمی‌شوند. "
"لطفاً آنها را به پیوندهای نمادی نسبی تغییر دهید و ارتقاء را از نو آغاز کنید.\n"
"\n"

#: upgrade.py:261
msgid "Absolute Symlinks"
msgstr "پیوندهای نمادی مطلق"

#: upgrade.py:272
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""
"آنچه در زیر آمده شاخه‌هایی هستند که باید پیوند نمادی می‌بودند، و باعث مشکلاتی "
"در ارتقاء دادن خواهند شد. لطفاً آنها را به وضعیت اصلی خود یعنی پیوند نمادی "
"برگردانید و ارتقاء را از نو آغاز کنید.\n"
"\n"

#: upgrade.py:278
msgid "Invalid Directories"
msgstr "شاخه‌های نامعتبر"

#: vnc.py:137
#, fuzzy, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr "نصب %s %s در میزبان %s"

#: vnc.py:143
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr ""

#: vnc.py:172
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr "در حال تلاش برای وصل شدن به کارگیر vnc در میزبان %s..."

#: vnc.py:186
msgid "Connected!"
msgstr "وصل شد!"

#: vnc.py:189
msgid "Will try to connect again in 15 seconds..."
msgstr "۱۵ ثانیهٔ دیگر باز برای اتصال تلاش خواهد شد..."

#: vnc.py:195
#, fuzzy, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] "بعد از ۵۰ بار تلاش برای وصل شدن از این کار چشم پوشی شد!\n"
msgstr[1] "بعد از ۵۰ بار تلاش برای وصل شدن از این کار چشم پوشی شد!\n"

#: vnc.py:206
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr "برای شروع نصب، لطفاً کارگیر vnc خود را دستی به %s متصل کنید."

#: vnc.py:208
msgid "Please manually connect your vnc client to begin the install."
msgstr "برای شروع نصب، لطفاً کارگیر vnc خود را دستی متصل کنید."

#: vnc.py:211
msgid "Starting VNC..."
msgstr "در حال آغاز VNC..."

#: vnc.py:236
msgid "The VNC server is now running."
msgstr "کارگزار VNC در حال اجرا است."

#: vnc.py:249
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: vnc.py:254
#, fuzzy
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""
"\n"
"\n"
"اخطار!!! کارگزار VNC دارد بدون گذرواژه اجرا می‌شود!\n"
"اگر مایل به امن کردن کارگزار هستید، می‌توانید از گزینهٔ راه‌اندازی\n"
"‏‪vncpassword=<گذرواژه>‬ استفاده کنید.\n"

#: vnc.py:258
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""

#: vnc.py:260
#, fuzzy
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr "خطای نامعلوم"

#: vnc.py:282 vnc.py:375
msgid "VNC Configuration"
msgstr "پیکربندی VNC"

#: vnc.py:285 vnc.py:379
msgid "No password"
msgstr "بدون گذرواژه"

#: vnc.py:287 vnc.py:382
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""
"گذرواژه مانع از آن خواهد شد که شنوندگان تأیید هویت نشده متصل شوند و پیشرفت "
"نصب شما را ببینند. لطفاً گذرواژه‌ای برای استفاده در نصب وارد کنید."

#: vnc.py:295 vnc.py:390 textw/userauth_text.py:47 textw/userauth_text.py:134
msgid "Password:"
msgstr "گذرواژه:"

#: vnc.py:296 vnc.py:391 textw/userauth_text.py:49 textw/userauth_text.py:136
msgid "Password (confirm):"
msgstr "گذرواژه (تکرار):"

#: vnc.py:314 vnc.py:413 textw/userauth_text.py:70 textw/userauth_text.py:162
msgid "Password Mismatch"
msgstr "گذرواژه مطابقت نمی‌کند"

#: vnc.py:315 vnc.py:414 textw/userauth_text.py:71 textw/userauth_text.py:163
msgid "The passwords you entered were different. Please try again."
msgstr "گذرواژه‌های که وارد کردید متفاوت بودند. لطفاً دوباره تلاش کنید."

#: vnc.py:320 vnc.py:419 textw/userauth_text.py:66 textw/userauth_text.py:158
msgid "Password Length"
msgstr "طول گذرواژه"

#: vnc.py:321 vnc.py:420
msgid "The password must be at least six characters long."
msgstr "طول گذرواژه باید حداقل شش نویسه باشد."

#: vnc.py:343
msgid "Unable to Start X"
msgstr "آغاز X ممکن نیست"

#: vnc.py:345
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""
"آغاز ‏X روی رایانهٔ شما ممکن نبود. می‌خواهید VNC را آغاز کنید تا این رایانه را "
"به رایانهٔ دیگری متصل کنید و نصب گرافیکی انجام دهید، یا این که نصب را در حالت "
"متنی ادامه می‌دهید؟"

#: vnc.py:364
msgid "Start VNC"
msgstr "آغاز VNC"

#: vnc.py:365 vnc.py:367
msgid "Use text mode"
msgstr "استفاده از حالت متنی"

#: yuminstall.py:85
#, python-format
msgid "%s MB"
msgstr "‏%s مگابایت"

#: yuminstall.py:88
#, python-format
msgid "%s KB"
msgstr "‏%s کیلوبایت"

#: yuminstall.py:90
#, fuzzy, python-format
msgid "%s Byte"
msgid_plural "%s Bytes"
msgstr[0] "‏%s بایت"
msgstr[1] "‏%s بایت"

#: yuminstall.py:134
msgid "Preparing to install"
msgstr ""

#: yuminstall.py:135
msgid "Preparing transaction from installation source"
msgstr ""

#: yuminstall.py:163
#, fuzzy, python-format
msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
msgstr "_نصب %s"

#: yuminstall.py:212
#, python-format
msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgstr[0] ""
msgstr[1] ""

#: yuminstall.py:227
msgid "Finishing upgrade"
msgstr ""

#: yuminstall.py:228
msgid "Finishing upgrade process.  This may take a little while."
msgstr ""

#: yuminstall.py:251
#, fuzzy
msgid "Error Installing Package"
msgstr "در حال نصب بسته‌ها"

#: yuminstall.py:252
#, python-format
msgid ""
"A fatal error occurred when installing the %s package.  This could indicate "
"errors when reading the installation media.  Installation cannot continue."
msgstr ""

#: yuminstall.py:353 iw/task_gui.py:321
#, fuzzy
msgid "Error Setting Up Repository"
msgstr "برچسب راه‌اندازی نامعتبر"

#: yuminstall.py:354
#, fuzzy, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Please provide the correct information for installing %(productName)s."
msgstr ""
"خطاهای زیر هنگام افرازبندی دیسک شما رخ داد:\n"
"\n"
"‏%s\n"
"\n"
"برای راه‌اندازی مجدد سیستم خود روی دکمهٔ «تأیید» کلیک کنید."

#: yuminstall.py:398
#, fuzzy
msgid "Change Disc"
msgstr "تعویض سی‌دی"

#: yuminstall.py:399
#, fuzzy, python-format
msgid "Please insert %(productName)s disc %(discnum)d to continue."
msgstr "برای ادامه لطفاً دیسک %s شمارهٔ %d را وارد کنید."

#: yuminstall.py:409
#, fuzzy
msgid "Wrong Disc"
msgstr "سی‌دی اشتباه است"

#: yuminstall.py:410
#, fuzzy, python-format
msgid "That's not the correct %s disc."
msgstr "این سی‌دی درست %s نیست."

#: yuminstall.py:417
#, fuzzy
msgid "Unable to access the disc."
msgstr "دست‌یابی به سی‌دی ممکن نیست."

#: yuminstall.py:575
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr ""

#: yuminstall.py:699 yuminstall.py:1165 iw/task_gui.py:309
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""

#: yuminstall.py:800 yuminstall.py:802
msgid "Re_boot"
msgstr "_راه‌اندازی مجدد"

#: yuminstall.py:800
#, fuzzy
msgid "_Eject"
msgstr "بیرون دادن سی‌دی"

#: yuminstall.py:806
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: yuminstall.py:854
#, fuzzy
msgid "Retrying"
msgstr "در حال بازیابی"

#: yuminstall.py:854
#, fuzzy
msgid "Retrying download."
msgstr "در حال بازیابی"

#: yuminstall.py:919
#, fuzzy, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr ""
"نام میزبان «%s» به دلایل زیر معتبر نیست:\n"
"\n"
"‏%s"

#: yuminstall.py:962 yuminstall.py:963
msgid "file conflicts"
msgstr ""

#: yuminstall.py:964
msgid "older package(s)"
msgstr ""

#: yuminstall.py:965
msgid "insufficient disk space"
msgstr ""

#: yuminstall.py:966
msgid "insufficient disk inodes"
msgstr ""

#: yuminstall.py:967
msgid "package conflicts"
msgstr ""

#: yuminstall.py:968
msgid "package already installed"
msgstr ""

#: yuminstall.py:969
msgid "required package"
msgstr ""

#: yuminstall.py:970
msgid "package for incorrect arch"
msgstr ""

#: yuminstall.py:971
msgid "package for incorrect os"
msgstr ""

#: yuminstall.py:985
msgid "You need more space on the following file systems:\n"
msgstr ""

#: yuminstall.py:998
#, fuzzy, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr "چه نوع رسانه‌ای حاوی بسته‌هایی است که باید نصب شوند؟"

#: yuminstall.py:1001
#, fuzzy, python-format
msgid ""
"There was an error running your transaction for the following reason(s): "
"%s.\n"
msgstr ""
"نام میزبان «%s» به دلایل زیر معتبر نیست:\n"
"\n"
"‏%s"

#: yuminstall.py:1008 yuminstall.py:1013
msgid "Error Running Transaction"
msgstr ""

#: yuminstall.py:1191
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""

#: yuminstall.py:1227
#, fuzzy
msgid "Retrieving installation information."
msgstr "در حال فرستادن درخواست برای اطلاعات IP‏ برای %s..."

#: yuminstall.py:1229
#, fuzzy, python-format
msgid "Retrieving installation information for %s."
msgstr "در حال فرستادن درخواست برای اطلاعات IP‏ برای %s..."

#: yuminstall.py:1231
msgid "Installation Progress"
msgstr ""

#: yuminstall.py:1239 textw/constants_text.py:64
msgid "Edit"
msgstr "ویرایش"

#: yuminstall.py:1250 yuminstall.py:1441
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""

#: yuminstall.py:1419
msgid ""
"Some of the packages you have selected for install are missing "
"dependencies.  You can exit the installation, go back and change your "
"package selections, or continue installing these packages without their "
"dependencies.  If you continue, these packages may not work correctly due to "
"missing components."
msgstr ""

#: yuminstall.py:1463
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""

#: yuminstall.py:1484
msgid "Reboot?"
msgstr "راه‌اندازی مجدد؟"

#: yuminstall.py:1485
msgid "The system will be rebooted now."
msgstr "سیستم اکنون مجدداً راه‌اندازی خواهد شد."

#: yuminstall.py:1623
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""
"به نظر می‌رسد سیستم شما قدیمی‌تر از آن است که بتوان آن را به این نسخهٔ %s "
"ارتقاء داد. مطمئنید که می‌خواهید روند ارتقاء را ادامه دهید؟"

#: yuminstall.py:1658
#, fuzzy, python-format
msgid ""
"The arch of the release of %(productName)s you are upgrading to appears to "
"be %(myarch)s which does not match your previously installed arch of "
"%(arch)s.  This is likely to not succeed.  Are you sure you wish to continue "
"the upgrade process?"
msgstr ""
"به نظر می‌رسد سیستم شما قدیمی‌تر از آن است که بتوان آن را به این نسخهٔ %s "
"ارتقاء داد. مطمئنید که می‌خواهید روند ارتقاء را ادامه دهید؟"

#: yuminstall.py:1705
msgid "Post Upgrade"
msgstr ""

#: yuminstall.py:1706
#, fuzzy
msgid "Performing post-upgrade configuration"
msgstr "در حال انجام پیکربندی پس از نصب..."

#: yuminstall.py:1708
#, fuzzy
msgid "Post Installation"
msgstr "پس از نصب"

#: yuminstall.py:1709
#, fuzzy
msgid "Performing post-installation configuration"
msgstr "در حال انجام پیکربندی پس از نصب..."

#: yuminstall.py:1932
#, fuzzy
msgid "Installation Starting"
msgstr "نصب آغاز می‌شود"

#: yuminstall.py:1933
#, fuzzy
msgid "Starting installation process"
msgstr "نصب گرافیکی آغاز می‌شود..."

#: yuminstall.py:1971
msgid "Dependency Check"
msgstr "بررسی وابستگی"

#: yuminstall.py:1972
#, fuzzy
msgid "Checking dependencies in packages selected for installation"
msgstr "در حال بررسی وابستگی‌ها در بسته‌های انتخاب شدهٔ برای نصب..."

#: installclasses/corecd.py:38
msgid "Kogaion Core CD"
msgstr ""

#: installclasses/corecd.py:48
msgid ""
"Select this installation type to just install a Core System without "
"graphical applications. This is the best choice for Server-oriented "
"deployments."
msgstr ""

#: installclasses/fluxbox.py:38
msgid "Kogaion Fluxbox"
msgstr ""

#: installclasses/fluxbox.py:47
msgid ""
"Select this installation type for a default installation with the Fluxbox "
"geeky minimal environment. After this installation process you will be able "
"to install additional packages."
msgstr ""

#: installclasses/gnome.py:45
msgid "Kogaion GNOME Desktop"
msgstr ""

#: installclasses/gnome.py:47
msgid ""
"Select this installation type for a default installation with the GNOME "
"desktop environment. After this installation process you will be able to "
"install additional packages."
msgstr ""

#: installclasses/kde.py:38
msgid "Kogaion KDE"
msgstr ""

#: installclasses/kde.py:47
msgid ""
"Select this installation type for a default installation with the KDE "
"desktop environment. After this installation process you will be able to "
"install additional packages."
msgstr ""

#: iw/GroupSelector.py:147
#, python-format
msgid "Packages in %s"
msgstr ""

#: iw/GroupSelector.py:424
#, python-format
msgid "Optional packages selected: %(inst)d of %(cnt)d"
msgstr ""

#: iw/GroupSelector.py:426
#, python-format
msgid "<i>%s</i>"
msgstr ""

#: iw/GroupSelector.py:486
msgid "Uncategorized"
msgstr ""

#: iw/account_gui.py:52
#, fuzzy
msgid "Root _Password:"
msgstr "_گذرواژهٔ root: "

#: iw/account_gui.py:54
#, fuzzy
msgid "_Confirm:"
msgstr "_تکرار گذرواژه: "

#: iw/account_gui.py:92
msgid "Caps Lock is on."
msgstr ""

#: iw/account_gui.py:102 iw/account_gui.py:110 iw/account_gui.py:117
#: iw/account_gui.py:137 textw/userauth_text.py:74 textw/userauth_text.py:166
msgid "Error with Password"
msgstr "خطا در گذرواژه"

#: iw/account_gui.py:103
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""
"برای ادامه باید گذرواژهٔ root خود را وارد کنید و با وارد کردن آن برای بار دوم "
"آن را تصدیق کنید."

#: iw/account_gui.py:111
msgid "The passwords you entered were different.  Please try again."
msgstr "گذرواژه‌هایی که وارد کردید فرق داشتند. لطفاً دوباره تلاش کنید."

#: iw/account_gui.py:118
msgid "The root password must be at least six characters long."
msgstr "طول گذرواژهٔ root باید حداقل شش نویسه باشد."

#: iw/account_gui.py:127 textw/userauth_text.py:83 textw/userauth_text.py:190
#, fuzzy
msgid "Weak Password"
msgstr "گذرواژه"

#: iw/account_gui.py:128
#, python-format
msgid "You have provided a weak password: %s"
msgstr ""

#: iw/account_gui.py:130
#, fuzzy
msgid "Use Anyway"
msgstr "نام کاربر"

#: iw/account_gui.py:138 textw/userauth_text.py:75 textw/userauth_text.py:167
#, fuzzy
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""
"گذرواژهٔ درخواست شده دارای نویسه‌های غیراَسکی است که استفاده از آنها در گذرواژه "
"مجاز نیست."

#: iw/advanced_storage.py:91
#, fuzzy
msgid "You must select a NIC to use."
msgstr "باید افرازی را برای ویرایش انتخاب کنید"

#: iw/advanced_storage.py:138
#, fuzzy
msgid "Invalid Initiator Name"
msgstr "نام نامعتبر گروه جلد"

#: iw/advanced_storage.py:139
#, fuzzy
msgid "You must provide an initiator name."
msgstr "باید نام کارگزاری وارد کنید."

#: iw/advanced_storage.py:168
msgid "Error with Data"
msgstr ""

#: iw/autopart_type.py:97
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""

#: iw/autopart_type.py:120
#, fuzzy
msgid "Resize FileSystem Error"
msgstr "در حال قالب‌بندی سیستم پرونده‌ای ‎%s..."

#: iw/autopart_type.py:121
#, python-format
msgid "%(device)s: %(msg)s"
msgstr ""

#: iw/autopart_type.py:130
msgid "Resize Device Error"
msgstr ""

#: iw/autopart_type.py:131
#, python-format
msgid "%(name)s: %(msg)s"
msgstr ""

#: iw/autopart_type.py:228
#, fuzzy
msgid "Use All Space"
msgstr "فضای استفاده شده:"

#: iw/autopart_type.py:230
msgid ""
"Removes all partitions on the selected device(s).  This includes partitions "
"created by other operating systems.\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: iw/autopart_type.py:236
msgid "Replace Existing Linux System(s)"
msgstr ""

#: iw/autopart_type.py:238
msgid ""
"Removes all Linux partitions on the selected device(s). This does not remove "
"other partitions you may have on your storage device(s) (such as VFAT or "
"FAT32).\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: iw/autopart_type.py:246
#, fuzzy
msgid "Shrink Current System"
msgstr "کلید رمزنگاری"

#: iw/autopart_type.py:248
msgid ""
"Shrinks existing partitions to create free space for the default layout."
msgstr ""

#: iw/autopart_type.py:250
#, fuzzy
msgid "Use Free Space"
msgstr "فضای آزادی وجود ندارد"

#: iw/autopart_type.py:252
msgid ""
"Retains your current data and partitions and uses only the unpartitioned "
"space on the selected device(s), assuming you have enough free space "
"available."
msgstr ""

#: iw/autopart_type.py:256
msgid "Create Custom Layout"
msgstr ""

#: iw/autopart_type.py:258
msgid ""
"Manually create your own custom layout on the selected device(s) using our "
"partitioning tool."
msgstr ""

#: iw/blpasswidget.py:44
msgid "_Use a boot loader password"
msgstr "_استفاده از گذرواژهٔ بارکنندهٔ راه‌انداز"

#: iw/blpasswidget.py:45
#, fuzzy
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""
"گذرواژهٔ بارکنندهٔ راه‌انداز مانع از آن می‌شود که کاربران گزینه‌های رد شده به "
"هسته را تغییر دهند. برای امنیت بیشتر سیستم، توصیه می‌شود گذرواژه‌ای تعیین کنید."

#: iw/blpasswidget.py:76
msgid "Change _password"
msgstr "تغییر _گذرواژه"

#: iw/blpasswidget.py:99
msgid "Enter Boot Loader Password"
msgstr "گذرواژهٔ بارکنندهٔ راه‌انداز را وارد کنید"

#: iw/blpasswidget.py:105
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""
"گذرواژه‌ای برای بارکنندهٔ راه‌انداز وارد کنید و سپس آن را در فیلد بعدی تکرار "
"کنید. (توجه کنید که نقشه‌کلید BIOS شما ممکن است با نقشه‌کلید واقعی که به آن "
"عادت دارید تفاوت داشته باشد.)"

#: iw/blpasswidget.py:112
msgid "_Password:"
msgstr "_گذرواژه:"

#: iw/blpasswidget.py:118
msgid "Con_firm:"
msgstr "ت_کرار:"

#: iw/blpasswidget.py:139
msgid "Passwords don't match"
msgstr "گذرواژه‌ها مطابقت نمی‌کنند"

#: iw/blpasswidget.py:140
msgid "Passwords do not match"
msgstr "گذرواژه‌ها مطابقت نمی‌کنند."

#: iw/blpasswidget.py:149
#, fuzzy
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""
"گذرواژهٔ بارکنندهٔ راه‌انداز شما کمتر از شش نویسه است. ما گذرواژهٔ طولانی‌تری "
"برای بارکنندهٔ راه‌انداز توصیه می‌کنیم.\n"
"\n"
"آیا می‌خواهید با همین گذرواژه ادامه دهید؟"

#: iw/bootloader_main_gui.py:36
msgid "Boot Loader Configuration"
msgstr "پیکربندی بارکنندهٔ راه‌انداز"

#: iw/bootloader_main_gui.py:180 iw/bootloader_main_gui.py:185
#: iw/bootloader_main_gui.py:223
#, fuzzy, python-format
msgid "_Install boot loader on /dev/%s."
msgstr "در حال نصب بارکنندهٔ راه‌انداز..."

#: iw/bootloader_main_gui.py:229
#, fuzzy
msgid "_Change device"
msgstr "تعویض سی‌دی"

#: iw/cleardisks_gui.py:33
msgid "Clear Disks Selector"
msgstr ""

#: iw/cleardisks_gui.py:44 iw/filter_gui.py:409
#, fuzzy
msgid "You must select at least one drive to be used for installation."
msgstr "باید حداقل یک دیسک‌سخت انتخاب کنید تا %s روی آن نصب شود."

#: iw/cleardisks_gui.py:54
#, fuzzy
msgid "You must select one drive to boot from."
msgstr "باید افرازی را برای ویرایش انتخاب کنید"

#: iw/cleardisks_gui.py:121 iw/cleardisks_gui.py:140 iw/filter_gui.py:444
#: iw/filter_gui.py:455 iw/filter_gui.py:485
msgid "Model"
msgstr ""

#: iw/cleardisks_gui.py:122 iw/cleardisks_gui.py:141 iw/filter_gui.py:445
#: iw/filter_gui.py:456 iw/filter_gui.py:464 iw/filter_gui.py:475
#: iw/filter_gui.py:486
msgid "Capacity"
msgstr ""

#: iw/cleardisks_gui.py:123 iw/filter_gui.py:446 iw/filter_gui.py:465
#: iw/filter_gui.py:476 iw/filter_gui.py:487
msgid "Vendor"
msgstr ""

#: iw/cleardisks_gui.py:124 iw/filter_gui.py:447 iw/filter_gui.py:466
#: iw/filter_gui.py:477 iw/filter_gui.py:488
msgid "Interconnect"
msgstr ""

#: iw/cleardisks_gui.py:125 iw/filter_gui.py:448 iw/filter_gui.py:478
#: iw/filter_gui.py:489
msgid "Serial Number"
msgstr ""

#: iw/cleardisks_gui.py:139
#, fuzzy
msgid "Boot"
msgstr "راه‌اندازی مجدد"

#: iw/cleardisks_gui.py:176
msgid ""
"<b>Tip:</b> All Linux filesystems on the install target devices will be "
"reformatted and wiped of any data.  Make sure you have backups."
msgstr ""

#: iw/cleardisks_gui.py:178
msgid ""
"<b>Tip:</b> The install target devices will be reformatted and wiped of any "
"data.  Make sure you have backups."
msgstr ""

#: iw/cleardisks_gui.py:180
msgid ""
"<b>Tip:</b> Your filesystems on the install target devices will not be "
"reformatted unless you choose to do so during customization."
msgstr ""

#: iw/congrats_gui.py:32
msgid "Congratulations"
msgstr "تبریک"

#: iw/congrats_gui.py:74 textw/complete_text.py:39
#, fuzzy, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""
"تبریک،  نصب %s شما تمام شد.\n"
"\n"
"‏%s%s"

#: iw/congrats_gui.py:77 textw/complete_text.py:42
msgid "Shutdown"
msgstr ""

#: iw/congrats_gui.py:79 textw/complete_text.py:44
#, fuzzy
msgid "Please shutdown to use the installed system.\n"
msgstr "برای شروع نصب لطفاً به %s متصل شوید..."

#: iw/congrats_gui.py:81 textw/complete_text.py:46
#, fuzzy
msgid "Please reboot to use the installed system.\n"
msgstr "برای شروع نصب لطفاً به %s متصل شوید..."

#: iw/congrats_gui.py:86 textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""

#: iw/congrats_gui.py:90 textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""

#: iw/examine_gui.py:39
msgid "Upgrade Examine"
msgstr "آزمایش ارتقاء"

#: iw/examine_gui.py:60
#, fuzzy
msgid "Fresh Installation"
msgstr "نصب بسته"

#: iw/examine_gui.py:62
#, fuzzy, python-format
msgid ""
"Choose this option to install a fresh copy of %s on your system.  Existing "
"software and data may be overwritten depending on your configuration choices."
msgstr ""
"برای نصب تازه روی سیستم‌تان این گزینه را انتخاب کنید. بسته به انتخاب‌های "
"پیکربندی شما، ممکن است نرم‌افزارها و داده‌های موجود رونویسی شوند."

#: iw/examine_gui.py:67
#, fuzzy
msgid "Bootloader Recovery"
msgstr "بارکنندهٔ راه‌انداز"

#: iw/examine_gui.py:69
#, fuzzy, python-format
msgid ""
"Choose this option if you would like to recover your existing %s system "
"bootloader. This option will preserve the existing data on your storage "
"device(s)."
msgstr ""
"اگر می‌خواهید سیستم %s موجود خود را ارتقاء دهید این گزینه را انتخاب کنید. این "
"گزینه داده‌های موجود در دیسک‌گردان‌های شما را حفظ خواهد کرد."

#: iw/examine_gui.py:109
#, fuzzy
msgid ""
"At least one existing installation has been detected on your system.  What "
"would you like to do?"
msgstr ""
"هیچ گردانندهٔ دستگاهی برای سیستم‌تان بار نشده است. آیا می‌خواهید حالا یک "
"گرداننده بار کنید؟"

#: iw/examine_gui.py:128
#, fuzzy, python-format
msgid "<b>Which %s installation would you like to recover?</b>"
msgstr "می‌خواهید چه‌کار کنید؟"

#: iw/examine_gui.py:142
msgid "Unknown Linux system"
msgstr "سیستم لینوکس ناشناس"

#: iw/filter_gui.py:147
#, python-format
msgid "<b>%s device(s) (%s MB) selected</b> out of %s device(s) (%s MB) total."
msgstr ""

#: iw/filter_gui.py:397
#, fuzzy
msgid "Device Filter"
msgstr "در حال بازیابی"

#: iw/filter_gui.py:449 iw/filter_gui.py:457 iw/filter_gui.py:468
#: iw/filter_gui.py:479 iw/filter_gui.py:494 iw/osbootwidget.py:66
#: iw/partition_gui.py:607
msgid "Device"
msgstr "دستگاه"

#: iw/filter_gui.py:463 iw/filter_gui.py:474 iw/filter_gui.py:490
msgid "Identifier"
msgstr ""

#: iw/filter_gui.py:467
msgid "Paths"
msgstr ""

#: iw/filter_gui.py:491
#, fuzzy
msgid "Port"
msgstr "_قطع شود"

#: iw/filter_gui.py:492
#, fuzzy
msgid "Target"
msgstr "آزمایش"

#: iw/filter_gui.py:493
msgid "LUN"
msgstr ""

#: iw/filter_type.py:55
#, fuzzy
msgid "What type of devices will your installation involve?"
msgstr "چه نوع رسانه‌ای حاوی تصویر نجات است؟"

#: iw/filter_type.py:61
#, fuzzy
msgid "Basic Storage Devices"
msgstr "بستهٔ ناموجود"

#: iw/filter_type.py:62
msgid ""
"Installs or upgrades to typical types of storage devices.  If you're not "
"sure which option is right for you, this is probably it."
msgstr ""

#: iw/filter_type.py:66
#, fuzzy
msgid "Specialized Storage Devices"
msgstr "بستهٔ ناموجود"

#: iw/filter_type.py:67
msgid ""
"Installs or upgrades to devices such as Storage Area Networks (SANs) or "
"mainframe attached disks (DASD), usually in an enterprise environment"
msgstr ""

#: iw/language_gui.py:33 textw/language_text.py:45
msgid "Language Selection"
msgstr "انتخاب زبان"

#: iw/language_gui.py:81 loader/lang.c:371 textw/language_text.py:46
msgid "What language would you like to use during the installation process?"
msgstr "در طی روند نصب می‌خواهید از چه زبانی استفاده کنید؟"

#: iw/lvm_dialog_gui.py:125 iw/lvm_dialog_gui.py:171 iw/lvm_dialog_gui.py:185
#: iw/lvm_dialog_gui.py:224 iw/lvm_dialog_gui.py:301 iw/lvm_dialog_gui.py:690
#: iw/lvm_dialog_gui.py:711
msgid "Not enough space"
msgstr "فضای کافی وجود ندارد"

#: iw/lvm_dialog_gui.py:126
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""
"نمی‌توان اندازهٔ گسترهٔ فیزیکی را تغییر داد چون در این صورت فضای مورد نیاز "
"جلد‌های منطقی مشخص شدهٔ کنونی به بیشتر از فضای موجود افزایش خواهد یافت."

#: iw/lvm_dialog_gui.py:135
msgid "Confirm Physical Extent Change"
msgstr "تصدیق تغییر گسترهٔ فیزیکی"

#: iw/lvm_dialog_gui.py:136
#, fuzzy
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""
"برای انجام این تغییر در مقدار گسترهٔ فیزیکی لازم خواهد بود که اندازه‌های "
"درخواست شدهٔ فعلی برای ‌جلد‌های منطقی به مضرب صحیحی از گسترهٔ فیزیکی به بالا گرد "
"شوند.\n"
"\n"
"این تغییر بلافاصله اثر خواهد کرد."

#: iw/lvm_dialog_gui.py:145 iw/lvm_dialog_gui.py:207
msgid "C_ontinue"
msgstr "ا_دامه"

#: iw/lvm_dialog_gui.py:172
#, fuzzy, python-format
msgid ""
"The physical extent size cannot be changed because the value selected "
"(%(curpe)10.2f MB) is larger than the smallest physical volume "
"(%(maxpvsize)10.2f MB) in the volume group."
msgstr ""
"نمی‌توان اندازهٔ گسترهٔ فیزیکی را تغییر داد چون مقدار انتخاب شده (%10.2f "
"مگابایت) بزرگتر از کوچکترین جلد فیزیکی در گروه جلدی (%10.2f مگابایت) است."

#: iw/lvm_dialog_gui.py:186
#, fuzzy, python-format
msgid ""
"The physical extent size cannot be changed because the value selected "
"(%(curpe)10.2f MB) is too large compared to the size of the smallest "
"physical volume (%(maxpvsize)10.2f MB) in the volume group."
msgstr ""
"نمی‌توان اندازهٔ گسترهٔ فیزیکی را تغییر داد چون مقدار انتخاب شده (%10.2f "
"مگابایت) در مقایسه با اندازهٔ کوچکترین جلد فیزیکی در گروه جلدی (%10.2f "
"مگابایت) خیلی بزرگ است."

#: iw/lvm_dialog_gui.py:200
msgid "Too small"
msgstr "خیلی کوچک است"

#: iw/lvm_dialog_gui.py:201
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""
"این تغییر در مقدار گسترهٔ فیزیکی فضای قابل توجهی را در یک یا چند جلد فیزیکی "
"در گروه جلدی تلف خواهد کرد."

#: iw/lvm_dialog_gui.py:225
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""
"نمی‌توان اندازهٔ گسترهٔ فیزیکی را تغییر داد چون حداکثر جلد منطقی نتیجه شده "
"(%10.2f مگابایت) از یک یا چند جلد فیزیکی تعیین شدهٔ فعلی کوچکتر خواهد شد."

#: iw/lvm_dialog_gui.py:302
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""
"نمی‌توانید این جلد فیزیکی را حذف کنید چون در این صورت گروه جلدی کوچکتر از آن "
"خواهد شد که بتواند جلد‌های منطقی تعیین شدهٔ فعلی را نگه دارد."

#: iw/lvm_dialog_gui.py:401
msgid "Make Logical Volume"
msgstr "ساختن جلد منطقی"

#: iw/lvm_dialog_gui.py:403
#, python-format
msgid "Edit Logical Volume: %s"
msgstr "ویرایش جلد منطقی: %s"

#: iw/lvm_dialog_gui.py:442 iw/raid_dialog_gui.py:443
msgid "_File System Type:"
msgstr "نوع  سیستم پرونده‌ای:"

#: iw/lvm_dialog_gui.py:448
msgid "_Logical Volume Name:"
msgstr "نام جلد _منطقی:"

#: iw/lvm_dialog_gui.py:457 iw/partition_dialog_gui.py:469
msgid "_Size (MB):"
msgstr "ا_ندازه (مگابایت):"

#: iw/lvm_dialog_gui.py:464
#, python-format
msgid "(Max size is %s MB)"
msgstr "(حداکثر اندازه %s مگابایت است)"

#: iw/lvm_dialog_gui.py:469 iw/partition_dialog_gui.py:521
#: iw/partition_ui_helpers_gui.py:331 iw/raid_dialog_gui.py:415
#, fuzzy
msgid "_Encrypt"
msgstr "کلید رمزنگاری"

#: iw/lvm_dialog_gui.py:478 iw/partition_dialog_gui.py:451
#: iw/raid_dialog_gui.py:453
msgid "Original File System Type:"
msgstr "نوع سیستم پرونده‌ای اصلی:"

#: iw/lvm_dialog_gui.py:483 iw/partition_dialog_gui.py:459
#: iw/raid_dialog_gui.py:460
msgid "Original File System Label:"
msgstr "برچسب سیستم پرونده‌ای اصلی:"

#: iw/lvm_dialog_gui.py:488
msgid "Logical Volume Name:"
msgstr "نام جلد منطقی:"

#: iw/lvm_dialog_gui.py:492
msgid "Size (MB):"
msgstr "اندازه (مگابایت):"

#: iw/lvm_dialog_gui.py:519 iw/partition_dialog_gui.py:407
#: iw/raid_dialog_gui.py:434
msgid "_Mount Point:"
msgstr "_نقطهٔ سوار کردن:"

#: iw/lvm_dialog_gui.py:596 iw/partition_dialog_gui.py:113
#: iw/partition_ui_helpers_gui.py:109 iw/partition_ui_helpers_gui.py:131
#: iw/partition_ui_helpers_gui.py:133 iw/raid_dialog_gui.py:169
msgid "<Not Applicable>"
msgstr "<مربوط نیست>"

#: iw/lvm_dialog_gui.py:604
msgid "Illegal Logical Volume Name"
msgstr "نام غیرمجاز جلد منطقی"

#: iw/lvm_dialog_gui.py:619
msgid "Illegal logical volume name"
msgstr "نام غیرمجاز جلد منطقی"

#: iw/lvm_dialog_gui.py:620
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""
"نام جلد منطقی «%s» از قبل مورد استفاده است، لطفاً یک نام دیگر انتخاب کنید."

#: iw/lvm_dialog_gui.py:661 iw/partition_dialog_gui.py:127
#: iw/raid_dialog_gui.py:183
msgid "Mount point in use"
msgstr "نقطهٔ سوار کردن مورد استفاده است"

#: iw/lvm_dialog_gui.py:662 iw/partition_dialog_gui.py:128
#: iw/raid_dialog_gui.py:184
#, fuzzy, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr "نقطهٔ سوار کردن «%s» مورد استفاده است، لطفاً یکی دیگر انتخاب کنید."

#: iw/lvm_dialog_gui.py:677
msgid "Illegal size"
msgstr "اندازهٔ غیرمجاز"

#: iw/lvm_dialog_gui.py:678
msgid "The requested size as entered is not a valid number greater than 0."
msgstr "اندازهٔ درخواست شدهٔ وارد شده یک عدد معتبر بزرگتر از ۰ نیست."

#: iw/lvm_dialog_gui.py:691
#, python-format
msgid ""
"The current requested size (%(size)10.2f MB) is larger than the maximum "
"logical volume size (%(maxlv)10.2f MB). To increase this limit you can "
"create more Physical Volumes from unpartitioned disk space and add them to "
"this Volume Group."
msgstr ""

#: iw/lvm_dialog_gui.py:712
#, python-format
msgid ""
"The logical volumes you have configured require %(size)d MB, but the volume "
"group only has %(tempvgsize)d MB.  Please either make the volume group "
"larger or make the logical volume(s) smaller."
msgstr ""

#: iw/lvm_dialog_gui.py:814
msgid "No free slots"
msgstr "شکاف آزادی وجود ندارد"

#: iw/lvm_dialog_gui.py:815
#, fuzzy, python-format
msgid "You cannot create more than %d logical volume per volume group."
msgid_plural "You cannot create more than %d logical volumes per volume group."
msgstr[0] "نمی‌توانید در هر گروه جلد بیش از %s جلد منطقی ایجاد کنید."
msgstr[1] "نمی‌توانید در هر گروه جلد بیش از %s جلد منطقی ایجاد کنید."

#: iw/lvm_dialog_gui.py:825
msgid "No free space"
msgstr "فضای آزادی وجود ندارد"

#: iw/lvm_dialog_gui.py:826
#, fuzzy
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""
"جایی برای ایجاد جلد منطقی در گروه جلدی باقی نمانده است. برای اضافه کردن یک "
"جلد منطقی باید اندازهٔ یک یا چند جلد منطقی موجود فعلی را کم کنید."

#: iw/lvm_dialog_gui.py:863
#, fuzzy, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr "آیا مطمئنید می‌خواهید جلد منطقی «%s» را حذف کنید؟"

#: iw/lvm_dialog_gui.py:969
msgid "Invalid Volume Group Name"
msgstr "نام نامعتبر گروه جلد"

#: iw/lvm_dialog_gui.py:978
msgid "Name in use"
msgstr "نام مورد استفاده است"

#: iw/lvm_dialog_gui.py:979
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr "نام گروه جلد «%s» از قبل مورد استفاده است. لطفاً نام دیگری انتخاب کنید."

#: iw/lvm_dialog_gui.py:1265
msgid "Not enough physical volumes"
msgstr "جلد‌های فیزیکی کافی وجود ندارند"

#: iw/lvm_dialog_gui.py:1266
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""
"حداقل یک افراز استفاده نشدهٔ جلد فیزیکی برای ایجاد یک گروه جلدی LVM لازم "
"است.\n"
"\n"
"یک افراز یا آرایهٔ RAID از نوع «جلد فیزیکی (LVM)» ایجاد کنید و سپس دوباره "
"گزینهٔ «LVM» را انتخاب کنید."

#: iw/lvm_dialog_gui.py:1277
msgid "Make LVM Volume Group"
msgstr "ساختن گروه جلد LVM"

#: iw/lvm_dialog_gui.py:1280
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr "ویرایش گروه جلد LVM: %s"

#: iw/lvm_dialog_gui.py:1282
msgid "Edit LVM Volume Group"
msgstr "ویرایش گروه جلد LVM"

#: iw/lvm_dialog_gui.py:1298
msgid "_Volume Group Name:"
msgstr "نام گروه _جلد:"

#: iw/lvm_dialog_gui.py:1306
msgid "Volume Group Name:"
msgstr "نام گروه جلد:"

#: iw/lvm_dialog_gui.py:1314
msgid "_Physical Extent:"
msgstr "گسترهٔ _فیزیکی:"

#: iw/lvm_dialog_gui.py:1329
msgid "Physical Volumes to _Use:"
msgstr "جلد‌های فیزیکی مورد ا_ستفاده:"

#: iw/lvm_dialog_gui.py:1335
msgid "Used Space:"
msgstr "فضای استفاده شده:"

#: iw/lvm_dialog_gui.py:1352
msgid "Free Space:"
msgstr "فضای آزاد:"

#: iw/lvm_dialog_gui.py:1370
msgid "Total Space:"
msgstr "فضای کل:"

#: iw/lvm_dialog_gui.py:1408
msgid "Logical Volume Name"
msgstr "نام جلد منطقی"

#: iw/lvm_dialog_gui.py:1411 iw/partition_gui.py:610
#: iw/upgrade_swap_gui.py:138 textw/upgrade_text.py:124
msgid "Mount Point"
msgstr "نقطهٔ سوار کردن"

#: iw/lvm_dialog_gui.py:1414 iw/partition_gui.py:609
msgid "Size (MB)"
msgstr "اندازه (مگابایت)"

#: iw/lvm_dialog_gui.py:1428 iw/osbootwidget.py:96
msgid "_Add"
msgstr "ا_ضافه کردن"

#: iw/lvm_dialog_gui.py:1431 iw/osbootwidget.py:100 iw/partition_gui.py:1815
#: iw/partition_gui.py:1825
msgid "_Edit"
msgstr "_ویرایش"

#: iw/lvm_dialog_gui.py:1446
#, fuzzy
msgid "_Logical Volumes"
msgstr "جلد‌های منطقی"

#: iw/netconfig_dialog.py:192 textw/netconfig_text.py:36
#, fuzzy, python-format
msgid ""
"An error occurred converting the value entered for \"%(field)s\":\n"
"%(errmsg)s"
msgstr ""
"در تبدیل مقدار وارد شده برای «%s» خطایی رخ داد:\n"
"‏%s"

#: iw/netconfig_dialog.py:195 iw/netconfig_dialog.py:204
#: textw/netconfig_text.py:35 textw/netconfig_text.py:42
msgid "Error With Data"
msgstr "خطای داده‌ها"

#: iw/netconfig_dialog.py:203
#, fuzzy, python-format
msgid "A value is required for the field %s."
msgstr "مقداری برای فیلد «%s» لازم است."

#: iw/netconfig_dialog.py:213
#, fuzzy, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr "هنگام نسخه‌برداری از عکس‌های ‌صفحه خطایی رخ داد."

#: iw/netconfig_dialog.py:215
#, fuzzy
msgid "Error Configuring Network"
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: iw/netconfig_dialog.py:241
#, fuzzy
msgid "Dynamic IP Address"
msgstr "‏IP‏ پویا"

#: iw/netconfig_dialog.py:242
#, fuzzy, python-format
msgid "Sending request for IP address information for %s"
msgstr "در حال فرستادن درخواست برای اطلاعات IP‏ برای %s..."

#: iw/netconfig_dialog.py:257 iw/netconfig_dialog.py:260
#: textw/netconfig_text.py:225 textw/netconfig_text.py:228
msgid "IP Address"
msgstr "نشانی IP"

#: iw/netconfig_dialog.py:267 textw/netconfig_text.py:235
#, fuzzy
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr "نشانی‌های IP باید حاوی اعداد بین ۱ و ۲۵۵ باشند"

#: iw/netconfig_dialog.py:268 iw/netconfig_dialog.py:274
#: iw/netconfig_dialog.py:282 iw/netconfig_dialog.py:285
#: textw/netconfig_text.py:236 textw/netconfig_text.py:242
#: textw/netconfig_text.py:250
#, fuzzy
msgid "IPv4 Network Mask"
msgstr "‏IP‏/نقاب شبکه"

#: iw/netconfig_dialog.py:295 textw/netconfig_text.py:263
msgid "Gateway"
msgstr "دروازه"

#: iw/netconfig_dialog.py:305 textw/netconfig_text.py:273
msgid "Nameserver"
msgstr "کارگزار نام"

#: iw/netconfig_dialog.py:314
#, fuzzy
msgid "Error configuring network device:"
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: iw/network_gui.py:73 iw/network_gui.py:79
#, fuzzy
msgid "Error with Hostname"
msgstr "خطای داده‌ها"

#: iw/network_gui.py:74
#, fuzzy
msgid "You must enter a valid hostname for this computer."
msgstr "باید نام کارگزاری وارد کنید."

#: iw/network_gui.py:80
#, fuzzy, python-format
msgid ""
"The hostname \"%(hostname)s\" is not valid for the following reason:\n"
"\n"
"%(herrors)s"
msgstr ""
"نام میزبان «%s» به دلایل زیر معتبر نیست:\n"
"\n"
"‏%s"

#: iw/osbootwidget.py:50
#, fuzzy
msgid "Boot loader operating system list"
msgstr "گذرواژهٔ بارکنندهٔ راه‌انداز خیلی کوتاه است"

#: iw/osbootwidget.py:66
msgid "Default"
msgstr "پیش‌فرض"

#: iw/osbootwidget.py:66 iw/partition_gui.py:608
msgid "Label"
msgstr "برچسب"

#: iw/osbootwidget.py:130
msgid "Image"
msgstr "تصویر"

#: iw/osbootwidget.py:137
#, fuzzy
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""
"برچسبی را که در منوی بارکنندهٔ راه‌انداز نمایش داده می‌شود وارد کنید. دستگاه "
"(یا دیسک سخت و شمارهٔ افراز) دستگاهی است که از آن راه‌اندازی می‌شود."

#: iw/osbootwidget.py:145
msgid "_Label"
msgstr "بر_چسب"

#: iw/osbootwidget.py:153
msgid "_Device"
msgstr "_دستگاه"

#: iw/osbootwidget.py:183
msgid "Default Boot _Target"
msgstr "هدف راه‌اندازی _پیش‌فرض"

#: iw/osbootwidget.py:212
msgid "You must specify a label for the entry"
msgstr "باید برچسبی برای ورودی مشخص کنید"

#: iw/osbootwidget.py:221
msgid "Boot label contains illegal characters"
msgstr "برچسب راه‌اندازی دارای نویسه‌های غیرمجاز است"

#: iw/osbootwidget.py:245
msgid "Duplicate Label"
msgstr "تکثیر برچسب"

#: iw/osbootwidget.py:246
msgid "This label is already in use for another boot entry."
msgstr "این برچسب از قبل برای مدخل راه‌اندازی دیگری استفاده می‌شود."

#: iw/osbootwidget.py:259
msgid "Duplicate Device"
msgstr "تکثیر دستگاه"

#: iw/osbootwidget.py:260
msgid "This device is already being used for another boot entry."
msgstr "این دستگاه از قبل برای مدخل راه‌اندازی دیگری استفاده می‌شود."

#: iw/osbootwidget.py:322
msgid "Cannot Delete"
msgstr "نمی‌توان حذف کرد"

#: iw/osbootwidget.py:323
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""
"نمی‌توان این هدف راه‌اندازی را حذف کرد چون متعلق به سیستم %s است که در حال نصب "
"آن هستید."

#: iw/partition_dialog_gui.py:57
msgid "Additional Size Options"
msgstr "گزینه‌های اضافی اندازه"

#: iw/partition_dialog_gui.py:62
msgid "_Fixed size"
msgstr "اندازهٔ _ثابت"

#: iw/partition_dialog_gui.py:64
msgid "Fill all space _up to (MB):"
msgstr "پر کردن همهٔ فضا _تا (مگابایت):"

#: iw/partition_dialog_gui.py:71
msgid "Fill to maximum _allowable size"
msgstr "پر کردن تا حداکثر اندازهٔ _ممکن"

#: iw/partition_dialog_gui.py:377
msgid "Add Partition"
msgstr "اضافه کردن افراز"

#: iw/partition_dialog_gui.py:379
#, fuzzy, python-format
msgid "Edit Partition: %s"
msgstr "ویرایش افراز: ‎/dev/%s"

#: iw/partition_dialog_gui.py:416
msgid "File System _Type:"
msgstr "_نوع سیستم پرونده‌ای:"

#: iw/partition_dialog_gui.py:432
msgid "Allowable _Drives:"
msgstr "دیسک‌گردان‌های _مجاز:"

#: iw/partition_dialog_gui.py:508
msgid "Force to be a _primary partition"
msgstr "حتماً افراز ا_صلی باشد"

#: iw/partition_gui.py:356
#, python-format
msgid "Drive %(drive)s (%(size)-0.f MB) (Model: %(model)s)"
msgstr ""

#: iw/partition_gui.py:396 iw/partition_gui.py:442 iw/partition_gui.py:509
#: iw/partition_gui.py:999 iw/partition_gui.py:1066
msgid "Free"
msgstr "آزاد"

#: iw/partition_gui.py:474
#, fuzzy, python-format
msgid "LVM Volume Group %s (%-0.f MB)"
msgstr "گروه‌های جلد LVM"

#: iw/partition_gui.py:540
#, python-format
msgid "MD RAID ARRAY %s (%-0.f MB)"
msgstr ""

#: iw/partition_gui.py:611
msgid "Type"
msgstr "نوع"

#: iw/partition_gui.py:612 storage/__init__.py:1784
msgid "Format"
msgstr "قالب‌بندی"

#: iw/partition_gui.py:649
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""
"نقطهٔ سوار کردن/\n"
"‏RAID/جلد"

#: iw/partition_gui.py:651
msgid ""
"Size\n"
"(MB)"
msgstr ""
"اندازه\n"
"(مگابایت)"

#: iw/partition_gui.py:756
msgid "Partitioning"
msgstr "افرازبندی"

#: iw/partition_gui.py:849
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""

#: iw/partition_gui.py:851
#, fuzzy, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr "پیش از ادامهٔ نصب %s این خطاها باید برطرف شوند."

#: iw/partition_gui.py:857
msgid "Partitioning Errors"
msgstr "خطاهای افرازبندی"

#: iw/partition_gui.py:864
msgid "The partitioning scheme you requested generated the following warnings."
msgstr ""

#: iw/partition_gui.py:866
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr "آیا می‌خواهید با همین شِمای افرازبندی درخواست شده ادامه دهید؟"

#: iw/partition_gui.py:871
msgid "Partitioning Warnings"
msgstr "اخطارهای افرازبندی"

#: iw/partition_gui.py:880
#, fuzzy
msgid ""
"The following pre-existing devices have been selected to be formatted, "
"destroying all data."
msgstr ""
"افرازهای از پیش موجود زیر برای قالب‌بندی انتخاب شده‌اند، همهٔ داده‌ها از بین "
"خواهد رفت."

#: iw/partition_gui.py:892
msgid "Format Warnings"
msgstr "اخطارهای قالب‌بندی"

#: iw/partition_gui.py:897 storage/dasd.py:139
msgid "_Format"
msgstr "_قالب‌بندی"

#: iw/partition_gui.py:987
msgid "LVM Volume Groups"
msgstr "گروه‌های جلد LVM"

#: iw/partition_gui.py:1009
msgid "RAID Devices"
msgstr "دستگاه‌های RAID"

#: iw/partition_gui.py:1020 loader/hdinstall.c:206
msgid "Hard Drives"
msgstr "‌دیسک‌گردان‌های سخت"

#: iw/partition_gui.py:1070
msgid "Extended"
msgstr "گسترش یافته"

#: iw/partition_gui.py:1349
msgid "Cannot perform any creation action"
msgstr ""

#: iw/partition_gui.py:1350
msgid ""
"Note that the creation action requires one of the following:\n"
"\n"
"* Free space in one of the Hard Drives.\n"
"* At least two free Software RAID partitions.\n"
"* At least one free physical volume (LVM) partition.\n"
"* At least one Volume Group with free space."
msgstr ""

#: iw/partition_gui.py:1460
#, fuzzy, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
msgstr ""
"‏RAID نرم‌افزاری شما را قادر می‌سازد چندین دیسک را با هم ترکیب کرده،  یک دستگاه "
"RAID بزرگتر تشکیل ‌دهید. می‌توان دستگاه‌های RAID را طوری پیکربندی کرد که در "
"مقایسه با استفاده از یک دیسک‌گردان تکی سرعت و اعتبار بیشتری داشته باشند. برای "
"اطلاعات بیشتر دربارهٔ استفاده از دستگاه‌های RAID لطفاً به مستندات %s مراجعه "
"کنید.\n"
"\n"
"شما در حال حاضر %s افراز RAID نرم‌افزاری قابل استفاده دارید.\n"
"\n"

#: iw/partition_gui.py:1466
#, fuzzy
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""
"برای استفاده از RAID باید اول حداقل دو افراز از نوع «RAID نرم‌افزاری» ایجاد "
"کنید. سپس می‌توانید یک دستگاه RAID ایجاد کنید که می‌تواند قالب‌بندی و سوار "
"شود.\n"
"\n"

#: iw/partition_gui.py:1470
#, fuzzy, python-format
msgid "You currently have %d software RAID partition free to use."
msgid_plural "You currently have %d software RAID partitions free to use."
msgstr[0] "ایجاد یک ا_فراز RAID نرم‌افزاری."
msgstr[1] "ایجاد یک ا_فراز RAID نرم‌افزاری."

#: iw/partition_gui.py:1474
#, fuzzy
msgid "About RAID"
msgstr "‏RAID نرم‌افزاری"

#: iw/partition_gui.py:1480
#, python-format
msgid ""
"Logical Volume Manager (LVM) is a 3 level construct. The first level is made "
"up of disks or partitions formatted with LVM metadata called Physical "
"Volumes (PV).  A Volume Group (VG) sits on top of one or more PVs. The VG, "
"in turn, is the base to create one or more Logical Volumes (LV).  Note that "
"a VG can be an aggregate of PVs from multiple physical disks.  For more "
"information on using LVM please consult the %s documentation\n"
msgstr ""

#: iw/partition_gui.py:1488
msgid ""
"To create a PV you need a partition with free space.  To create a VG you "
"need a PV that is not part of any existing VG.  To create an LV you need a "
"VG with free space.\n"
"\n"
msgstr ""

#: iw/partition_gui.py:1492
#, python-format
msgid "You currently have %d available PV free to use.\n"
msgid_plural "You currently have %d available PVs free to use.\n"
msgstr[0] ""
msgstr[1] ""

#: iw/partition_gui.py:1496
msgid "You currently have free space to create PVs."
msgstr ""

#: iw/partition_gui.py:1499
msgid "About LVM"
msgstr ""

#: iw/partition_gui.py:1530
msgid "Couldn't Create Drive Clone Editor"
msgstr "ویرایشگر همانندسازی دیسک‌گردان ایجاد نشد"

#: iw/partition_gui.py:1531
msgid "The drive clone editor could not be created for some reason."
msgstr "ویرایشگر همانندسازی دیسک‌گردان به دلیلی ایجاد نشد."

#: iw/partition_gui.py:1589 storage/partitioning.py:196
#: storage/partitioning.py:239
msgid "Error Partitioning"
msgstr "خطا در افرازبندی"

#: iw/partition_gui.py:1590
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr "نمی‌توان افرازهای درخواست شده را تخصیص داد: %s."

#: iw/partition_gui.py:1599
#, python-format
msgid "Warning: %s."
msgstr "اخطار: %s."

#: iw/partition_gui.py:1630
msgid "Unable To Edit"
msgstr "ویرایش ممکن نیست"

#: iw/partition_gui.py:1631
#, fuzzy, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""
"نمی‌توانید این افراز را ویرایش کنید:\n"
"\n"

#: iw/partition_gui.py:1814
msgid "_Create"
msgstr ""

#: iw/partition_gui.py:1817
msgid "Re_set"
msgstr "_مقادیر اولیه"

#: iw/partition_gui.py:1840
#, fuzzy
msgid "Please Select A Device"
msgstr "لطفاً دیسک‌گردان مبدأ را انتخاب کنید."

#: iw/partition_ui_helpers_gui.py:312
#, fuzzy
msgid "_Format as:"
msgstr "قالب‌بندی به شکل:"

#: iw/partition_ui_helpers_gui.py:334
#, fuzzy
msgid "Mi_grate filesystem to:"
msgstr "کوچ‌دادن سیستم پرونده‌ای"

#: iw/partition_ui_helpers_gui.py:364
#, fuzzy
msgid "_Resize"
msgstr "اندازهٔ _ثابت"

#: iw/partition_ui_helpers_gui.py:435
#, fuzzy, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""
"افرازهایی از نوع «%s» باید درون یک دیسک‌گردان جا بگیرند. این کار با انتخاب "
"دیسک‌گردان در فهرست «دیسک‌گردان‌های مجاز» انجام می‌گیرد."

#: iw/raid_dialog_gui.py:382
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""
"حداقل دو افراز RAID نرم‌افزاری استفاده نشده برای ایجاد یک دستگاه RAID لازم "
"است.\n"
"\n"
"اول حداقل دو افراز از نوع «RAID نرم‌افزاری» ایجاد کنید، و سپس  دوباره گزینهٔ "
"«RAID» را انتخاب کنید."

#: iw/raid_dialog_gui.py:396
msgid "Make RAID Device"
msgstr "ساختن دستگاه RAID"

#: iw/raid_dialog_gui.py:399
#, fuzzy, python-format
msgid "Edit RAID Device: %s"
msgstr "ویرایش دستگاه RAID"

#: iw/raid_dialog_gui.py:401
msgid "Edit RAID Device"
msgstr "ویرایش دستگاه RAID"

#: iw/raid_dialog_gui.py:468
msgid "RAID _Device:"
msgstr "_دستگاه RAID:"

#: iw/raid_dialog_gui.py:486
msgid "RAID _Level:"
msgstr "_سطح RAID:"

#: iw/raid_dialog_gui.py:528
msgid "_RAID Members:"
msgstr "_اعضای RAID:"

#: iw/raid_dialog_gui.py:545
msgid "Number of _spares:"
msgstr "تعداد  قطعات_یدکی:"

#: iw/raid_dialog_gui.py:555
msgid "_Format partition?"
msgstr "افراز _قالب‌بندی شود؟"

#: iw/raid_dialog_gui.py:629
msgid ""
"The source drive has no partitions to be cloned.  You must first define "
"partitions of type 'software RAID' on this drive before it can be cloned."
msgstr ""
"دیسک‌گردان مبدأ افرازی برای همانندسازی ندارد. باید اول افرازهایی از نوع «RAID "
"نرم‌افزاری» روی این دیسک‌گردان تعیین کنید، سپس می‌توانید آن را همانندسازی کنید."

#: iw/raid_dialog_gui.py:633 iw/raid_dialog_gui.py:639
#: iw/raid_dialog_gui.py:653 iw/raid_dialog_gui.py:666
msgid "Source Drive Error"
msgstr "خطای دیسک‌گردان مبدأ"

#: iw/raid_dialog_gui.py:640
#, fuzzy
msgid ""
"The source drive you selected has partitions which are not of type 'software "
"RAID'.\n"
"\n"
"You must remove these partitions before this drive can be cloned. "
msgstr ""
"دیسک‌گردان مبدأ دارای افرازهایی است که از نوع «RAID نرم‌افزاری» نیستند.\n"
"\n"
"قبل از این که بتوان این دیسک‌گردان را همانندسازی کرد باید این افرازها حذف "
"شوند."

#: iw/raid_dialog_gui.py:654
#, fuzzy, python-format
msgid ""
"The source drive you selected has partitions which are not constrained to "
"the drive %s.\n"
"\n"
"You must remove these partitions or restrict them to this drive before this "
"drive can be cloned. "
msgstr ""
"دیسک‌گردان مبدأ انتخاب شده دارای افرازهایی است که در دیسک‌گردان ‏/dev/%s جا "
"داده نشده‌اند.\n"
"\n"
"قبل از این که بتوان این دیسک‌گردان را همانندسازی کرد، این افرازها باید حذف یا "
"محدود به این دیسک‌گردان شوند."

#: iw/raid_dialog_gui.py:667
#, fuzzy
msgid ""
"The source drive you selected has software RAID partition(s) which are "
"members of an active software RAID device.\n"
"\n"
"You must remove these partitions before this drive can be cloned."
msgstr ""
"دیسک‌گردان مبدأ دارای افرازهای RAID نرم‌افزاری است که اعضای یک دستگاه RAID "
"نرم‌افزاری فعال هستند.\n"
"\n"
"قبل از این که بتوان این دیسک‌گردان را همانندسازی کرد، این افرازها باید حذف "
"شوند."

#: iw/raid_dialog_gui.py:681 iw/raid_dialog_gui.py:687
#: iw/raid_dialog_gui.py:703
msgid "Target Drive Error"
msgstr "خطای دیسک‌گردان هدف"

#: iw/raid_dialog_gui.py:682
msgid "Please select the target drives for the clone operation."
msgstr "لطفاً دیسک‌گردان‌های هدف را برای عملیات همانندسازی انتخاب کنید."

#: iw/raid_dialog_gui.py:688
#, fuzzy, python-format
msgid "The source drive %s cannot be selected as a target drive as well."
msgstr ""
"نمی‌توان دیسک‌گردان مبدأ ‎/dev/%s را به عنوان دیسک‌گردان هدف هم انتخاب کرد."

#: iw/raid_dialog_gui.py:704
#, fuzzy, python-format
msgid ""
"The target drive %(path)s has a partition which cannot be removed for the "
"following reason:\n"
"\n"
"\"%(rc)s\"\n"
"\n"
"You must remove this partition before this drive can be a target."
msgstr ""
"دیسک‌گردان هدف ‎/dev/%s دارای افرازی است که به دلایل زیر نمی‌توان آن را حذف "
"کرد:\n"
"\n"
"«%s»\n"
"\n"
"قبل از این که این دیسک‌گردان بتواند هدف قرار گیرد، این افراز باید حذف شود."

#: iw/raid_dialog_gui.py:767
msgid "Please select a source drive."
msgstr "لطفاً دیسک‌گردان مبدأ را انتخاب کنید."

#: iw/raid_dialog_gui.py:787
#, fuzzy, python-format
msgid ""
"The drive %s will now be cloned to the following drives:\n"
"\n"
msgstr ""
"دیسک‌گردان ‎/dev/%s اکنون به دیسک‌گردان‌های زیر همانندسازی خواهد شد:\n"
"\n"

#: iw/raid_dialog_gui.py:792
msgid ""
"\n"
"\n"
"WARNING! ALL DATA ON THE TARGET DRIVES WILL BE DESTROYED."
msgstr ""
"\n"
"\n"
"اخطار! همهٔ داده‌های روی دیسک‌گردان‌های هدف از بین خواهد رفت."

#: iw/raid_dialog_gui.py:795
msgid "Final Warning"
msgstr "اخطار نهایی"

#: iw/raid_dialog_gui.py:797
msgid "Clone Drives"
msgstr "همانندسازی دیسک‌گردان‌ها"

#: iw/raid_dialog_gui.py:806
msgid "There was an error clearing the target drives.  Cloning failed."
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: iw/raid_dialog_gui.py:829
#, fuzzy
msgid "Clone Drive Tool"
msgstr "همانندسازی دیسک‌گردان‌ها"

#: iw/raid_dialog_gui.py:838
msgid ""
"This tool clones the layout from a partitioned source onto other similar "
"sized drives. The source must have partitions which are restricted to that "
"drive and must ONLY contain unused software RAID partitions.  EVERYTHING on "
"the target drive(s) will be destroyed.\n"
msgstr ""

#: iw/raid_dialog_gui.py:849
msgid "Source Drive:"
msgstr "دیسک‌گردان مبدأ:"

#: iw/raid_dialog_gui.py:857
msgid "Target Drive(s):"
msgstr "دیسک‌گردان‌(های) هدف:"

#: iw/raid_dialog_gui.py:865
msgid "Drives"
msgstr "دیسک‌گردان‌ها"

#: iw/task_gui.py:55
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""

#: iw/task_gui.py:132
#, fuzzy
msgid "Edit Repository"
msgstr "برچسب راه‌اندازی نامعتبر"

#: iw/task_gui.py:156
#, fuzzy, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""
"نقطهٔ سوار کردن «%s» از قبل مورد استفاده است، لطفاً نقطهٔ سوار کردن دیگری "
"انتخاب کنید."

#: iw/task_gui.py:236
#, fuzzy
msgid "Invalid Proxy URL"
msgstr "برچسب راه‌اندازی نامعتبر"

#: iw/task_gui.py:237
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr ""

#: iw/task_gui.py:249 iw/task_gui.py:430
#, fuzzy
msgid "Invalid Repository URL"
msgstr "برچسب راه‌اندازی نامعتبر"

#: iw/task_gui.py:250 iw/task_gui.py:431
#, fuzzy
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr "باید نام کارگزاری وارد کنید."

#: iw/task_gui.py:271 iw/task_gui.py:440
#, fuzzy
msgid "No Media Found"
msgstr "دیسک‌گردانی پیدا نشد"

#: iw/task_gui.py:272 iw/task_gui.py:441
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr ""

#: iw/task_gui.py:303 iw/task_gui.py:464
#, fuzzy
msgid "Please enter an NFS server and path."
msgstr "لطفاً نام گروه جلد را وارد کنید."

#: iw/task_gui.py:322
#, fuzzy, python-format
msgid ""
"The following error occurred while setting up the repository:\n"
"\n"
"%s"
msgstr ""
"خطای زیر هنگام تجزیهٔ پیکربندی kickstart پیدا شد:\n"
"\n"
"‏%s"

#: iw/task_gui.py:345
#, fuzzy
msgid "Invalid Repository Name"
msgstr "نام میزبان نامعتبر"

#: iw/task_gui.py:346
#, fuzzy
msgid "You must provide a repository name."
msgstr "باید نام کارگزاری وارد کنید."

#: iw/task_gui.py:496 ui/addrepo.glade.h:3
msgid "Add Repository"
msgstr ""

#: iw/task_gui.py:501
msgid "No Software Repos Enabled"
msgstr ""

#: iw/task_gui.py:502
#, fuzzy
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr "پیش از ادامهٔ نصب %s این خطاها باید برطرف شوند."

#: iw/timezone_gui.py:63 textw/timezone_text.py:95
msgid "Time Zone Selection"
msgstr "انتخاب منطقهٔ زمانی"

#: iw/upgrade_bootloader_gui.py:37 textw/upgrade_bootloader_text.py:137
msgid "Upgrade Boot Loader Configuration"
msgstr "ارتقاء دادن پیکربندی بارکنندهٔ راه‌انداز"

#: iw/upgrade_bootloader_gui.py:123
msgid "_Update boot loader configuration"
msgstr "_به‌هنگام‌سازی پیکربندی بارکنندهٔ راه‌انداز"

#: iw/upgrade_bootloader_gui.py:124
msgid "This will update your current boot loader."
msgstr "این کار بارکنندهٔ راه‌انداز فعلی شما را به‌هنگام خواهد کرد."

#: iw/upgrade_bootloader_gui.py:128 textw/upgrade_bootloader_text.py:108
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr ""

#: iw/upgrade_bootloader_gui.py:131 textw/upgrade_bootloader_text.py:112
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr ""
"برنامهٔ نصب نمی‌تواند بارکنندهٔ راه‌انداز مورد استفاده در سیستم شما را تشخیص دهد."

#: iw/upgrade_bootloader_gui.py:138 textw/upgrade_bootloader_text.py:121
#, fuzzy, python-format
msgid ""
"The installer has detected the %(type)s boot loader currently installed on "
"%(bootDev)s."
msgstr "برنامهٔ نصب، بارکنندهٔ راه‌انداز %s را که اکنون بر %s نصب شده تشخیص داد."

#: iw/upgrade_bootloader_gui.py:142
msgid "This is the recommended option."
msgstr "این گزینه توصیه می‌شود."

#: iw/upgrade_bootloader_gui.py:147
msgid "_Create new boot loader configuration"
msgstr "ای_جاد پیکربندی بارکنندهٔ راه‌انداز جدید"

#: iw/upgrade_bootloader_gui.py:149
#, fuzzy
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""
"این کار شما را قادر می‌سازد پیکربندی بارکنندهٔ راه‌انداز جدیدی ایجاد کنید. اگر "
"می‌خواهید بتوانید بارکننده‌های راه‌انداز را تعویض کنید، باید این گزینه را "
"انتخاب کنید."

#: iw/upgrade_bootloader_gui.py:156
msgid "_Skip boot loader updating"
msgstr "_پرش از روی به‌هنگام‌سازی بارکنندهٔ راه‌انداز"

#: iw/upgrade_bootloader_gui.py:157
#, fuzzy
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""
"این کار هیچ تغییری در پیکربندی بارکنندهٔ راه‌انداز نخواهد داد. اگر از بارکنندهٔ "
"راه‌اندازی طرف سومی استفاده می‌کنید، باید این را انتخاب کنید."

#: iw/upgrade_bootloader_gui.py:168
msgid "What would you like to do?"
msgstr "می‌خواهید چه‌کار کنید؟"

#: iw/upgrade_migratefs_gui.py:38 textw/upgrade_text.py:42
msgid "Migrate File Systems"
msgstr "کوچ‌دادن سیستم پرونده‌ای"

#: iw/upgrade_migratefs_gui.py:69 textw/upgrade_text.py:44
#, fuzzy, python-format
msgid ""
"This release of %(productName)s supports an updated file system, which has "
"several benefits over the file system traditionally shipped in "
"%(productName)s.  This installation program can migrate formatted partitions "
"without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""
"این انتشار %s از سیستم پرونده‌ای روزنگار ext3 پشتیبانی می‌کند. این سیستم "
"پرونده‌ای نسبت به سیستم پروندهٔ ext2 که به طور سنتی با %s ارسال می‌شود چند "
"برتری دارد. افرازهای قالب‌بندی شدهٔ ext2 را می‌توان بدون از دست دادن داده‌ها به "
"ext3 کوچ داد.\n"
"\n"
"کدام یک از این افرازها را می‌خواهید کوچ دهید؟"

#: iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr "ارتقاء دادن افراز حافظهٔ مبادله"

#: iw/upgrade_swap_gui.py:92 textw/upgrade_text.py:107
#, fuzzy, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have "
"%dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""
"هستهٔ ‎2.4 به حافظهٔ مبادلهٔ بسیار بیشتری از هسته‌های قدیمی‌تر نیاز دارد، تقریباً "
"به اندازهٔ دو برابر حافظهٔ RAM سیستم. شما در حال حاضر %d مگابایت حافظهٔ مبادله "
"پیکربندی کرده‌اید، ولی اکنون می‌توانید فضای مبادلهٔ بیشتری روی یکی از سیستم‌های "
"پرونده‌ای خود ایجاد کنید."

#: iw/upgrade_swap_gui.py:99
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""
"\n"
"\n"
"برنامهٔ نصب کننده %s مگابایت RAM تشخیص داد.\n"

#: iw/upgrade_swap_gui.py:111
msgid "I _want to create a swap file"
msgstr "_پروندهٔ مبادله ایجاد شود"

#: iw/upgrade_swap_gui.py:120
msgid "Select the _partition to put the swap file on:"
msgstr "افرازی انتخاب کنید تا پروندهٔ مبادله روی آن قرار گیرد:"

#: iw/upgrade_swap_gui.py:138 textw/upgrade_text.py:125
msgid "Partition"
msgstr "افراز"

#: iw/upgrade_swap_gui.py:138
msgid "Free Space (MB)"
msgstr "فضای آزاد (مگابایت)"

#: iw/upgrade_swap_gui.py:156
#, fuzzy, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""
"توصیه می‌شود که پروندهٔ مبادله حداقل %d مگابایت باشد. لطفاً اندازهٔ پروندهٔ "
"مبادله را وارد کنید:"

#: iw/upgrade_swap_gui.py:171
msgid "Swap file _size (MB):"
msgstr "ا_ندازهٔ پروندهٔ مبادله (مگابایت):"

#: iw/upgrade_swap_gui.py:181
msgid "I _don't want to create a swap file"
msgstr "_نمی خواهم پروندهٔ مبادله ایجاد کنم"

#: iw/upgrade_swap_gui.py:191
#, fuzzy
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""
"اکیداً توصیه می‌شود پروندهٔ مبادله‌ای ایجاد کنید. شکست در این کار ممکن است باعث "
"شود برنامهٔ نصب به طور غیر عادی قطع شود. آیا مطمئنید که می‌خواهید ادامه دهید؟"

#: iw/upgrade_swap_gui.py:199 textw/upgrade_text.py:194
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr "اندازهٔ پروندهٔ مبادله باید بین ۱ و ۲۰۰۰ مگابایت باشد."

#: iw/upgrade_swap_gui.py:206 textw/upgrade_text.py:189
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr ""
"روی دستگاهی که برای افراز حافظهٔ مبادله انتخاب کرده‌اید فضای کافی وجود ندارد."

#: iw/welcome_gui.py:56 textw/welcome_text.py:36
msgid "Network Install Required"
msgstr ""

#: iw/welcome_gui.py:57 textw/welcome_text.py:37
msgid ""
"Your installation source is set to a network location, but no netork devices "
"were found on your system.  To avoid a network installation, boot with the "
"full DVD, full CD set, or do not pass a repo= parameter that specifies a "
"network source."
msgstr ""

#: iw/welcome_gui.py:67
#, fuzzy
msgid "E_xit Installer"
msgstr "نصب کنندهٔ %s"

#: iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr "پیکربندی بارکنندهٔ راه‌انداز z/IPL"

#: iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr "بارکنندهٔ راه‌انداز z/IPL بر سیستم شما نصب خواهد شد."

#: iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""
"بارکنندهٔ راه‌انداز z/IPL اکنون بر سیستم شما نصب خواهد شد.\n"
"\n"
"افراز root همان افرازی خواهد بود که قبلاً در برپاسازی افراز انتخاب کردید.\n"
"\n"
"هستهٔ مورد استفاده برای راه‌اندازی رایانه همان خواهد بود که به طور پیش‌فرض نصب "
"می‌شود.\n"
"\n"
"اگر بعد از نصب خواستید تغییراتی ایجاد کنید پروندهٔ پیکربندی ‎/etc/zipl.conf را "
"تغییر دهید.\n"
"\n"
"حالا می‌توانید پارامترهای هستهٔ اضافی‌ای را که رایانه یا برپاسازی شما لازم "
"داشته باشد وارد کنید."

#: iw/zipl_gui.py:90 textw/zipl_text.py:72
msgid "Kernel Parameters"
msgstr "پارامترهای هسته"

#: iw/zipl_gui.py:93 iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr "پارامترهای Chandev"

#: loader/cdinstall.c:185 loader/cdinstall.c:206 loader/mediacheck.c:61
msgid "Media Check"
msgstr "بررسی رسانه"

#: loader/cdinstall.c:185 loader/cdinstall.c:188 loader/cdinstall.c:206
#: loader/cdinstall.c:214 loader/method.c:324
msgid "Test"
msgstr "آزمایش"

#: loader/cdinstall.c:185 loader/cdinstall.c:189
#, fuzzy
msgid "Eject Disc"
msgstr "بیرون دادن سی‌دی"

#: loader/cdinstall.c:186
#, fuzzy, c-format
msgid ""
"Choose \"%s\" to test the disc currently in the drive, or \"%s\" to eject "
"the disc and insert another for testing."
msgstr ""
"برای آزمایش سی‌دی داخل دیسک‌گردان روی دکمهٔ «%s»، و برای بیرون دادن سی‌دی و داخل "
"کردن سی‌دی دیگری برای آزمایش روی دکمهٔ «%s» کلیک کنید."

#: loader/cdinstall.c:207
#, c-format
msgid ""
"If you would like to test additional media, insert the next disc and press "
"\"%s\". Testing each disc is not strictly required, however it is highly "
"recommended.  Minimally, the discs should be tested prior to using them for "
"the first time. After they have been successfully tested, it is not required "
"to retest each disc prior to using it again."
msgstr ""

#: loader/cdinstall.c:229
#, fuzzy, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""
"سی‌دی %s در هیچ یک از سی‌دی‌گردان‌های شما پیدا نشد. لطفاً سی‌دی %s را وارد کنید و "
"برای تلاش مجدد روی دکمهٔ «%s» کلیک کنید."

#: loader/cdinstall.c:248
#, fuzzy
msgid "Disc Found"
msgstr "سی‌دی پیدا شد"

#: loader/cdinstall.c:249
#, fuzzy, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""
"برای شروع آزمایش رسانهٔ سی‌دی قبل از نصب روی دکمهٔ %s کلیک کنید.\n"
"\n"
"برای پرش از آزمایش رسانه و شروع نصب روی دکمهٔ %s کلیک کنید."

#: loader/cdinstall.c:330
#, fuzzy
msgid "Scanning"
msgstr "اخطار"

#: loader/cdinstall.c:330
#, c-format
msgid "Looking for installation images on CD device %s\n"
msgstr ""

#: loader/cdinstall.c:332
#, c-format
msgid "Looking for installation images on CD device %s"
msgstr ""

#: loader/cdinstall.c:424
#, fuzzy, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""
"سی‌دی %s در هیچ یک از سی‌دی‌گردان‌های شما پیدا نشد. لطفاً سی‌دی %s را وارد کنید و "
"برای تلاش مجدد روی دکمهٔ «%s» کلیک کنید."

#: loader/cdinstall.c:430
#, fuzzy
msgid "Disc Not Found"
msgstr "سی‌دی پیدا نشد"

#: loader/cdinstall.c:431 loader/driverdisk.c:423 loader/driverdisk.c:461
#: loader/driverdisk.c:542 loader/driverselect.c:79 loader/driverselect.c:153
#: loader/driverselect.c:179 loader/hdinstall.c:206 loader/hdinstall.c:260
#: loader/kbd.c:119 loader/loader.c:506 loader/loader.c:523
#: loader/loader.c:1375 loader/loader.c:1408 loader/net.c:578 loader/net.c:949
#: loader/net.c:1848 loader/net.c:1867 loader/nfsinstall.c:91
#: loader/urls.c:259 storage/__init__.py:157 storage/devicetree.py:90
#: textw/constants_text.py:52
msgid "Back"
msgstr "عقب"

#: loader/cdinstall.c:505
msgid "Cannot find kickstart file on CDROM."
msgstr "نمی‌توان پروندهٔ راه‌اندازی را روی سی‌دی یافت."

#: loader/copy.c:51 loader/method.c:278
#, fuzzy, c-format
msgid "Failed to read directory %s: %m"
msgstr "خواندن شاخهٔ %s شکست خورد: %s"

#: loader/driverdisk.c:304
msgid "Loading"
msgstr "در حال بار کردن"

#: loader/driverdisk.c:304
#, fuzzy
msgid "Reading driver disk"
msgstr "در حال خواندن دیسک گرداننده..."

#: loader/driverdisk.c:417 loader/driverdisk.c:456
msgid "Driver Disk Source"
msgstr "مبدأ دیسک گرداننده"

#: loader/driverdisk.c:418
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""
"شما دارای چندین دستگاه هستید که می‌توانند به عنوان مبدأ دیسک گرداننده عمل "
"کنند. از کدام یک می‌خواهید استفاده کنید؟"

#: loader/driverdisk.c:457
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""
"چند افراز روی این دستگاه است که می‌تواند حاوی تصویر دیسک گرداننده باشند. از "
"کدام یک می‌خواهید استفاده کنید؟"

#: loader/driverdisk.c:492
msgid "Failed to mount partition."
msgstr "سوار کردن افراز شکست خورد."

#: loader/driverdisk.c:497
msgid "Select driver disk image"
msgstr "انتخاب تصویر دیسک گرداننده"

#: loader/driverdisk.c:498
msgid "Select the file which is your driver disk image."
msgstr "پرونده‌ای را که تصویر دیسک گردانندهٔ شماست انتخاب کنید."

#: loader/driverdisk.c:527
msgid "Failed to load driver disk from file."
msgstr "بار کردن تصویر دیسک گرداننده از پرونده شکست خورد."

#: loader/driverdisk.c:539
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr ""
"دیسک گردانندهٔ خود را در ‎/dev/%s وارد کنید و  برای ادامه روی دکمهٔ «تأیید» "
"کلیک کنید."

#: loader/driverdisk.c:542
msgid "Insert Driver Disk"
msgstr "دیسک گرداننده را وارد کنید"

#: loader/driverdisk.c:555
msgid "Failed to mount driver disk."
msgstr "سوار کردن دیسک گرداننده شکست خورد."

#: loader/driverdisk.c:563
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr ""

#: loader/driverdisk.c:626
msgid "Manually choose"
msgstr "انتخاب دستی"

#: loader/driverdisk.c:627
msgid "Load another disk"
msgstr "بار کردن یک دیسک دیگر"

#: loader/driverdisk.c:628
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""
"هیچ دستگاهی از نوع مناسب روی این دیسک گرداننده پیدا شد. آیا می‌خواهید به طور "
"دستی گرداننده را انتخاب کنید، با وجود پیدا نشدن گرداننده ادامه دهید، یا دیسک "
"گردانندهٔ دیگری را بار کنید؟"

#: loader/driverdisk.c:666
msgid "Driver disk"
msgstr "دیسک گرداننده"

#: loader/driverdisk.c:667
msgid "Do you have a driver disk?"
msgstr "آیا دیسک گرداننده دارید؟"

#: loader/driverdisk.c:676
msgid "More Driver Disks?"
msgstr "دیسک گردانندهٔ دیگری هست؟"

#: loader/driverdisk.c:677
msgid "Do you wish to load any more driver disks?"
msgstr "آیا می‌خواهید دیسک‌های گردانندهٔ دیگری را بار کنید؟"

#: loader/driverdisk.c:722 loader/driverdisk.c:761 loader/hdinstall.c:352
#: loader/kickstart.c:132 loader/kickstart.c:142 loader/kickstart.c:184
#: loader/kickstart.c:189 loader/kickstart.c:510 loader/modules.c:382
#: loader/modules.c:398 loader/net.c:1557 loader/net.c:1578
#: loader/nfsinstall.c:366 loader/urlinstall.c:372 loader/urlinstall.c:383
#: loader/urlinstall.c:390
msgid "Kickstart Error"
msgstr "خطای راه‌اندازی"

#: loader/driverdisk.c:723
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr "مبدأ نامعلوم راه‌اندازی دیسک گرداننده: %s"

#: loader/driverdisk.c:762
#, fuzzy, c-format
msgid ""
"The following invalid argument was specified for the kickstart driver disk "
"command: %s"
msgstr ""
"آرگومان‌های نامعتبر زیر برای فرمان راه‌اندازی دیسک گرداننده مشخص شده بودند: %s:"
"%s"

#: loader/driverselect.c:68
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""

#: loader/driverselect.c:89
msgid "Enter Module Parameters"
msgstr "پارامترهای پیمانه را وارد کنید"

#: loader/driverselect.c:152
msgid "No drivers found"
msgstr "گرداننده‌ای پیدا نشد"

#: loader/driverselect.c:152
msgid "Load driver disk"
msgstr "بار کردن دیسک گرداننده"

#: loader/driverselect.c:153
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""
"هیچ گرداننده‌ای پیدا نشد که دستی وارد شود. آیا می‌خواهید از دیسک گرداننده "
"استفاده کنید؟"

#: loader/driverselect.c:171
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""
"لطفاً گرداننده‌ای را که می‌خواهید بار کنید از فهرست زیر انتخاب کنید. اگر چیزی "
"ظاهر نشد و شما دیسک گرداننده‌ای دارید، F2 را فشار دهید."

#: loader/driverselect.c:180
msgid "Specify optional module arguments"
msgstr "آرگومان‌های اختیاری پیمانه را مشخص کنید"

#: loader/driverselect.c:205
msgid "Select Device Driver to Load"
msgstr "گردانندهٔ دستگاه را انتخاب کنید تا بار شود"

#: loader/hdinstall.c:116
#, fuzzy
msgid ""
"An error occured finding the installation image on your hard drive.  Please "
"check your images and try again."
msgstr ""
"هنگام خواندن نصب از تصاویر ایزو خطایی رخ داد. لطفاً تصاویر ایزو خود را بررسی "
"کرده و دوباره امتحان کنید."

#: loader/hdinstall.c:207
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""
"به‌نظر نمی‌رسد که هیچ دیسک سختی روی سیستم‌تان داشته باشید! آیا می‌خواهید "
"دستگاه‌های اضافی را پیکربندی کنید؟"

#: loader/hdinstall.c:220
#, fuzzy, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""
"کدام افراز و شاخهٔ روی آن افزار حاوی تصاویر (iso9660) سی‌دی برای %s است؟ اگر "
"دیسک‌گردانی که از آن استفاده می‌کنید اینجا فهرست نشده است، F2 را برای پیکربندی "
"دستگاه‌های بیشتر فشار دهید."

#: loader/hdinstall.c:244
#, fuzzy
msgid "Directory holding image:"
msgstr "شاخهٔ حاوی تصاویر:"

#: loader/hdinstall.c:272
msgid "Select Partition"
msgstr "انتخاب افراز"

#: loader/hdinstall.c:319
#, fuzzy, c-format
msgid "Device %s does not appear to contain an installation image."
msgstr "به‌نظر نمی‌رسد دستگاه %s حاوی تصاویر سی‌دی %s باشد."

#: loader/hdinstall.c:353
#, fuzzy, c-format
msgid "Bad argument to HD kickstart method command: %s"
msgstr "آرگومان بد برای فرمان روش راه‌اندازی دیسک سخت %s : %s"

#: loader/hdinstall.c:425 loader/hdinstall.c:481
msgid "Cannot find kickstart file on hard drive."
msgstr "نمی‌توان پروندهٔ راه‌اندازی را روی دیسک سخت پیدا کرد."

#: loader/hdinstall.c:468
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr "نمی‌توان دیسک سخت را برای دیسک BIOS ‏%s پیدا کرد"

#: loader/kbd.c:117
msgid "Keyboard Type"
msgstr "نوع صفحه‌کلید"

#: loader/kbd.c:118
msgid "What type of keyboard do you have?"
msgstr "چه نوع صفحه‌کلیدی دارید؟"

#: loader/kickstart.c:133
#, fuzzy, c-format
msgid "Error opening kickstart file %s: %m"
msgstr "خطا در باز کردن پروندهٔ راه‌اندازی %s:‏ %s"

#: loader/kickstart.c:143
#, fuzzy, c-format
msgid "Error reading contents of kickstart file %s: %m"
msgstr "خطا در خواندن محتویات پروندهٔ راه‌اندازی %s: ‏%s"

#: loader/kickstart.c:185
#, c-format
msgid "Error in %s on line %d of kickstart file %s."
msgstr "خطا در %s در سطر %Id پروندهٔ راه‌اندازی %s."

#: loader/kickstart.c:190
#, fuzzy, c-format
msgid "Missing options on line %d of kickstart file %s."
msgstr "خطا در %s در سطر %Id پروندهٔ راه‌اندازی %s."

#: loader/kickstart.c:297
#, fuzzy
msgid "Cannot find ks.cfg on removable media."
msgstr "نمی‌توان ks.cfg را در دیسک نرم راه‌اندازی پیدا کرد."

#: loader/kickstart.c:333
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""

#: loader/kickstart.c:342
#, fuzzy
msgid "Error downloading kickstart file"
msgstr "خطا در باز کردن پروندهٔ راه‌اندازی %s:‏ %s"

#: loader/kickstart.c:511
#, fuzzy, c-format
msgid "Bad argument to shutdown kickstart method command: %s"
msgstr "آرگومان بد برای فرمان روش راه‌اندازی دیسک سخت %s : %s"

#: loader/lang.c:63 loader/loader.c:223
#, fuzzy, c-format
msgid "Welcome to %s for %s"
msgstr "به %s خوش آمدید"

#: loader/lang.c:64
#, fuzzy, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr "به %s خوش آمدید ‐ حالت نجات"

#: loader/lang.c:65 loader/loader.c:247
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr "  <Tab>/<Alt-Tab> بین اجزا  | <Space> انتخاب | <F12> صفحهٔ بعد "

#: loader/lang.c:370
msgid "Choose a Language"
msgstr "یک زبان انتخاب کنید"

#: loader/loader.c:130
#, fuzzy
msgid "Local CD/DVD"
msgstr "سی‌دی محلی"

#: loader/loader.c:131
msgid "Hard drive"
msgstr "دیسک سخت"

#: loader/loader.c:132
#, fuzzy
msgid "NFS directory"
msgstr "شاخهٔ %s:"

#: loader/loader.c:460 loader/loader.c:501
msgid "Update Disk Source"
msgstr "به‌هنگام‌سازی مبدأ دیسک"

#: loader/loader.c:461
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""
"شما چند دستگاه دارید که می‌توانند به عنوان مبدأ دیسک به‌هنگام‌سازی مورد استفاده "
"قرار بگیرند. از کدام یک می‌خواهید استفاده کنید؟"

#: loader/loader.c:502
#, fuzzy
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""
"چند افراز روی این دستگاه است که می‌تواند حاوی تصویر دیسک گرداننده باشند. از "
"کدام یک می‌خواهید استفاده کنید؟"

#: loader/loader.c:520
#, fuzzy, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr ""
"دیسک به‌هنگام‌سازی خود را در ‎/dev/%s وارد کنید و برای ادامه روی دکمهٔ «تأیید» "
"کلیک کنید."

#: loader/loader.c:523
msgid "Updates Disk"
msgstr "دیسک به‌هنگام‌سازی"

#: loader/loader.c:541
msgid "Failed to mount updates disk"
msgstr "سوار کردن دیسک به‌هنگام‌سازی شکست خورد"

#: loader/loader.c:546
msgid "Updates"
msgstr "به‌هنگام‌سازی"

#: loader/loader.c:546
#, fuzzy
msgid "Reading anaconda updates"
msgstr "در حال خواندن به‌هنگام‌سازی‌های آناکوندا..."

#: loader/loader.c:582
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""

#: loader/loader.c:591
#, fuzzy
msgid "Error downloading updates image"
msgstr "خطا در باز کردن پروندهٔ راه‌اندازی %s:‏ %s"

#: loader/loader.c:1190
#, c-format
msgid "You do not have enough RAM to install %s on this machine."
msgstr "شما برای نصب %s روی این رایانه به اندازهٔ کافی حافظهٔ RAM ندارید."

#: loader/loader.c:1244
msgid "Media Detected"
msgstr "رسانه تشخیص داده شد"

#: loader/loader.c:1245
#, fuzzy
msgid "Found local installation media"
msgstr "رسانهٔ نصب محلی کشف شد..."

#: loader/loader.c:1367
msgid "Rescue Method"
msgstr "روش نجات"

#: loader/loader.c:1368
msgid "Installation Method"
msgstr "روش نصب"

#: loader/loader.c:1370
msgid "What type of media contains the rescue image?"
msgstr "چه نوع رسانه‌ای حاوی تصویر نجات است؟"

#: loader/loader.c:1372
#, fuzzy
msgid "What type of media contains the installation image?"
msgstr "چه نوع رسانه‌ای حاوی تصویر نجات است؟"

#: loader/loader.c:1407
msgid "No driver found"
msgstr "گرداننده‌ای پیدا نشد"

#: loader/loader.c:1407
msgid "Select driver"
msgstr "انتخاب گرداننده"

#: loader/loader.c:1408
msgid "Use a driver disk"
msgstr "استفاده از دیسکِ گرداننده"

#: loader/loader.c:1409
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""
"نمی‌توان هیچ دستگاهی از نوعی که برای این نوع نصب لازم است پیدا کرد. آیا "
"می‌خواهید گردانندهٔ خود را دستی انتخاب کنید یا از یک دیسکِ گرداننده استفاده "
"کنید؟"

#: loader/loader.c:1637
msgid "The following devices have been found on your system."
msgstr "دستگاه‌های زیر در سیستم شما پیدا شده‌اند."

#: loader/loader.c:1639
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""
"هیچ گردانندهٔ دستگاهی برای سیستم‌تان بار نشده است. آیا می‌خواهید حالا یک "
"گرداننده بار کنید؟"

#: loader/loader.c:1643
msgid "Devices"
msgstr "دستگاه‌ها"

#: loader/loader.c:1644
msgid "Done"
msgstr "انجام شد"

#: loader/loader.c:1645
msgid "Add Device"
msgstr "اضافه کردن دستگاه"

#: loader/loader.c:1921
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr "بارکننده قبلاً اجرا شده است. پوسته آغاز می‌شود.\n"

#: loader/loader.c:2361
#, fuzzy, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr "در حال اجرای آناکوندا، حالت نجات %s ‐ لطفاً صبر کنید...\n"

#: loader/loader.c:2363
#, fuzzy, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr "در حال اجرای آناکوندا، برنامهٔ نصب سیستم %s ‐ لطفاً صبر کنید...\n"

#: loader/mediacheck.c:47
#, c-format
msgid "Unable to find install image %s"
msgstr "تصویر نصب %s پیدا نشد"

#: loader/mediacheck.c:53
#, fuzzy, c-format
msgid "Checking \"%s\"."
msgstr "در حال بررسی «%s»..."

#: loader/mediacheck.c:55
#, fuzzy, c-format
msgid "Checking media."
msgstr "حالا در حال بررسی رسانه..."

#: loader/mediacheck.c:87
#, fuzzy
msgid ""
"Unable to find the checksum in the image.  This probably means the disc was "
"created without adding the checksum."
msgstr ""
"خواندن مجموعِ مقابله‌ای دیسک از توصیفگرِ جلدِ اصلی ممکن نیست. این احتمالاً به این "
"معنیاست که دیسک بدون اضافه کردن مجموع مقابله‌ای ایجاد شده است."

#: loader/mediacheck.c:94
#, fuzzy
msgid "Unable to open the image."
msgstr "تصویر نصب %s پیدا نشد"

#: loader/mediacheck.c:98
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""

#: loader/mediacheck.c:106
msgid "Success"
msgstr ""

#: loader/mediacheck.c:107
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""

#: loader/method.c:321
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""
"آیا می‌خواهید تصویر ایزو را از نظر مجموع مقابله‌ای آزمایش کنید:\n"
"\n"
"‏   %s؟"

#: loader/method.c:324
msgid "Checksum Test"
msgstr "آزمایش مجموع مقابله‌ای"

#: loader/modules.c:383
#, fuzzy, c-format
msgid "Bad argument to device kickstart method command: %s"
msgstr "آرگومان بد برای فرمان روش راه‌اندازی دستگاه %s : %s"

#: loader/modules.c:399
#, fuzzy
msgid "A module name must be specified for the kickstart device command."
msgstr ""
"آرگومان‌های نامعتبر زیر برای فرمان راه‌اندازی دیسک گرداننده مشخص شده بودند: %s:"
"%s"

#: loader/net.c:109
#, fuzzy
msgid "Invalid Prefix"
msgstr "رشتهٔ IP‏ نامعتبر"

#: loader/net.c:110
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""

#: loader/net.c:463 loader/net.c:471 loader/net.c:526
#, fuzzy, c-format
msgid "There was an error configuring your network interface."
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: loader/net.c:465
#, c-format
msgid ""
"\n"
"This cannot be corrected in cmdline mode.\n"
"Halting.\n"
msgstr ""

#: loader/net.c:470 loader/net.c:525
msgid "Network Error"
msgstr ""

#: loader/net.c:556 textw/netconfig_text.py:133
msgid "Enable IPv4 support"
msgstr ""

#: loader/net.c:570
msgid "Enable IPv6 support"
msgstr ""

#: loader/net.c:610
msgid "Configure TCP/IP"
msgstr "پیکربندی TCP/IP"

#: loader/net.c:667
#, fuzzy
msgid "Missing Protocol"
msgstr "گروه ناموجود"

#: loader/net.c:668
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr ""

#: loader/net.c:675
msgid "IPv4 Needed for NFS"
msgstr ""

#: loader/net.c:676
msgid "NFS installation method requires IPv4 support."
msgstr ""

#: loader/net.c:775
#, fuzzy
msgid "IPv4 address:"
msgstr "نشانی IP:"

#: loader/net.c:787 loader/net.c:854
msgid "/"
msgstr ""

#: loader/net.c:842
#, fuzzy
msgid "IPv6 address:"
msgstr "نشانی IP:"

#: loader/net.c:908 textw/netconfig_text.py:160
msgid "Gateway:"
msgstr "دروازه:"

#: loader/net.c:916
#, fuzzy
msgid "Name Server:"
msgstr "کارگزار نام"

#: loader/net.c:955
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""

#: loader/net.c:972
#, fuzzy
msgid "Manual TCP/IP Configuration"
msgstr "پیکربندی z/IPL"

#: loader/net.c:1096 loader/net.c:1104
msgid "Missing Information"
msgstr "اطلاعات کم است"

#: loader/net.c:1097
#, fuzzy
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr "باید هم نشانی IP معتبر و هم نقاب شبکه را وارد کنید."

#: loader/net.c:1105
#, fuzzy
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr "باید هم نشانی IP معتبر و هم نقاب شبکه را وارد کنید."

#: loader/net.c:1558
#, fuzzy, c-format
msgid "Bad argument to kickstart network command: %s"
msgstr "آرگومان بد برای فرمان شبکه‌ای راه‌اندازی %s: ‏%s"

#: loader/net.c:1579
#, c-format
msgid "Bad bootproto %s specified in network command"
msgstr ""

#: loader/net.c:1656
#, fuzzy
msgid "Seconds:"
msgstr "کارگزار نام دوم:"

#: loader/net.c:1843
msgid "Networking Device"
msgstr "دستگاه شبکه‌ای"

#: loader/net.c:1844
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""
"شما چند دستگاه شبکهٔ مختلف روی این سیستم دارید. از طریق کدام یک می‌خواهید نصب "
"کنید؟"

#: loader/net.c:1848
msgid "Identify"
msgstr ""

#: loader/net.c:1857
msgid "You can identify the physical port for"
msgstr ""

#: loader/net.c:1859
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""

#: loader/net.c:1866
msgid "Identify NIC"
msgstr ""

#: loader/net.c:1879
#, fuzzy
msgid "Invalid Duration"
msgstr "اطلاعات IP‏ نامعتبر است"

#: loader/net.c:1880
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr ""

#: loader/net.c:1892
#, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr ""

#: loader/net.c:2061 loader/net.c:2065
#, fuzzy, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr "در انتظار برای اتصال تِلنِت..."

#: loader/nfsinstall.c:68
msgid "NFS server name:"
msgstr "نام کارگزار NFS:"

#: loader/nfsinstall.c:72
#, c-format
msgid "%s directory:"
msgstr "شاخهٔ %s:"

#: loader/nfsinstall.c:76
msgid "NFS mount options (optional):"
msgstr ""

#: loader/nfsinstall.c:82
#, c-format
msgid ""
"Please enter the server and path to your %s installation image and "
"optionally additional NFS mount options."
msgstr ""

#: loader/nfsinstall.c:90
msgid "NFS Setup"
msgstr "برپاسازی NFS"

#: loader/nfsinstall.c:280
msgid "That directory could not be mounted from the server."
msgstr "سوار کردن این شاخه از کارگزار ممکن نشد."

#: loader/nfsinstall.c:292
#, fuzzy, c-format
msgid "That directory does not seem to contain a %s installation image."
msgstr "به نظر نمی‌رسد که این شاخه درخت نصب %s داشته باشد."

#: loader/nfsinstall.c:367
#, fuzzy, c-format
msgid "Bad argument to NFS kickstart method command: %s"
msgstr "آرگومان بد برای فرمان روش راه‌اندازی NFS ‏%s : %s"

#: loader/telnetd.c:90 loader/telnetd.c:129
msgid "Telnet"
msgstr "تِلنِت"

#: loader/telnetd.c:90
#, fuzzy
msgid "Waiting for telnet connection."
msgstr "در انتظار برای اتصال تِلنِت..."

#: loader/telnetd.c:129
#, fuzzy
msgid "Running anaconda via telnet."
msgstr "در حال اجرای آناکوندا از طریق تِلنِت..."

#: loader/urlinstall.c:138
#, fuzzy, c-format
msgid "Unable to retrieve %s."
msgstr "بازیابی ‎%s://%s/%s/%s‏ ممکن نیست."

#: loader/urlinstall.c:222
msgid "Unable to retrieve the install image."
msgstr "بازیابی تصویر نصب ممکن نیست."

#: loader/urlinstall.c:373
#, fuzzy, c-format
msgid "Bad argument to URL kickstart method command: %s"
msgstr "آرگومان بد برای فرمان روش راه‌اندازی نشانی اینترنتی %s: %s"

#: loader/urlinstall.c:384
msgid "Must supply a --url argument to Url kickstart method."
msgstr "باید یک آرگومان ‎--url به روش راه‌اندازی نشانی اینترنتی بدهید."

#: loader/urlinstall.c:391
#, c-format
msgid "Unknown Url method %s"
msgstr "روش نشانی اینترنتی نامعلوم %s"

#: loader/urls.c:168 loader/urls.c:176
msgid "Retrieving"
msgstr "در حال بازیابی"

#: loader/urls.c:262
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr ""

#: loader/urls.c:280
msgid "Enable HTTP proxy"
msgstr ""

#: loader/urls.c:294
#, fuzzy
msgid "Proxy URL"
msgstr "برچسب راه‌اندازی نامعتبر"

#: loader/urls.c:299
#, fuzzy
msgid "Username"
msgstr "نام کاربر"

#: loader/urls.c:304
msgid "Password"
msgstr "گذرواژه"

#: loader/urls.c:323
#, fuzzy
msgid "URL Setup"
msgstr "برپاسازی NFS"

#: loader/urls.c:331
#, fuzzy
msgid "You must enter a URL."
msgstr "باید شاخه‌ای وارد کنید."

#: loader/urls.c:337
msgid "URL must be either an ftp or http URL"
msgstr ""

#: loader/windows.c:66
msgid "Loading SCSI driver"
msgstr "در حال بار کردن گردانندهٔ SCSI"

#: loader/windows.c:67
#, fuzzy, c-format
msgid "Loading %s driver"
msgstr "در حال بار کردن گردانندهٔ %s..."

#: storage/__init__.py:108
#, fuzzy
msgid "Unknown Device"
msgstr "نامعلوم"

#: storage/__init__.py:109
#, fuzzy, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""
"هنگام خواندن نصب از تصاویر ایزو خطایی رخ داد. لطفاً تصاویر ایزو خود را بررسی "
"کرده و دوباره امتحان کنید."

#: storage/__init__.py:120
msgid "Installation cannot continue."
msgstr "نمی‌توان نصب را ادامه داد."

#: storage/__init__.py:121
#, fuzzy
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""
"گزینه‌های افرازبندی‌ای که انتخاب کرده‌اید از قبل فعال شده‌اند. دیگر نمی‌توانید به "
"صفحهٔ ویرایش دیسک بازگردید. آیا می‌خواهید روند نصب را ادامه دهید؟"

#: storage/__init__.py:149
#, fuzzy
msgid "Encrypt device?"
msgstr "کلید رمزنگاری"

#: storage/__init__.py:150
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""

#: storage/__init__.py:173
#, fuzzy
msgid "Writing storage configuration to disk"
msgstr "_کوچ دادن افراز به:"

#: storage/__init__.py:174
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost."
msgstr ""

#: storage/__init__.py:179
#, fuzzy
msgid "Go _back"
msgstr "_عقب"

#: storage/__init__.py:180
msgid "_Write changes to disk"
msgstr ""

#: storage/__init__.py:199
msgid "Running..."
msgstr ""

#: storage/__init__.py:200
msgid "Storing encryption keys"
msgstr ""

#: storage/__init__.py:216
#, python-format
msgid "Error storing an encryption key: %s\n"
msgstr ""

#: storage/__init__.py:359
#, fuzzy
msgid "Finding Devices"
msgstr "بستهٔ ناموجود"

#: storage/__init__.py:360
#, fuzzy
msgid "Finding storage devices"
msgstr "بستهٔ ناموجود"

#: storage/__init__.py:639
msgid "This partition is holding the data for the hard drive install."
msgstr "این افراز داده‌های نصب از دیسک سخت را نگه می‌دارد."

#: storage/__init__.py:644
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr "نمی‌توانید یک افراز DASD با قالب‌بندی LDL را حذف کنید."

#: storage/__init__.py:650
#, fuzzy, python-format
msgid "This device is part of the RAID device %s."
msgstr "این افراز بخشی از دستگاه RAID ‏‎/dev/md%s است."

#: storage/__init__.py:653
#, fuzzy
msgid "This device is part of a RAID device."
msgstr "این افراز بخشی از یک دستگاه RAID است."

#: storage/__init__.py:658
#, fuzzy, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr "این افراز بخشی از گروه جلدی LVM‏ «%s» است."

#: storage/__init__.py:661
#, fuzzy
msgid "This device is part of a LVM volume group."
msgstr "این افراز بخشی از یک گروه جلدی LVM است."

#: storage/__init__.py:677
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""

#: storage/__init__.py:953
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr "شما افراز ریشه (/) را که برای ادامهٔ نصب %s ضروری است تعیین نکرده‌اید."

#: storage/__init__.py:958
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""
"افراز ریشهٔ شما کوچکتر از ۲۵۰ مگابایت است که معمولاً کوچکتر از آن است که بتوان "
"%s را روی آن نصب کرد."

#: storage/__init__.py:964
#, fuzzy, python-format
msgid ""
"Your / partition is less than %(min)s MB which is lower than recommended for "
"a normal %(productName)s install."
msgstr ""
"افراز %s شما کوچکتر از %s مگابایت است که کمتر از مقدار توصیه شده برای نصب "
"معمولی %s است."

#: storage/__init__.py:974
#, python-format
msgid ""
"Your / partition does not match the the live image you are installing from.  "
"It must be formatted as %s."
msgstr ""

#: storage/__init__.py:981
#, fuzzy, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s megabytes which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""
"افراز %s شما کوچکتر از %s مگابایت است که کمتر از مقدار توصیه شده برای نصب "
"معمولی %s است."

#: storage/__init__.py:1010
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""

#: storage/__init__.py:1013
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""

#: storage/__init__.py:1020
#, fuzzy
msgid ""
"You have not specified a swap partition.  Due to the amount of memory "
"present, a swap partition is required to complete installation."
msgstr ""
"شما افراز حافظهٔ مبادله‌ای مشخص نکرده‌اید. هرچند وجود چنین افرازی در همهٔ موارد "
"اجباری نیست ولی وجود آن به نحو محسوسی کارآیی بیشتر نصب‌ها را بهبود خواهد "
"بخشید."

#: storage/__init__.py:1025
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""
"شما افراز حافظهٔ مبادله‌ای مشخص نکرده‌اید. هرچند وجود چنین افرازی در همهٔ موارد "
"اجباری نیست ولی وجود آن به نحو محسوسی کارآیی بیشتر نصب‌ها را بهبود خواهد "
"بخشید."

#: storage/__init__.py:1032
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: storage/__init__.py:1036
#, fuzzy, python-format
msgid "The mount point %s must be on a linux file system."
msgstr "نقطهٔ سوار کردن «%s» مورد استفاده است، لطفاً یکی دیگر انتخاب کنید."

#: storage/__init__.py:1047
msgid "No Drives Found"
msgstr "دیسک‌گردانی پیدا نشد"

#: storage/__init__.py:1048
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""
"خطایی رخ داد ‐ دستگاه معتبری برای ایجاد سیستم‌های پرونده‌ای جدید پیدا نشد. "
"لطفاً برای پیدا کردن علت این مشکل سخت‌افزارتان را بررسی کنید."

#: storage/__init__.py:1330 storage/__init__.py:1339
msgid "Dirty File Systems"
msgstr "سیستم‌های پرونده‌ای کثیف"

#: storage/__init__.py:1331
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""
"سیستم‌های پرونده‌ای زیر در سیستم لینوکس شما درست پیاده نشده‌اند. لطفاً نصب "
"لینوکس خود را راه‌اندازی کنید، اجازه دهید سیستم‌های پرونده‌ای کنترل شوند و "
"رایانه درست خاموش شود، سپس سیستم را ارتقاء دهید.\n"
"‏%s"

#: storage/__init__.py:1340
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""
"سیستم‌های پرونده‌ای زیر در سیستم لینوکس شما درست پیاده نشده‌اند. آیا با این "
"وجود می‌خواهید آنها را سوار کنید؟\n"
"‏%s"

#: storage/__init__.py:1817
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for "
"swap space, you must reformat as a new-style Linux swap partition."
msgstr ""

#: storage/__init__.py:1828
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""

#: storage/__init__.py:1836
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""

#: storage/__init__.py:1848
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a supported swap volume.  In order to continue "
"installation, you will need to format the device or skip it."
msgstr ""

#: storage/__init__.py:1859
#, fuzzy, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""
"خطا هنگام به کار انداختن دستگاه مبادلهٔ %s: %s\n"
"\n"
"‏‎/etc/fstab در افراز ارتقای شما به افراز مبادلهٔ معتبری ارجاع نمی‌دهد.\n"
"\n"
"برای راه‌اندازی مجدد سیستم خود روی دکمهٔ «تأیید» کلیک کنید."

#: storage/__init__.py:1866
#, fuzzy, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""
"خطا هنگام به کارانداختن دستگاه مبادلهٔ %s: %s\n"
"\n"
"این احتمالاً بدین معنی است که افراز مبادله راه‌اندازی نشده است.\n"
"\n"
"برای راه‌اندازی مجدد سیستم خود روی دکمهٔ «تأیید» کلیک کنید."

#: storage/__init__.py:1929 storage/__init__.py:1941
msgid "Invalid mount point"
msgstr "نقطهٔ سوار کردن نامعتبر"

#: storage/__init__.py:1930
#, fuzzy, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"هنگام تلاش برای ایجاد %s خطایی رخ داد. عنصری از این مسیر، شاخه نیست. این "
"خطایی مهلک است و نمی‌توان نصب را ادامه داد.\n"
"\n"
"برای راه‌اندازی مجدد سیستم خود <ورود> را فشار دهید."

#: storage/__init__.py:1942
#, fuzzy, python-format
msgid ""
"An error occurred when trying to create %(mountpoint)s: %(msg)s.  This is a "
"fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"هنگام تلاش برای ایجاد %s خطایی رخ داد: %s.  این خطایی مهلک است و نمی‌توان نصب "
"را ادامه داد.\n"
"\n"
"برای راه‌اندازی مجدد سیستم خود <ورود> را فشار دهید."

#: storage/__init__.py:1958 storage/__init__.py:1982
#, fuzzy
msgid "Unable to mount filesystem"
msgstr "بار کردن پرونده ممکن نیست!"

#: storage/__init__.py:1959
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s.  You may "
"continue installation, but there may be problems."
msgstr ""

#: storage/__init__.py:1983
#, fuzzy, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s: %(msg)s. This "
"is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""
"هنگام تلاش برای ایجاد %s خطایی رخ داد: %s.  این خطایی مهلک است و نمی‌توان نصب "
"را ادامه داد.\n"
"\n"
"برای راه‌اندازی مجدد سیستم خود <ورود> را فشار دهید."

#: storage/dasd.py:120
msgid "Unformatted DASD Device Found"
msgid_plural "Unformatted DASD Devices Found"
msgstr[0] ""
msgstr[1] ""

#: storage/dasd.py:122
#, python-format
msgid ""
"Format uninitialized DASD device?\n"
"\n"
"There is %d uninitialized DASD device on this system.  To continue "
"installation, the device must be formatted.  Formatting will remove any data "
"on this device."
msgid_plural ""
"Format uninitialized DASD devices?\n"
"\n"
"There are %d uninitialized DASD devices on this system.  To continue "
"installation, the devices must be formatted.  Formatting will remove any "
"data on these devices."
msgstr[0] ""
msgstr[1] ""

#: storage/dasd.py:151
#, fuzzy
msgid "Formatting DASD Device"
msgid_plural "Formatting DASD Devices"
msgstr[0] "در حال قالب‌بندی"
msgstr[1] "در حال قالب‌بندی"

#: storage/dasd.py:152
#, python-format
msgid "Preparing %d DASD device for use with Linux..."
msgid_plural "Preparing %d DASD devices for use with Linux..."
msgstr[0] ""
msgstr[1] ""

#: storage/devicelibs/lvm.py:317
#, python-format
msgid "vginfo failed for %s"
msgstr ""

#: storage/devicelibs/lvm.py:346
#, python-format
msgid "lvs failed for %s"
msgstr ""

#: storage/devices.py:1258 storage/devices.py:1929 storage/devices.py:2332
#: storage/devices.py:2803 storage/devices.py:3293
#, fuzzy
msgid "Creating"
msgstr "کرواتی"

#: storage/devices.py:1259 storage/devices.py:1930 storage/devices.py:2333
#: storage/devices.py:2804
#, fuzzy, python-format
msgid "Creating device %s"
msgstr "تعویض سی‌دی"

#: storage/devices.py:2426
#, python-format
msgid "A RAID%d set requires atleast %d members"
msgstr ""

#: storage/devices.py:3294
#, fuzzy, python-format
msgid "Creating file %s"
msgstr "بررسی برای بلوک‌های خراب روی ‎/dev/%s‎..."

#: storage/devicetree.py:99
#, fuzzy
msgid "Confirm"
msgstr "تکرار گذرواژه:"

#: storage/devicetree.py:100
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""

#: storage/devicetree.py:1847 storage/devicetree.py:1895
#, fuzzy
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr "این افراز بخشی از یک گروه جلدی LVM است."

#: storage/fcoe.py:67 storage/fcoe.py:68
msgid "Connecting to FCoE SAN"
msgstr ""

#: storage/fcoe.py:120
#, fuzzy
msgid "FCoE not available"
msgstr "راهنما موجود نیست"

#: storage/formats/fs.py:102
msgid "filesystem configuration missing a type"
msgstr ""

#: storage/formats/fs.py:354 storage/formats/luks.py:194
#: storage/formats/lvmpv.py:103 storage/formats/swap.py:161
msgid "Formatting"
msgstr "در حال قالب‌بندی"

#: storage/formats/fs.py:355
#, fuzzy, python-format
msgid "Creating %s filesystem on %s"
msgstr "بررسی برای بلوک‌های خراب روی ‎/dev/%s‎..."

#: storage/formats/fs.py:463
#, fuzzy
msgid "Resizing"
msgstr "در حال بازیابی"

#: storage/formats/fs.py:464
#, fuzzy, python-format
msgid "Resizing filesystem on %s"
msgstr "در حال قالب‌بندی سیستم پرونده‌ای ‎%s..."

#: storage/formats/fs.py:499
#, fuzzy, python-format
msgid "Unknown return code: %d."
msgstr "روش نشانی اینترنتی نامعلوم %s"

#: storage/formats/fs.py:513
#, fuzzy
msgid "Checking"
msgstr "در حال بررسی «%s»..."

#: storage/formats/fs.py:514
#, fuzzy, python-format
msgid "Checking filesystem on %s"
msgstr "بررسی برای بلوک‌های خراب روی ‎/dev/%s‎..."

#: storage/formats/fs.py:531
#, python-format
msgid "%(type)s filesystem check failure on %(device)s: "
msgstr ""

#: storage/formats/fs.py:537
msgid ""
"Errors like this usually mean there is a problem with the filesystem that "
"will require user interaction to repair.  Before restarting installation, "
"reboot to rescue mode or another system that allows you to repair the "
"filesystem interactively.  Restart installation after you have corrected the "
"problems on the filesystem."
msgstr ""

#: storage/formats/fs.py:545 storage/partitioning.py:275
msgid "Unrecoverable Error"
msgstr "خطای غیرقابل ترمیم"

#: storage/formats/fs.py:871
msgid "File system errors left uncorrected."
msgstr ""

#: storage/formats/fs.py:872
#, fuzzy
msgid "Operational error."
msgstr "خطا"

#: storage/formats/fs.py:873
msgid "Usage or syntax error."
msgstr ""

#: storage/formats/fs.py:874
msgid "e2fsck cancelled by user request."
msgstr ""

#: storage/formats/fs.py:875
msgid "Shared library error."
msgstr ""

#: storage/formats/fs.py:1034
msgid ""
"Recoverable errors have been detected or dosfsck has discovered an internal "
"inconsistency."
msgstr ""

#: storage/formats/fs.py:1036
#, fuzzy
msgid "Usage error."
msgstr "خطای دیسک‌گردان هدف"

#: storage/formats/luks.py:48
#, fuzzy
msgid "Encrypted"
msgstr "کلید رمزنگاری"

#: storage/formats/luks.py:195
#, fuzzy, python-format
msgid "Encrypting %s"
msgstr "کلید رمزنگاری"

#: storage/formats/lvmpv.py:104 storage/formats/swap.py:162
#, fuzzy, python-format
msgid "Creating %s on %s"
msgstr "بررسی برای بلوک‌های خراب روی ‎/dev/%s‎..."

#: storage/iscsi.py:83 storage/iscsi.py:84
#, fuzzy
msgid "Scanning iSCSI nodes"
msgstr "در حال بار کردن گردانندهٔ SCSI"

#: storage/iscsi.py:177 storage/iscsi.py:178
#, fuzzy
msgid "Initializing iSCSI initiator"
msgstr "راه‌اندازی"

#: storage/iscsi.py:214
#, fuzzy
msgid "iSCSI not available"
msgstr "راهنما موجود نیست"

#: storage/iscsi.py:216
#, fuzzy
msgid "No initiator name set"
msgstr "نام نامعتبر گروه جلد"

#: storage/iscsi.py:230
msgid "No iSCSI nodes discovered"
msgstr ""

#: storage/iscsi.py:233 storage/iscsi.py:234
msgid "Logging in to iSCSI nodes"
msgstr ""

#: storage/iscsi.py:260
msgid "No new iSCSI nodes discovered"
msgstr ""

#: storage/iscsi.py:263
msgid "Could not log in to any of the discovered nodes"
msgstr ""

#: storage/partitioning.py:190
msgid ""
"Could not find enough free space for automatic partitioning.  Press 'OK' to "
"exit the installer."
msgstr ""

#: storage/partitioning.py:193
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""

#: storage/partitioning.py:222
msgid "Warnings During Automatic Partitioning"
msgstr "اخطارهای هنگام افرازبندی خودکار"

#: storage/partitioning.py:224
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""
"اخطارهای زیر در حین افرازبندی خودکار رخ داده‌اند:\n"
"\n"
"‏%s"

#: storage/partitioning.py:238 storage/partitioning.py:259
#, fuzzy
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""
"\n"
"\n"
"برای راه‌اندازی مجدد سیستم خود روی دکمهٔ «تأیید» کلیک کنید."

#: storage/partitioning.py:240
#, fuzzy, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%(msg)s.%(extra)s"
msgstr ""
"تخصیص افرازهای خواسته شده ممکن نبود: \n"
"\n"
"‏%s.%s"

#: storage/partitioning.py:261
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""

#: storage/partitioning.py:263
msgid "Automatic Partitioning Errors"
msgstr "خطاهای افرازبندی خودکار"

#: storage/partitioning.py:264
#, fuzzy, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %(extra)s"
msgstr ""
"خطاهای زیر در افرازبندی دیسک شما رخ داده‌اند:\n"
"\n"
"‏%s\n"
"\n"
"ممکن است دلیل این اتفاق این باشد که در دیسک(های) شما فضای کافی برای نصب وجود "
"ندارد.%s"

#: storage/partitioning.py:276
#, fuzzy
msgid "The system will now reboot."
msgstr "سیستم اکنون مجدداً راه‌اندازی خواهد شد."

#: storage/zfcp.py:50
msgid "You have not specified a device number or the number is invalid"
msgstr "شمارهٔ دستگاهی مشخص نکرده‌اید یا شماره معتبر نیست."

#: storage/zfcp.py:52
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr "نام درگاه جهانی‌ای مشخص نکرده‌اید یا نام معتبر نیست."

#: storage/zfcp.py:54
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr "شمارهٔ قطعهٔ منطقی FCP را مشخص نکرده‌اید یا شماره معتبر نیست."

#: storage/zfcp.py:131
#, python-format
msgid "Could not free zFCP device %(devnum)s from device ignore list (%(e)s)."
msgstr ""

#: storage/zfcp.py:137
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr ""

#: storage/zfcp.py:149
#, python-format
msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
msgstr ""

#: storage/zfcp.py:160
#, python-format
msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:167
#, python-format
msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
msgstr ""

#: storage/zfcp.py:182
#, python-format
msgid ""
"Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""

#: storage/zfcp.py:188
#, python-format
msgid ""
"LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already configured."
msgstr ""

#: storage/zfcp.py:200
#, python-format
msgid ""
"Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on zFCP "
"device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:209
#, python-format
msgid ""
"Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
"again."
msgstr ""

#: storage/zfcp.py:266
#, python-format
msgid ""
"Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s "
"%(fcplun)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:275
#, python-format
msgid ""
"Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""

#: storage/zfcp.py:293
#, python-format
msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: storage/zfcp.py:319
#, python-format
msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
msgstr ""

#: textw/complete_text.py:33
msgid "<Enter> to exit"
msgstr "<ورود> برای خروج"

#: textw/complete_text.py:62
msgid "Complete"
msgstr "تمام شد"

#: textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr "انتخاب صفحه‌کلید"

#: textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr "چه مدل صفحه‌کلیدی به این رایانه متصل شده است؟"

#: textw/netconfig_text.py:43
#, fuzzy, python-format
msgid "A value is required for the field %s"
msgstr "مقداری برای فیلد «%s» لازم است."

#: textw/netconfig_text.py:88 ui/netconfig.glade.h:1
msgid "Enable network interface"
msgstr ""

#: textw/netconfig_text.py:91
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""

#: textw/netconfig_text.py:130
#, fuzzy
msgid "Use dynamic IP configuration (DHCP)"
msgstr "استفاده از پیکربندی پویای IP‏ (BOOTP/DHCP)"

#: textw/netconfig_text.py:140
#, fuzzy
msgid "IPv4 Address:"
msgstr "نشانی IP:"

#: textw/netconfig_text.py:163
#, fuzzy
msgid "Nameserver:"
msgstr "کارگزار نام"

#: textw/netconfig_text.py:193
#, fuzzy
msgid "Missing Device"
msgstr "بستهٔ ناموجود"

#: textw/netconfig_text.py:194
#, fuzzy
msgid "You must select a network device"
msgstr "باید افرازی را برای ویرایش انتخاب کنید"

#: textw/netconfig_text.py:253
#, fuzzy
msgid "IPv4 Network Mask "
msgstr "‏IP‏/نقاب شبکه"

#: textw/netconfig_text.py:276
#, fuzzy
msgid "Configuring Network Interfaces"
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: textw/netconfig_text.py:276
#, fuzzy
msgid "Waiting for NetworkManager"
msgstr "در انتظار برای اتصال تِلنِت..."

#: textw/netconfig_text.py:282
#, fuzzy
msgid "Error configuring network device"
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: textw/netconfig_text.py:282
#, fuzzy, python-format
msgid "Error configuring network device %s"
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: textw/partition_text.py:58
msgid "Partitioning Type"
msgstr ""

#: textw/partition_text.py:60
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target."
msgstr ""

#: textw/partition_text.py:63
#, fuzzy
msgid "Use entire drive"
msgstr "دیسک‌گردان _نادیده گرفته شود"

#: textw/partition_text.py:64
msgid "Replace existing Linux system"
msgstr ""

#: textw/partition_text.py:65
#, fuzzy
msgid "Use free space"
msgstr "فضای آزادی وجود ندارد"

#: textw/partition_text.py:80
msgid "Which drive(s) do you want to use for this installation?"
msgstr "برای این نصب از کدام دیسک‌گردان(ها) می‌خواهید استفاده کنید؟"

#: textw/partition_text.py:95
#, fuzzy
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr "<Space>,<+>,<-> انتخاب   |   <F2> جزئیات گروه   |   <F12> صفحهٔ بعد"

#: textw/partition_text.py:173
#, fuzzy
msgid "Advanced Storage Options"
msgstr "گزینه‌های اضافی اندازه"

#: textw/partition_text.py:174
#, fuzzy
msgid "How would you like to modify your drive configuration?"
msgstr "می‌خواهید سیستم پرونده‌ای این افراز چطور آماده شود؟"

#: textw/partition_text.py:203
#, fuzzy
msgid "Add FCP Device"
msgstr "اضافه کردن دستگاه FCP"

#: textw/partition_text.py:204 ui/zfcp-config.glade.h:5
#, fuzzy
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""
"رایانه‌های سری z می‌توانند از طریق کانال فیبری (FCP) به دستگاه‌های SCSI صنعتی‐"
"استاندارد دسترسی داشته باشند. برای هر دستگاه ۵ پارامتر باید داده شود: شمارهٔ "
"دستگاه ۱۶ بیتی، شناسهٔ SCSI ‏۱۶ بیتی، نام درگاه جهانی (WWPN) ‏۶۴ بیتی، LUN ‏SCSI ‏"
"۱۶ بیتی و LUN ‏FCP ‏۶۴ بیتی."

#: textw/partition_text.py:228
msgid "No network cards present."
msgstr ""

#: textw/partition_text.py:231
#, fuzzy
msgid "Add FCoE SAN"
msgstr "‏FCP LUN"

#: textw/partition_text.py:235
msgid "Select which NIC is connected to the FCoE SAN."
msgstr ""

#: textw/partition_text.py:251 ui/fcoe-config.glade.h:5
msgid "Use DCB"
msgstr ""

#: textw/partition_text.py:280 ui/iscsi-config.glade.h:7
#, fuzzy
msgid "Configure iSCSI Parameters"
msgstr "پارامترهای Chandev"

#: textw/partition_text.py:281 ui/iscsi-config.glade.h:8
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: textw/partition_text.py:282
#, fuzzy
msgid "Target IP Address"
msgstr "نشانی IP"

#: textw/partition_text.py:283
#, fuzzy
msgid "iSCSI Initiator Name"
msgstr "نام نامعتبر گروه جلد"

#: textw/partition_text.py:284
#, fuzzy
msgid "CHAP username"
msgstr "نام پایگاه FTP:"

#: textw/partition_text.py:285
#, fuzzy
msgid "CHAP password"
msgstr "گذرواژه"

#: textw/partition_text.py:286
msgid "Reverse CHAP username"
msgstr ""

#: textw/partition_text.py:287
msgid "Reverse CHAP password"
msgstr ""

#: textw/progress_text.py:57
msgid "Package Installation"
msgstr "نصب بسته"

#: textw/timezone_text.py:74
#, fuzzy
msgid "In which time zone are you located?"
msgstr "در چه منطقهٔ زمانی‌ای قرار دارید؟"

#: textw/timezone_text.py:92
msgid "System clock uses UTC"
msgstr "ساعت سیستم از وقت گرینویچ استفاده می‌کند"

#: textw/upgrade_bootloader_text.py:116 textw/upgrade_bootloader_text.py:126
msgid "Update boot loader configuration"
msgstr "به‌هنگام‌سازی پیکربندی بارکنندهٔ راه‌انداز"

#: textw/upgrade_bootloader_text.py:129
msgid "Skip boot loader updating"
msgstr "پرش از روی به‌هنگام‌سازی بارکنندهٔ راه‌انداز"

#: textw/upgrade_bootloader_text.py:131
msgid "Create new boot loader configuration"
msgstr "ایجاد پیکربندی جدید بارکنندهٔ راه‌انداز"

#: textw/upgrade_text.py:125
msgid "Free Space"
msgstr "فضای آزاد"

#: textw/upgrade_text.py:143
msgid "RAM detected (MB):"
msgstr "حافظهٔ RAM تشخیص داده شده (مگابایت):"

#: textw/upgrade_text.py:146
msgid "Suggested size (MB):"
msgstr "اندازهٔ پیشنهادی (مگابایت):"

#: textw/upgrade_text.py:149
msgid "Swap file size (MB):"
msgstr "اندازهٔ پروندهٔ مبادله (مگابایت):"

#: textw/upgrade_text.py:157
msgid "Add Swap"
msgstr "اضافه کردن حافظهٔ مبادله"

#: textw/upgrade_text.py:182
msgid "The value you entered is not a valid number."
msgstr "مقداری که وارد کرده‌اید عدد معتبری نیست."

#: textw/upgrade_text.py:214
msgid "Reinstall System"
msgstr "نصب مجدد سیستم"

#: textw/upgrade_text.py:226
msgid "System to Upgrade"
msgstr "سیستمی که ارتقاء داده شود"

#: textw/upgrade_text.py:227
#, fuzzy
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to rescue, or select 'Reinstall System' to freshly install "
"your system."
msgstr ""
"یک یا چند نصب موجود لینوکس روی سیستم شما پیدا شده‌اند.\n"
"\n"
"لطفاً یکی از آنها را برای ارتقاء دادن انتخاب کنید یا برای نصب تازه روی سیستم "
"خود، «نصب مجدد سیستم» را انتخاب کنید."

#: textw/userauth_text.py:30
msgid "Root Password"
msgstr "گذرواژهٔ root"

#: textw/userauth_text.py:33
#, fuzzy
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""
"گذرواژهٔ root را انتخاب کنید. برای اطمینان از دانستن آن و عدم اشتباه در تایپ "
"باید آن را دوبار تایپ کنید. به یاد داشته باشید که گذرواژهٔ root بخشی بحرانی "
"از امنیت سیستم است!"

#: textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr "گذرواژهٔ root باید دارای حداقل ۶ نویسه باشد."

#: textw/userauth_text.py:84 textw/userauth_text.py:191
#, fuzzy, python-format
msgid ""
"You have provided a weak password: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr "آیا مایلید به ارتقاء دادن ادامه دهید؟"

#: textw/userauth_text.py:111
#, fuzzy
msgid "User configuration"
msgstr "پیکربندی VNC"

#: textw/userauth_text.py:114
msgid "Setup a username for regular (non-administrative) use."
msgstr ""

#: textw/userauth_text.py:130
#, fuzzy
msgid "Username:"
msgstr "نام کاربر"

#: textw/userauth_text.py:132
msgid "Full name:"
msgstr ""

#: textw/userauth_text.py:159
#, fuzzy
msgid "User password must be at least 6 characters long."
msgstr "گذرواژهٔ root باید دارای حداقل ۶ نویسه باشد."

#: textw/userauth_text.py:172 textw/userauth_text.py:178
#, fuzzy
msgid "Error with username"
msgstr "خطای داده‌ها"

#: textw/userauth_text.py:173
#, fuzzy
msgid "Username too short"
msgstr "نام کاربر"

#: textw/userauth_text.py:179
#, fuzzy
msgid ""
"Requested username contains non-ASCII characters or spaces, which are not "
"allowed."
msgstr ""
"گذرواژهٔ درخواست شده دارای نویسه‌های غیراَسکی است که استفاده از آنها در گذرواژه "
"مجاز نیست."

#: textw/welcome_text.py:29
#, python-format
msgid "%s"
msgstr "‏%s"

#: textw/welcome_text.py:30
#, python-format
msgid ""
"Welcome to %s!\n"
"\n"
msgstr ""
"به %s خوش آمدید!\n"
"\n"

#: textw/zipl_text.py:36
#, fuzzy
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""
"بارکنندهٔ راه‌انداز z/IPL بعد از تمام شدن نصب روی سیستم شما نصب خواهد شد. "
"اکنون می‌توانید هر پارامتر اضافی chandev و هسته را که رایانه یا برپاسازی شما "
"ممکن است به آنها نیاز داشته باشد وارد کنید."

#: textw/zipl_text.py:68
msgid "z/IPL Configuration"
msgstr "پیکربندی z/IPL"

#: textw/zipl_text.py:76 textw/zipl_text.py:80
msgid "Chandev line "
msgstr "خط Chandev "

#: liveinst/liveinst.desktop.in.h:1
#, fuzzy
msgid "Install"
msgstr "_نصب %s"

#: liveinst/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr ""

#: liveinst/liveinst.desktop.in.h:3
#, fuzzy
msgid "Install to Hard Drive"
msgstr "نصب آغاز می‌شود"

#: liveinst:5
#, fuzzy
msgid "Starting Install to Hard Drive"
msgstr "نصب آغاز می‌شود"

#: ui/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""

#: ui/GroupSelector.glade.h:2
#, fuzzy
msgid "_Deselect"
msgstr "_حذف"

#: ui/GroupSelector.glade.h:3
#, fuzzy
msgid "_Deselect all optional packages"
msgstr "در حال نصب بسته‌ها"

#: ui/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr ""

#: ui/GroupSelector.glade.h:5
#, fuzzy
msgid "_Select"
msgstr "_حذف"

#: ui/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr ""

#: ui/GroupSelector.glade.h:7
msgid "dialog1"
msgstr ""

#: ui/account.glade.h:1
msgid "Confirm:"
msgstr "تکرار گذرواژه:"

#: ui/account.glade.h:2
#, fuzzy
msgid "Root Password:"
msgstr "گذرواژهٔ root"

#: ui/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""
"حساب root برای مدیریت سیستم استفاده می‌شود. برای کاربر root گذرواژه‌ای وارد "
"کنید."

#: ui/adddrive.glade.h:1
#, fuzzy
msgid "_Add drive"
msgstr "دیسک سخت"

#: ui/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr ""

#: ui/addrepo.glade.h:2
#, fuzzy
msgid "<b>Repository _type:</b>"
msgstr "کارگزار نام"

#: ui/addrepo.glade.h:4
#, fuzzy
msgid "Configure _proxy"
msgstr "پیکربندی TCP/IP"

#: ui/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""

#: ui/addrepo.glade.h:9
msgid ""
"Please provide the configuration information for this software repository."
msgstr ""

#: ui/addrepo.glade.h:10
#, fuzzy
msgid "Proxy U_RL (host:port)"
msgstr "برچسب راه‌اندازی نامعتبر"

#: ui/addrepo.glade.h:11
#, fuzzy
msgid "Proxy pass_word"
msgstr "بدون گذرواژه"

#: ui/addrepo.glade.h:12
#, fuzzy
msgid "Proxy u_sername"
msgstr "نام پایگاه FTP:"

#: ui/addrepo.glade.h:13
#, fuzzy
msgid "Repository _URL"
msgstr "برچسب راه‌اندازی نامعتبر"

#: ui/addrepo.glade.h:14
#, fuzzy
msgid "Select A Directory"
msgstr "انتخاب گرداننده"

#: ui/addrepo.glade.h:15
msgid "URL is a _mirror list"
msgstr ""

#: ui/addrepo.glade.h:16
#, fuzzy
msgid "_Directory"
msgstr "شاخهٔ %s:"

#: ui/addrepo.glade.h:17
#, fuzzy
msgid "_Options"
msgstr "گزینه‌های RAID"

#: ui/addrepo.glade.h:18
#, fuzzy
msgid "_Partition"
msgstr "افراز"

#: ui/addrepo.glade.h:19
msgid "_Path"
msgstr ""

#: ui/addrepo.glade.h:20
#, fuzzy
msgid "_Server"
msgstr "کارگزار نام"

#: ui/anaconda.glade.h:1
msgid "Reboo_t"
msgstr "_راه‌اندازی مجدد"

#: ui/anaconda.glade.h:4
msgid "_Next"
msgstr "_بعد"

#: ui/autopart.glade.h:1
#, fuzzy
msgid "<b>Shrink partition _to size (in MB):</b>"
msgstr "کارگزار نام"

#: ui/autopart.glade.h:2
msgid "Volume to Shrink"
msgstr ""

#: ui/autopart.glade.h:3
#, fuzzy
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr "افراز ریشهٔ نصب شما در کدام افراز قرار دارد؟"

#: ui/autopart.glade.h:4
msgid "Which type of installation would you like?"
msgstr ""

#: ui/autopart.glade.h:5
#, fuzzy
msgid "_Encrypt system"
msgstr "کلید رمزنگاری"

#: ui/autopart.glade.h:6
msgid "_Shrink"
msgstr ""

#: ui/blwhere.glade.h:1
#, fuzzy
msgid "/boot"
msgstr "راه‌اندازی مجدد"

#: ui/blwhere.glade.h:2
#, fuzzy
msgid "BIOS Drive Order"
msgstr "ویرایش ترتیب دیسک‌گردان‌ها"

#: ui/blwhere.glade.h:3
#, fuzzy
msgid "Boot loader device"
msgstr "بارکنندهٔ راه‌انداز"

#: ui/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:6
msgid "MBR"
msgstr ""

#: ui/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr ""

#: ui/blwhere.glade.h:9
#, fuzzy
msgid "Where would you like to install the boot loader for your system?"
msgstr "کجا می‌خواهید بارکنندهٔ راه‌انداز را نصب کنید؟"

#: ui/cleardisks.glade.h:1
msgid "<b>Data Storage Devices</b> (to be mounted only)"
msgstr ""

#: ui/cleardisks.glade.h:2
#, fuzzy
msgid "<b>Install Target Devices</b>"
msgstr "_نصب %s"

#: ui/cleardisks.glade.h:3
msgid ""
"<b>Tip:</b> Install target devices will be reformatted and wiped of any "
"data.  Make sure you have backups."
msgstr ""

#: ui/cleardisks.glade.h:4
msgid ""
"Below are the storage devices you've selected to be a part of this "
"installation.  Please indicate using the arrows below which devices you'd "
"like to use as data drives (these will not be formatted, only mounted) and "
"which devices you'd like to use as system drives (these may be formatted)."
msgstr ""

#: ui/create-storage.glade.h:1
msgid ""
"<span size=\"small\" color=\"gray\">Create a RAID formatted partition</span>"
msgstr ""

#: ui/create-storage.glade.h:2
msgid ""
"<span size=\"small\" color=\"gray\">Create a logical volume on selected "
"volume group</span>"
msgstr ""

#: ui/create-storage.glade.h:3
msgid ""
"<span size=\"small\" color=\"gray\">Create an LVM formatted partition</span>"
msgstr ""

#: ui/create-storage.glade.h:4
msgid ""
"<span size=\"small\" color=\"gray\">Facilitates RAID device creation</span>"
msgstr ""

#: ui/create-storage.glade.h:5
msgid ""
"<span size=\"small\" color=\"gray\">General purpose partition creation</span>"
msgstr ""

#: ui/create-storage.glade.h:6
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 1 free LVM formatted "
"partition</span>"
msgstr ""

#: ui/create-storage.glade.h:7
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 2 free RAID formatted "
"partitions</span>"
msgstr ""

#: ui/create-storage.glade.h:8
#, fuzzy
msgid "Create"
msgstr "آزاد"

#: ui/create-storage.glade.h:9
msgid "Create LVM"
msgstr ""

#: ui/create-storage.glade.h:10
#, fuzzy
msgid "Create Partition"
msgstr "انتخاب افراز"

#: ui/create-storage.glade.h:11
#, fuzzy
msgid "Create Software RAID"
msgstr "‏RAID نرم‌افزاری"

#: ui/create-storage.glade.h:12
msgid "Create Storage"
msgstr ""

#: ui/create-storage.glade.h:13
#, fuzzy
msgid "LVM Logical Volume"
msgstr "ساختن جلد منطقی"

#: ui/create-storage.glade.h:14
#, fuzzy
msgid "LVM Physical Volume"
msgstr "جلد‌های منطقی"

#: ui/create-storage.glade.h:15
#, fuzzy
msgid "LVM Volume Group"
msgstr "گروه‌های جلد LVM"

#: ui/create-storage.glade.h:16
#, fuzzy
msgid "RAID Clone"
msgstr "گزینه‌های RAID"

#: ui/create-storage.glade.h:18
#, fuzzy
msgid "RAID Partition"
msgstr "اضافه کردن افراز"

#: ui/create-storage.glade.h:19
#, fuzzy
msgid "Standard Partition"
msgstr "اضافه کردن افراز"

#: ui/detailed-dialog.glade.h:1
#, fuzzy
msgid "Info"
msgstr "نادیده گرفته شود"

#: ui/detailed-dialog.glade.h:2
msgid "_Details"
msgstr ""

#: ui/fcoe-config.glade.h:1
#, fuzzy
msgid "Configure FCoE Parameters"
msgstr "پارامترهای Chandev"

#: ui/fcoe-config.glade.h:2
msgid "NIC:"
msgstr ""

#: ui/fcoe-config.glade.h:3
msgid ""
"Please select the network interface which is connected to\n"
"your FCoE switch."
msgstr ""

#: ui/fcoe-config.glade.h:6
#, fuzzy
msgid "_Add FCoE Disk(s)"
msgstr "‏FCP LUN"

#: ui/filter.glade.h:2
#, no-c-format
msgid "<b>%s devices (%s) selected</b> out of %s devices (%s) total."
msgstr ""

#: ui/filter.glade.h:3
msgid ""
"<b>Tip:</b> Selected drives will not be formatted unless you choose to do so "
"later during installation.  Also, note that post-installation you may mount "
"drives you did not select here by modifying your /etc/fstab file."
msgstr ""

#: ui/filter.glade.h:4
#, fuzzy
msgid "Add Advanced Target"
msgstr "گزینه‌های اضافی اندازه"

#: ui/filter.glade.h:5
#, fuzzy
msgid "Basic Devices"
msgstr "دستگاه‌ها"

#: ui/filter.glade.h:6
msgid "Filter By:"
msgstr ""

#: ui/filter.glade.h:7
#, fuzzy
msgid "Firmware RAID"
msgstr "‏RAID نرم‌افزاری"

#: ui/filter.glade.h:8
msgid "Identifier:"
msgstr ""

#: ui/filter.glade.h:9
msgid ""
"Interconnect\n"
"Vendor\n"
"Identifier"
msgstr ""

#: ui/filter.glade.h:12
msgid "LUN:"
msgstr ""

#: ui/filter.glade.h:13
#, fuzzy
msgid "Multipath Devices"
msgstr "تکثیر دستگاه"

#: ui/filter.glade.h:14
#, fuzzy
msgid "Other SAN Devices"
msgstr "دستگاه‌های RAID"

#: ui/filter.glade.h:15
msgid ""
"Please select the drives you'd like to install the operating system on, as "
"well as any drives you'd like to automatically mount on your system:"
msgstr ""

#: ui/filter.glade.h:16
msgid ""
"Port / Target / LUN\n"
"Target Identifier"
msgstr ""

#: ui/filter.glade.h:18
msgid "Port:"
msgstr ""

#: ui/filter.glade.h:19
msgid "Search"
msgstr ""

#: ui/filter.glade.h:20
msgid "Search By:"
msgstr ""

#: ui/filter.glade.h:21
msgid "Search Results:"
msgstr ""

#: ui/filter.glade.h:22
msgid "Show Identifiers that Include:"
msgstr ""

#: ui/filter.glade.h:23
msgid "Show Only Devices From:"
msgstr ""

#: ui/filter.glade.h:24
msgid "Show Only Devices Using:"
msgstr ""

#: ui/filter.glade.h:25
#, fuzzy
msgid "Target:"
msgstr "دیسک‌گردان‌(های) هدف:"

#: ui/iscsi-config.glade.h:1
#, fuzzy
msgid "<b>CHAP _Password:</b>"
msgstr "_گذرواژه:"

#: ui/iscsi-config.glade.h:2
#, fuzzy
msgid "<b>CHAP _Username:</b>"
msgstr "نام کاربر"

#: ui/iscsi-config.glade.h:3
#, fuzzy
msgid "<b>Reverse CHAP P_assword:</b>"
msgstr "_گذرواژه:"

#: ui/iscsi-config.glade.h:4
#, fuzzy
msgid "<b>Reverse CHAP U_sername:</b>"
msgstr "نام کاربر"

#: ui/iscsi-config.glade.h:5
msgid "<b>_Target IP Address:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:6
msgid "<b>iSCSI Initiator _Name:</b>"
msgstr ""

#: ui/iscsi-config.glade.h:9
msgid "_Add target"
msgstr ""

#: ui/lukspassphrase.glade.h:1
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""

#: ui/lukspassphrase.glade.h:2
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""

#: ui/lukspassphrase.glade.h:3
#, fuzzy
msgid "Confirm passphrase:"
msgstr "تصدیق برگرداندن به حالت پیش‌فرض"

#: ui/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr ""

#: ui/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr ""

#: ui/network.glade.h:1
#, fuzzy
msgid "Configure Network"
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: ui/network.glade.h:2
#, fuzzy
msgid "Hostname:"
msgstr "نام میزبان"

#: ui/network.glade.h:3
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""

#: ui/tasksel.glade.h:1
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""

#: ui/tasksel.glade.h:2
msgid "_Add additional software repositories"
msgstr ""

#: ui/tasksel.glade.h:3
#, fuzzy
msgid "_Modify repository"
msgstr "_تغییر افراز"

#: ui/zfcp-config.glade.h:1
#, fuzzy
msgid "<b>Device number:</b>"
msgstr "شمارهٔ دستگاه"

#: ui/zfcp-config.glade.h:2
#, fuzzy
msgid "<b>FCP LUN:</b>"
msgstr "‏FCP LUN"

#: ui/zfcp-config.glade.h:3
#, fuzzy
msgid "<b>WWPN:</b>"
msgstr "_گذرواژه:"

#: ui/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr "اضافه کردن دستگاه FCP"

#: lang-table.h:1
msgid "Afrikaans"
msgstr "آفریکانس"

#: lang-table.h:2
msgid "Arabic"
msgstr "عربی"

#: lang-table.h:3
msgid "Assamese"
msgstr ""

#: lang-table.h:4
msgid "Bengali"
msgstr "بنگالی"

#: lang-table.h:5
msgid "Bengali(India)"
msgstr "بنگالی (هند)‏"

#: lang-table.h:6
msgid "Bulgarian"
msgstr "بلغاری"

#: lang-table.h:7
msgid "Catalan"
msgstr "کاتالونیایی"

#: lang-table.h:8
msgid "Chinese(Simplified)"
msgstr "چینی (ساده شده)‏"

#: lang-table.h:9
msgid "Chinese(Traditional)"
msgstr "چینی (سنتی)‏"

#: lang-table.h:10
msgid "Croatian"
msgstr "کرواتی"

#: lang-table.h:11
msgid "Czech"
msgstr "چکی"

#: lang-table.h:12
msgid "Danish"
msgstr "دانمارکی"

#: lang-table.h:13
msgid "Dutch"
msgstr "هلندی"

#: lang-table.h:14
msgid "English"
msgstr "انگلیسی"

#: lang-table.h:15
msgid "Estonian"
msgstr "استونیایی"

#: lang-table.h:16
msgid "Finnish"
msgstr "فنلاندی"

#: lang-table.h:17
msgid "French"
msgstr "فرانسوی"

#: lang-table.h:18
msgid "German"
msgstr "آلمانی"

#: lang-table.h:19
#, fuzzy
msgid "Greek"
msgstr "آزاد"

#: lang-table.h:20
msgid "Gujarati"
msgstr "گجراتی"

#: lang-table.h:21
msgid "Hebrew"
msgstr ""

#: lang-table.h:22
msgid "Hindi"
msgstr "هندی"

#: lang-table.h:23
msgid "Hungarian"
msgstr "مجاری"

#: lang-table.h:24
msgid "Icelandic"
msgstr "ایسلندی"

#: lang-table.h:25
msgid "Iloko"
msgstr ""

#: lang-table.h:26
msgid "Indonesian"
msgstr "اندونزیایی"

#: lang-table.h:27
msgid "Italian"
msgstr "ایتالیایی"

#: lang-table.h:28
msgid "Japanese"
msgstr "ژاپنی"

#: lang-table.h:29
msgid "Kannada"
msgstr ""

#: lang-table.h:30
msgid "Korean"
msgstr "کره‌ای"

#: lang-table.h:31
msgid "Macedonian"
msgstr "مقدونی"

#: lang-table.h:32
#, fuzzy
msgid "Maithili"
msgstr "گجراتی"

#: lang-table.h:33
msgid "Malay"
msgstr "مالزیایی"

#: lang-table.h:34
#, fuzzy
msgid "Malayalam"
msgstr "مالزیایی"

#: lang-table.h:35
#, fuzzy
msgid "Marathi"
msgstr "گجراتی"

#: lang-table.h:36
#, fuzzy
msgid "Nepali"
msgstr "بنگالی"

#: lang-table.h:37
#, fuzzy
msgid "Norwegian(Bokmål)"
msgstr "نروژی"

#: lang-table.h:38
msgid "Northern Sotho"
msgstr "سوتویی شمالی"

#: lang-table.h:39
msgid "Oriya"
msgstr ""

#: lang-table.h:40
msgid "Persian"
msgstr "فارسی"

#: lang-table.h:41
msgid "Polish"
msgstr "لهستانی"

#: lang-table.h:42
msgid "Portuguese"
msgstr "پرتغالی"

#: lang-table.h:43
msgid "Portuguese(Brazilian)"
msgstr "پرتغالی (برزیل)‏"

#: lang-table.h:44
msgid "Punjabi"
msgstr "پنجابی"

#: lang-table.h:45
#, fuzzy
msgid "Romanian"
msgstr "کرواتی"

#: lang-table.h:46
msgid "Russian"
msgstr "روسی"

#: lang-table.h:47
#, fuzzy
msgid "Serbian"
msgstr "فارسی"

#: lang-table.h:48
msgid "Serbian(Latin)"
msgstr ""

#: lang-table.h:49
msgid "Sinhala"
msgstr ""

#: lang-table.h:50
msgid "Slovak"
msgstr "اسلواکی"

#: lang-table.h:51
msgid "Slovenian"
msgstr "اسلوینیایی"

#: lang-table.h:52
msgid "Spanish"
msgstr "اسپانیایی"

#: lang-table.h:53
msgid "Swedish"
msgstr "سوئدی"

#: lang-table.h:54
#, fuzzy
msgid "Tajik"
msgstr "تامیلی"

#: lang-table.h:55
msgid "Tamil"
msgstr "تامیلی"

#: lang-table.h:56
msgid "Telugu"
msgstr ""

#: lang-table.h:57
msgid "Turkish"
msgstr "ترکی"

#: lang-table.h:58
msgid "Ukrainian"
msgstr "اوکراینی"

#: lang-table.h:59
msgid "Vietnamese"
msgstr "ویتنامی"

#: lang-table.h:60
msgid "Welsh"
msgstr "ویلزی"

#: lang-table.h:61
msgid "Zulu"
msgstr "زولویی"

#: sabayon/livecd.py:65
#, python-format
msgid ""
"The given location [%s] isn't a valid %s live CD to use as an installation "
"source."
msgstr ""

#: sabayon/livecd.py:108
#, fuzzy
msgid "Recovering Kogaion."
msgstr "پیکربندی VNC"

#: sabayon/livecd.py:114
#, fuzzy
msgid "Installing Kogaion onto hard drive."
msgstr "در حال نصب بارکنندهٔ راه‌انداز..."

#: sabayon/livecd.py:128
#, fuzzy
msgid "Configuring Kogaion"
msgstr "پیکربندی VNC"

#: sabayon/livecd.py:158
msgid "Kogaion configuration complete"
msgstr ""

#: sabayon/utils.py:405
#, fuzzy
msgid "Configuring System Services"
msgstr "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#: sabayon/utils.py:684
msgid "An issue occured while installing"
msgstr ""

#: sabayon/utils.py:686
msgid "Meh."
msgstr ""

#: sabayon/utils.py:687
#, fuzzy
msgid "Drivers installation issue"
msgstr "نصب گرافیکی آغاز می‌شود..."

#: sabayon/utils.py:716
#, fuzzy
msgid "System Installation"
msgstr "پس از نصب"

#: sabayon/utils.py:848
#, fuzzy
msgid "Cleaning packages"
msgstr "در حال نصب بسته‌ها"

#: sabayon/utils.py:873
#, fuzzy
msgid "Cleaning package"
msgstr "بستهٔ ناموجود"

#: sabayon/utils.py:917
#, fuzzy
msgid "Installation complete"
msgstr "روش نصب"

#: sabayon/utils.py:925
msgid ""
"The following language packs are available for download (you need Internet), "
"would you like to install them?"
msgstr ""

#: sabayon/utils.py:929
#, fuzzy
msgid "Language packs download"
msgstr "نصب بسته"

#: sabayon/utils.py:950
msgid "No language packs are available for download, sorry!"
msgstr ""

#: sabayon/utils.py:951 sabayon/utils.py:961
#, fuzzy
msgid "Language packs"
msgstr "نصب بسته"

#: sabayon/utils.py:960
msgid "No language packs are available for install, sorry!"
msgstr ""

#: sabayon/utils.py:970
#, fuzzy
msgid "Downloading package"
msgstr "بستهٔ ناموجود"

#: sabayon/utils.py:979
#, fuzzy
msgid "Installing package"
msgstr "در حال نصب بسته‌ها"

#: sabayon/utils.py:996 sabayon/utils.py:1006
msgid "Reordering Entropy mirrors"
msgstr ""

#: sabayon/utils.py:996
msgid "can take some time..."
msgstr ""

#: sabayon/utils.py:1034
msgid "No repositories specified in"
msgstr ""

#: sabayon/utils.py:1036 sabayon/utils.py:1041 sabayon/utils.py:1048
msgid "Repositories update"
msgstr ""

#: sabayon/utils.py:1040
msgid "Unhandled exception"
msgstr ""

#: sabayon/utils.py:1047
msgid "Cannot download repositories right now, no big deal"
msgstr ""

#: sabayon/utils.py:1170
#, fuzzy
msgid ""
"This medium contains many extra languages, would you like to keep them "
"installed?"
msgstr "در طی روند نصب می‌خواهید از چه زبانی استفاده کنید؟"

#: sabayon/utils.py:1173
#, fuzzy
msgid "Language packs installation"
msgstr "نصب بسته"

#: sabayon/utils.py:1184
msgid "Generating list of files to copy"
msgstr ""

#, fuzzy
#~ msgid "Upgrade an Existing Installation"
#~ msgstr "_ارتقاء دادن نصب موجود"

#~ msgid "Reboot"
#~ msgstr "راه‌اندازی مجدد"

#~ msgid "Install class forcing text mode installation"
#~ msgstr "ردهٔ نصبی‌ای که نصب در حالت متنی را اجباری می‌کند"

#, fuzzy
#~ msgid ""
#~ "The following error was found while parsing thekickstart configuration "
#~ "file:\n"
#~ "\n"
#~ "%s"
#~ msgstr ""
#~ "خطای زیر هنگام تجزیهٔ پیکربندی kickstart پیدا شد:\n"
#~ "\n"
#~ "‏%s"

#, fuzzy
#~ msgid "An error was encountered while resizing device %s."
#~ msgstr ""
#~ "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#, fuzzy
#~ msgid "Device Creation Failed"
#~ msgstr "سوار کردن شکست خورد"

#, fuzzy
#~ msgid "An error was encountered while creating device %s."
#~ msgstr ""
#~ "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#, fuzzy
#~ msgid "An error was encountered while removing device %s."
#~ msgstr ""
#~ "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#, fuzzy
#~ msgid "An error was encountered while setting up device %s."
#~ msgstr ""
#~ "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#, fuzzy
#~ msgid "An error was encountered while formatting device %s."
#~ msgstr ""
#~ "هنگام پاک کردن دیسک‌گردان‌های هدف خطایی پیش آمد. همانندسازی شکست خورد."

#, fuzzy
#~ msgid ""
#~ "An error was encountered while activating your storage configuration."
#~ msgstr ""
#~ "خطای زیر هنگام تجزیهٔ پیکربندی kickstart پیدا شد:\n"
#~ "\n"
#~ "‏%s"

#, fuzzy
#~ msgid "Invalid Key"
#~ msgstr "رشتهٔ IP‏ نامعتبر"

#, fuzzy
#~ msgid "The key you entered is invalid."
#~ msgstr "مقداری که وارد کرده‌اید عدد معتبری نیست."

#, fuzzy
#~ msgid "_Skip"
#~ msgstr "پرش"

#~ msgid "The installer will now exit..."
#~ msgstr "حالا برنامه نصب خارج می‌شود..."

#, fuzzy
#~ msgid "/boot is not on an ext2 filesystem."
#~ msgstr "افرازهای قابل راه‌اندازی نمی‌توانند روی یک جلد منطقی باشند."

#, fuzzy
#~ msgid "Please enter your %(instkey)s"
#~ msgstr "لطفاً نام گروه جلد را وارد کنید."

#~ msgid "%s not found"
#~ msgstr "‏%s پیدا نشد"

#~ msgid "%s %s installation"
#~ msgstr "نصب %s %s"

#, fuzzy
#~ msgid "Web Server"
#~ msgstr "کارگزار نام"

#, fuzzy
#~ msgid "Red Hat Enterprise Linux"
#~ msgstr "‏Red Hat Enterprise Linux WS"

#, fuzzy
#~ msgid "Virtualization"
#~ msgstr "تبریک"

#, fuzzy
#~ msgid "Installation Number"
#~ msgstr "نصبی که آغاز شود"

#~ msgid "_Install %s"
#~ msgstr "_نصب %s"

#~ msgid "The following installed system will be upgraded:"
#~ msgstr "سیستم نصب‌شدهٔ زیر ارتقاء داده خواهد شد:"

#~ msgid "Unknown"
#~ msgstr "نامعلوم"

#~ msgid "Not supported"
#~ msgstr "پشتیبانی نمی‌شود"

#~ msgid "LVM is NOT supported on this platform."
#~ msgstr "‏LVM در این بستر پشتیبانی نمی‌شود."

#~ msgid "Software RAID is NOT supported on this platform."
#~ msgstr "‏RAID نرم‌افزاری در این بستر پشتیبانی نمی‌شود."

#~ msgid "No RAID minor device numbers available"
#~ msgstr "هیچ شمارهٔ دستگاه RAID کوچکتری موجود نیست"

#~ msgid ""
#~ "A software RAID device cannot be created because all of the available "
#~ "RAID minor device numbers have been used."
#~ msgstr ""
#~ "نمی‌توان دستگاه RAID نرم‌افزاری ایجاد کرد چون همهٔ شماره‌های کوچکتر موجود "
#~ "برای دستگاه‌های RAID استفاده شده‌اند."

#~ msgid "What do you want to do now?"
#~ msgstr "حالا می‌خواهید چه‌کار کنید؟"

#~ msgid "Create a RAID _device [default=/dev/md%s]."
#~ msgstr "ایجاد _دستگاه RAID [پیش‌فرض=‎/dev/md%s]."

#~ msgid "Clone a _drive to create a RAID device [default=/dev/md%s]."
#~ msgstr "همانندسازی یک دیسک_گردان برای ایجاد دستگاه RAID [پیش‌فرض=‎/dev/md%s]."

#~ msgid "Ne_w"
#~ msgstr "_جدید"

#~ msgid "R_AID"
#~ msgstr "‏R_AID"

#~ msgid "_LVM"
#~ msgstr "‏_LVM"

#~ msgid "Hide RAID device/LVM Volume _Group members"
#~ msgstr "مخفی کردن دستگاه RAID/اعضای _گروه جلدی LVM"

#, fuzzy
#~ msgid ""
#~ "Clone Drive Tool\n"
#~ "\n"
#~ "This tool allows you to significantly reduce the amount of effort "
#~ "required to setup RAID arrays.  This tool uses a source drive which has "
#~ "been prepared with the desired partitioning layout, and clones this "
#~ "layout onto other similar sized drives.  Then a RAID device can be "
#~ "created.\n"
#~ "\n"
#~ "NOTE: The source drive must have partitions which are restricted to be on "
#~ "that drive only, and can only contain unused software RAID partitions.  "
#~ "Other partition types are not allowed.\n"
#~ "\n"
#~ "EVERYTHING on the target drive(s) will be destroyed by this process."
#~ msgstr ""
#~ "ابزار همانندسازی دیسک‌گردان\n"
#~ "\n"
#~ "این ابزار شما را قادر می‌سازد با کمترین دردسر آرایه‌های RAID را برپاسازی "
#~ "کنید. طرح این است که دیسک‌گردان مبدأ را که با چیدمان افرازبندی مورد نظر "
#~ "آماده شده بگیریم و این چیدمان را روی دیسک‌گردان‌های هم‌اندازهٔ دیگر "
#~ "همانندسازی کنیم. سپس می‌توان یک دستگاه RAID ایجاد کرد.\n"
#~ "\n"
#~ "توجه: دیسک‌گردان مبدأ باید شامل افرازهایی باشد که فقط به همان دیسک‌گردان "
#~ "محدود شده باشند، و می‌تواند فقط حاوی افرازهایRAID نرم‌افزاری استفاده نشده "
#~ "باشند. دیگر انواع افراز مجاز نیستند.\n"
#~ "\n"
#~ "همه چیز روی دیسک‌گردان(های) مبدأ در این روند از بین خواهد رفت."

#~ msgid "Unknown Host"
#~ msgstr "میزبان نامعلوم"

#~ msgid "%s is not a valid hostname."
#~ msgstr "%s نام میزبان معتبری نیست."

#~ msgid "_Ignore drive"
#~ msgstr "دیسک‌گردان _نادیده گرفته شود"

#, fuzzy
#~ msgid "NIC device name"
#~ msgstr "گرداننده‌ای پیدا نشد"

#, fuzzy
#~ msgid "%s is not a valid NIC device name."
#~ msgstr "%s نام میزبان معتبری نیست."

#, fuzzy
#~ msgid ""
#~ "This release of %s supports the an updated file system, which has several "
#~ "benefits over the file system traditionally shipped in %s.  This "
#~ "installation program can migrate formatted partitions without data loss.\n"
#~ "\n"
#~ "Which of these partitions would you like to migrate?"
#~ msgstr ""
#~ "این انتشار %s از سیستم پرونده‌ای روزنگار ext3 پشتیبانی می‌کند. این سیستم "
#~ "پرونده‌ای نسبت به سیستم پروندهٔ ext2 که به طور سنتی با %s ارسال می‌شود چند "
#~ "برتری دارد. افرازهای قالب‌بندی شدهٔ ext2 را می‌توان بدون از دست دادن داده‌ها "
#~ "به ext3 کوچ داد.\n"
#~ "\n"
#~ "کدام یک از این افرازها را می‌خواهید کوچ دهید؟"

#, fuzzy
#~ msgid "Add _ZFCP LUN"
#~ msgstr "‏FCP LUN"

#, fuzzy
#~ msgid "Add _FCoE SAN"
#~ msgstr "‏FCP LUN"

#, fuzzy
#~ msgid "_Advanced storage configuration"
#~ msgstr "پیکربندی پیشرفتهٔ بارکنندهٔ راه‌انداز"

#, fuzzy
#~ msgid "What drive would you like to _boot this installation from?"
#~ msgstr "در طی روند نصب می‌خواهید از چه زبانی استفاده کنید؟"

#, fuzzy
#~ msgid "Please enter your %(instkey)s."
#~ msgstr "لطفاً نام گروه جلد را وارد کنید."

#, fuzzy
#~ msgid "label1"
#~ msgstr "برچسب"

#, fuzzy
#~ msgid "Use _dynamic IP configuration (DHCP)"
#~ msgstr "استفاده از پیکربندی پویای IP‏ (BOOTP/DHCP)"

#, fuzzy
#~ msgid "<b>IPv4 Address:</b>"
#~ msgstr "نشانی IP"

#, fuzzy
#~ msgid "<b>IPv6 Address:</b>"
#~ msgstr "نشانی IP"

#, fuzzy
#~ msgid "<b>Gateway:</b>"
#~ msgstr "دروازه:"

#, fuzzy
#~ msgid "<b>Nameserver:</b>"
#~ msgstr "کارگزار نام"

#, fuzzy
#~ msgid ""
#~ "You do not have enough RAM to install %s on this machine.\n"
#~ "\n"
#~ "Press <return> to reboot your system.\n"
#~ msgstr "شما برای نصب %s روی این رایانه به اندازهٔ کافی حافظهٔ RAM ندارید."

#, fuzzy
#~ msgid "Unable To File Bug"
#~ msgstr "حذف ممکن نیست"

#, fuzzy
#~ msgid "Bug Filing Not Supported"
#~ msgstr "پشتیبانی نمی‌شود"

#, fuzzy
#~ msgid "Invalid Bug Information"
#~ msgstr "اطلاعات IP‏ نامعتبر است"

#, fuzzy
#~ msgid "Unable To Login"
#~ msgstr "ویرایش ممکن نیست"

#, fuzzy
#~ msgid "Bug Updated"
#~ msgstr "به‌هنگام‌سازی"

#~ msgid "Dump Written"
#~ msgstr "تخلیه نوشته شد"

#~ msgid "Debug"
#~ msgstr "اشکال‌زدایی"

#~ msgid "Exception Occurred"
#~ msgstr "استثنایی رخ داد"

#~ msgid "None"
#~ msgstr "هیچ‌کدام"

#~ msgid ""
#~ "The device %s is LDL formatted instead of CDL formatted.  LDL formatted "
#~ "DASDs are not supported for usage during an install of %s.  If you wish "
#~ "to use this disk for installation, it must be re-initialized causing the "
#~ "loss of ALL DATA on this drive.\n"
#~ "\n"
#~ "Would you like to reformat this DASD using CDL format?"
#~ msgstr ""
#~ "دستگاه %s به جای قالب‌بندی CDL، قالب‌بندی LDL شده است. استفاده از DASD های "
#~ "با قالب‌بندی LDL در طول نصب %s پشتیبانی نمی‌شود. اگر می‌خواهید از این دیسک "
#~ "برای نصب استفاده کنید، باید آن را دوباره راه‌اندازی کرد که باعث از دست "
#~ "رفتن همهٔ داده‌های روی این دیسک‌گردان می‌شود.\n"
#~ "\n"
#~ "آیا می‌خواهید این DASD را با استفاده از قالب CDL دوباره قالب‌بندی کنید؟"

#, fuzzy
#~ msgid ""
#~ "/dev/%s currently has a %s partition layout.  To use this drive for the "
#~ "installation of %s, it must be re-initialized, causing the loss of ALL "
#~ "DATA on this drive.\n"
#~ "\n"
#~ "Would you like to re-initialize this drive?"
#~ msgstr ""
#~ "‏‎/dev/%s در حال حاضر یک چیدمان افراز %s دارد. برای استفاده از این دیسک "
#~ "برای نصب %s، باید آن را دوباره راه‌اندازی کرد که باعث از رفتن همهٔ داده‌های "
#~ "روی آن می‌شود.\n"
#~ "\n"
#~ "آیا می‌خواهید این دیسک‌گردان را قالب‌بندی کنید؟"

#~ msgid "Please wait while formatting drive %s...\n"
#~ msgstr "لطفاً در مدت قالب‌بندی دیسک‌گردان %s صبر کنید...\n"

#, fuzzy
#~ msgid ""
#~ "The partition table on device %s (%s %-0.f MB) was unreadable.\n"
#~ "\n"
#~ "To create new partitions it must be initialized, causing the loss of ALL "
#~ "DATA on this drive.\n"
#~ "\n"
#~ "This operation will override any previous installation choices about "
#~ "which drives to ignore.\n"
#~ "\n"
#~ "Would you like to initialize this drive, erasing ALL DATA?"
#~ msgstr ""
#~ "جدول افرازبندی در دستگاه %s ‏(%s) غیر قابل خواندن است. برای ایجاد افرازهای "
#~ "جدید  باید راه‌اندازی شود، که باعث از دست رفتن همهٔ داده‌های روی این "
#~ "دیسک‌گردان خواهد شد.\n"
#~ "\n"
#~ "این عملیات بر همهٔ انتخاب‌های پیشین نصب شما دربارهٔ این که کدام دیسک‌گردان‌ها "
#~ "نادیده گرفته شوند پیشی خواهد گرفت.\n"
#~ "\n"
#~ "آیا می‌خواهید این دیسک‌گردان را با پاک کردن همهٔ داده‌ها راه‌اندازی کنید؟"

#~ msgid "Save"
#~ msgstr "ذخیره"

#, fuzzy
#~ msgid "Destination file"
#~ msgstr "سوار کردن شکست خورد"

#, fuzzy
#~ msgid ""
#~ "Would you like to upgrade  the installed system to the %s architecture?"
#~ msgstr "می‌خواهید سیستم پرونده‌ای این افراز چطور آماده شود؟"

#~ msgid "%s Bytes"
#~ msgstr "‏%s بایت"

#~ msgid "Processing"
#~ msgstr "در حال پردازش"

#, fuzzy
#~ msgid "boot flag not available for this partition"
#~ msgstr ""
#~ "نمی‌توانید این افراز را حذف کنید:\n"
#~ "\n"

#, fuzzy
#~ msgid "_Ignore drive(s)"
#~ msgstr "دیسک‌گردان _نادیده گرفته شود"

#~ msgid "Your system will now be rebooted."
#~ msgstr "حالا سیستم شما دوباره راه‌اندازی خواهد شد."

#, fuzzy
#~ msgid "Select A File"
#~ msgstr "انتخاب گرداننده"

#, fuzzy
#~ msgid "_Password"
#~ msgstr "_گذرواژه:"
